{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Biohack_extractionipynb",
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "81sGwrhGOn17",
        "outputId": "59085426-3831-4d11-fd96-13e988e19292"
      },
      "source": [
        "#filtering out experiment accessions with annotations of interest\n",
        "experimentacc = []\n",
        "annotcheck = []\n",
        "annotofinterest = [\"Spiral morphotype exponential phase\", \"Spiral morphotype stationary phase\", \"Filamentous morphotype stationary phase\"]\n",
        "exp_annot = open(\"/content/drive/MyDrive/Biohackathon2021/Campylobacter jejuni_exp_anno.txt\", \"r\").readlines()\n",
        "\n",
        "for i in exp_annot:\n",
        "  total = i.split(\"\\t\")\n",
        "  for z in annotofinterest:\n",
        "    if z in total:\n",
        "      experimentacc.append(i.split(\"\\t\")[0])\n",
        "      annotcheck.append(z)\n",
        "print(experimentacc)\n",
        "print(len(experimentacc))\n",
        "print(set(annotcheck))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "['SRR7478650', 'SRR7478651', 'SRR7478652', 'SRR7478653', 'SRR7478654', 'SRR7478655', 'SRR7478656', 'SRR7478657', 'SRR7478658', 'SRR7478659', 'SRR7478660', 'SRR7478661', 'SRR7478662', 'SRR7478663', 'SRR7478664', 'SRR7478665', 'SRR7478666', 'SRR7478667', 'SRR7478668', 'SRR7478669', 'SRR7478670', 'SRR7478671', 'SRR7478672', 'SRR7478673', 'SRR7478674', 'SRR7478675', 'SRR7478676', 'SRR7478677', 'SRR7478678', 'SRR7478679', 'SRR7478680', 'SRR7478681', 'SRR7478682', 'SRR7478683', 'SRR7478684', 'SRR7478685', 'SRR7500694', 'SRR7500695', 'SRR7500696', 'SRR7500697', 'SRR7500698', 'SRR7500699', 'SRR7500700', 'SRR7500701', 'SRR7500702', 'SRR7500703', 'SRR7500704', 'SRR7500705', 'SRR7500706', 'SRR7500707', 'SRR7500708', 'SRR7500709', 'SRR7500710', 'SRR7500711', 'SRR7500712', 'SRR7500713', 'SRR7500714', 'SRR7500715', 'SRR7500716', 'SRR7500717', 'SRR7500718', 'SRR7500719', 'SRR7500720', 'SRR7500721', 'SRR7500722', 'SRR7500723', 'SRR7500724', 'SRR7500725', 'SRR7500726', 'SRR7500727', 'SRR7500728', 'SRR7500729']\n",
            "72\n",
            "{'Spiral morphotype exponential phase', 'Filamentous morphotype stationary phase', 'Spiral morphotype stationary phase'}\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "TNOKk7ZhTszf",
        "outputId": "8ef0a7ab-e234-4b29-c7ad-af53a35dce3c"
      },
      "source": [
        "# filtering for ecoli accessions\n",
        "import pandas as pd\n",
        "df= pd.read_csv(\"/content/drive/MyDrive/c.jejuni/new_ecoliannot.csv\", sep = \",\")\n",
        "ecoliacc = df.iloc[:, 0].tolist()\n",
        "print(ecoliacc)\n",
        "print(len(ecoliacc))"
      ],
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "['SRR8909206', 'SRR8909408', 'SRR8909193', 'SRR8909195', 'SRR8909197', 'SRR8909226', 'SRR8909249', 'SRR8909270', 'SRR8909303', 'SRR8909310', 'SRR8909318', 'SRR8909346', 'SRR8909348', 'SRR8909404', 'SRR8909413', 'SRR8909447', 'SRR8909454', 'SRR8909472', 'SRR8909488', 'SRR8909492', 'SRR8909194', 'SRR8909231', 'SRR8909276', 'SRR8909287', 'SRR8909415', 'SRR8909448', 'SRR8909474', 'SRR8909196', 'SRR8909219', 'SRR8909290', 'SRR8909386', 'SRR8909411', 'SRR8909459', 'SRR8909467', 'SRR8909198', 'SRR8909300', 'SRR8909362', 'SRR8909199', 'SRR8909245', 'SRR8909363', 'SRR8909378', 'SRR8909425', 'SRR8909455', 'SRR8909493', 'SRR8909200', 'SRR8909264', 'SRR8909282', 'SRR8909465', 'SRR8909201', 'SRR8909417', 'SRR8909444', 'SRR8909202', 'SRR8909420', 'SRR8909487', 'SRR8909203', 'SRR8909235', 'SRR8909285', 'SRR8909367', 'SRR8909379', 'SRR8909204', 'SRR8909323', 'SRR8909475', 'SRR8909205', 'SRR8909295', 'SRR8909296', 'SRR8909189', 'SRR8909207', 'SRR8909283', 'SRR8909299', 'SRR8909331', 'SRR8909400', 'SRR8909414', 'SRR8909419', 'SRR8909428', 'SRR8909208', 'SRR8909271', 'SRR8909376', 'SRR8909406', 'SRR8909209', 'SRR8909232', 'SRR8909244', 'SRR8909293', 'SRR8909452', 'SRR8517249', 'SRR8517250', 'SRR8517254', 'SRR8517255', 'SRR8517421', 'SRR8517424', 'SRR8517425', 'SRR8517458', 'SRR8517459', 'SRR8517460', 'SRR8517563', 'SRR8517578', 'SRR8909210', 'SRR8909263', 'SRR8909266', 'SRR8909211', 'SRR8909325', 'SRR8909374', 'SRR8909390', 'SRR8836010', 'SRR8836016', 'SRR8836017', 'SRR8836018', 'SRR8836019', 'SRR8836020', 'SRR4896361', 'SRR4896362', 'SRR4896363', 'SRR4896364', 'SRR4896365', 'SRR4896366', 'SRR6111092', 'SRR6111094', 'SRR8909212', 'SRR8909246', 'SRR8909395', 'SRR8909215', 'SRR8909301', 'SRR8909460', 'SRR8909216', 'SRR8909233', 'SRR8909258', 'SRR8909485', 'SRR8909217', 'SRR8909424', 'SRR8909481', 'SRR8909218', 'SRR8909242', 'SRR8909389', 'SRR8909393', 'SRR8909445', 'SRR8909221', 'SRR8909267', 'SRR8909361', 'SRR8909433', 'SRR8909222', 'SRR8909289', 'SRR8909477', 'SRR8909482', 'SRR8909223', 'SRR8909277', 'SRR8909451', 'SRR8909224', 'SRR8909255', 'SRR8909278', 'SRR8909489', 'SRR8909225', 'SRR8909254', 'SRR8909316', 'SRR8909319', 'SRR8909373', 'SRR8909402', 'SRR8909456', 'SRR8909483', 'SRR8909228', 'SRR8909230', 'SRR8909337', 'SRR8909340', 'SRR8909229', 'SRR8909321', 'SRR8909458', 'SRR8909237', 'SRR8909462', 'SRR2914536', 'SRR2914537', 'SRR2914540', 'SRR2914541', 'SRR2914542', 'SRR2914543', 'SRR2914544', 'SRR2914545', 'SRR2914546', 'SRR2914547', 'SRR2914563', 'SRR2914564', 'SRR2914565', 'SRR2914566', 'SRR2914567', 'SRR2914568', 'SRR2914569', 'SRR2914570', 'SRR2914571', 'SRR8909238', 'SRR8909274', 'SRR8909284', 'SRR8909357', 'SRR3171776', 'SRR3171777', 'SRR3146206', 'SRR3146207', 'SRR8909239', 'SRR8909422', 'SRR8909240', 'SRR8909260', 'SRR8909280', 'SRR8909355', 'SRR8909463', 'SRR8909243', 'SRR8909250', 'SRR8909306', 'SRR8909327', 'SRR8909391', 'SRR8909403', 'SRR8909429', 'SRR8909441', 'SRR8909251', 'SRR8909291', 'SRR8909305', 'SRR8909307', 'SRR8909382', 'SRR8909434', 'SRR8909494', 'SRR8521424', 'SRR8521425', 'SRR8521426', 'SRR8521427', 'SRR8521428', 'SRR8521429', 'SRR8521430', 'SRR8521431', 'SRR8521432', 'SRR8521433', 'SRR8521444', 'SRR8521445', 'SRR8521447', 'SRR8521448', 'SRR8521451', 'SRR8521452', 'SRR8521453', 'SRR8521454', 'SRR8909256', 'SRR8909302', 'SRR8909257', 'SRR8909322', 'SRR8909384', 'SRR8909259', 'SRR8909326', 'SRR8909366', 'SRR8909368', 'SRR8909372', 'SRR8909383', 'SRR8909261', 'SRR8909315', 'SRR8909418', 'SRR8909430', 'SRR8909262', 'SRR8909265', 'SRR8909292', 'SRR8909324', 'SRR8909268', 'SRR8909308', 'SRR8909344', 'SRR8909484', 'SRR8909269', 'SRR8909410', 'SRR8909490', 'SRR8909273', 'SRR8909392', 'SRR8909394', 'SRR8909401', 'SRR8909412', 'SRR8909191', 'SRR8909281', 'SRR8909381', 'SRR8909431', 'SRR8909476', 'SRR8909288', 'SRR8909343', 'SRR1582908', 'SRR1583069', 'SRR1583070', 'SRR1583071', 'SRR1583074', 'SRR1583075', 'SRR1583077', 'SRR1583079', 'SRR5282299', 'SRR5282300', 'SRR5282301', 'SRR5282302', 'SRR5282303', 'SRR5282304', 'SRR5282305', 'SRR5282306', 'SRR5282307', 'SRR5282308', 'SRR5282309', 'SRR5282310', 'SRR5282311', 'SRR5282312', 'SRR8909304', 'SRR8909314', 'SRR8909353', 'SRR8909437', 'SRR8909312', 'SRR8909313', 'SRR8909347', 'SRR8909480', 'SRR8909317', 'SRR8909469', 'SRR8909328', 'SRR8909466', 'SRR8909332', 'SRR8909371', 'SRR8909375', 'SRR8909333', 'SRR8909334', 'SRR8909407', 'SRR8909478', 'SRR8909339', 'SRR8909345', 'SRR8909470', 'SRR8909341', 'SRR8909453', 'SRR8909473', 'SRR8909359', 'SRR8909387', 'SRR8909398', 'SRR8909399', 'SRR8909190', 'SRR8909409', 'SRR8909432', 'SRR8909450', 'SRR8909440', 'SRR8909461', 'SRR8909464', 'SRR8909479', 'SRR9613401', 'SRR9613406', 'SRR9613404', 'SRR9613405', 'SRR8844842', 'SRR8844843', 'SRR8844845', 'SRR8844849', 'SRR8844850', 'SRR8844851', 'SRR8844853', 'SRR8844856', 'SRR8844857', 'SRR2982421', 'SRR2982422', 'SRR2982423', 'SRR2982424', 'SRR2982429', 'SRR2982430', 'SRR2982431', 'SRR2982432', 'SRR2982437', 'SRR2982438', 'SRR2982439', 'SRR2982440', 'SRR2982441', 'SRR2982442', 'SRR2982445', 'SRR2982446', 'SRR2982447', 'SRR2982448', 'SRR2982449', 'SRR2982450', 'SRR2982451', 'SRR8521434', 'SRR8521435', 'SRR8521436', 'SRR8521437', 'SRR8521438', 'SRR8521439', 'SRR8521440', 'SRR8521441', 'SRR8521442', 'SRR8521443', 'SRR8521455', 'SRR8521456', 'SRR8521457', 'SRR8521458', 'SRR8521459', 'SRR8521460', 'SRR8521461', 'SRR8521462', 'ERR2226558', 'ERR2226559', 'ERR2226560', 'ERR2226561', 'ERR2226562', 'ERR2226563', 'ERR2226564', 'ERR2226565', 'ERR2226566', 'ERR2226567', 'ERR2226568', 'ERR2226569', 'ERR2226570', 'ERR2226571', 'ERR2226572', 'ERR2226573', 'ERR2226574', 'ERR2226575', 'ERR2226576', 'ERR2226577', 'ERR2226578', 'ERR2226579', 'ERR2226580', 'ERR2226581', 'ERR2226582', 'ERR2780171', 'ERR2780172', 'ERR2780173', 'ERR2780174', 'ERR2780175', 'ERR2780176']\n",
            "413\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "8uyaYWuRRh2r",
        "outputId": "2ad03234-4ca0-40a4-d097-9bbec8b0de37"
      },
      "source": [
        "import os, time, shutil\n",
        "if 'kallisto' not in os.listdir('/bin/'): \n",
        "    !wget https://github.com/pachterlab/kallisto/releases/download/v0.46.1/kallisto_linux-v0.46.1.tar.gz \n",
        "    !tar -xf kallisto_linux-v0.46.1.tar.gz \n",
        "    !cp /content/kallisto/kallisto /bin/kallisto \n",
        "\n",
        "!kallisto "
      ],
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "--2021-05-19 04:02:52--  https://github.com/pachterlab/kallisto/releases/download/v0.46.1/kallisto_linux-v0.46.1.tar.gz\n",
            "Resolving github.com (github.com)... 192.30.255.113\n",
            "Connecting to github.com (github.com)|192.30.255.113|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://github-releases.githubusercontent.com/26562905/24093700-ff24-11e9-9f0e-db31aa3914e8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210519%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210519T040252Z&X-Amz-Expires=300&X-Amz-Signature=80c4791d8ae887fa9f02bd3766b20a292ebd2939bff5acb9a69bfee77082ba4f&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=26562905&response-content-disposition=attachment%3B%20filename%3Dkallisto_linux-v0.46.1.tar.gz&response-content-type=application%2Foctet-stream [following]\n",
            "--2021-05-19 04:02:52--  https://github-releases.githubusercontent.com/26562905/24093700-ff24-11e9-9f0e-db31aa3914e8?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20210519%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210519T040252Z&X-Amz-Expires=300&X-Amz-Signature=80c4791d8ae887fa9f02bd3766b20a292ebd2939bff5acb9a69bfee77082ba4f&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=26562905&response-content-disposition=attachment%3B%20filename%3Dkallisto_linux-v0.46.1.tar.gz&response-content-type=application%2Foctet-stream\n",
            "Resolving github-releases.githubusercontent.com (github-releases.githubusercontent.com)... 185.199.108.154, 185.199.109.154, 185.199.110.154, ...\n",
            "Connecting to github-releases.githubusercontent.com (github-releases.githubusercontent.com)|185.199.108.154|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 7274037 (6.9M) [application/octet-stream]\n",
            "Saving to: ‘kallisto_linux-v0.46.1.tar.gz’\n",
            "\n",
            "kallisto_linux-v0.4 100%[===================>]   6.94M  19.8MB/s    in 0.4s    \n",
            "\n",
            "2021-05-19 04:02:52 (19.8 MB/s) - ‘kallisto_linux-v0.46.1.tar.gz’ saved [7274037/7274037]\n",
            "\n",
            "kallisto 0.46.1\n",
            "\n",
            "Usage: kallisto <CMD> [arguments] ..\n",
            "\n",
            "Where <CMD> can be one of:\n",
            "\n",
            "    index         Builds a kallisto index \n",
            "    quant         Runs the quantification algorithm \n",
            "    bus           Generate BUS files for single-cell data \n",
            "    pseudo        Runs the pseudoalignment step \n",
            "    merge         Merges several batch runs \n",
            "    h5dump        Converts HDF5-formatted results to plaintext\n",
            "    inspect       Inspects and gives information about an index\n",
            "    version       Prints version information\n",
            "    cite          Prints citation information\n",
            "\n",
            "Running kallisto <CMD> without arguments prints usage information for <CMD>\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "-1ovXw6DSOIR",
        "outputId": "2956f79f-7297-4d01-b759-d28aca70720b"
      },
      "source": [
        "import os\n",
        "path = '/content/drive/My Drive/c.jejuni'  \n",
        "\n",
        "if not os.path.exists(path): \n",
        "    os.mkdir(path) \n",
        "os.chdir(path)\n",
        "\n",
        "indexName = 'Campylobacter_jejuni_gca_001877255.ASM187725v1.idx'\n",
        "\n",
        "if indexName not in os.listdir():\n",
        "  escapedPath = path.split(\" \")[0] + \"\\\\ \" + path.split(\" \")[1] \n",
        "  !wget ftp://ftp.ensemblgenomes.org/pub/release-51/bacteria//fasta/bacteria_67_collection/campylobacter_jejuni_gca_001877255/cdna/Campylobacter_jejuni_gca_001877255.ASM187725v1.cdna.all.fa.gz\n",
        "  !gunzip Campylobacter_jejuni_gca_001877255.ASM187725v1.cdna.all.fa.gz\n",
        "  !kallisto index -i $indexName $escapedPath/Campylobacter_jejuni_gca_001877255.ASM187725v1.cdna.all.fa\n",
        "else:\n",
        "  print('idx already exists!')"
      ],
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "idx already exists!\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "T717GkwpSdgZ"
      },
      "source": [
        "def ENA_url(RunID):\n",
        "  dir2 = \"\"\n",
        "  if 9 < len(RunID) <= 12:\n",
        "      dir2 = \"0\"*(12 - len(RunID)) + RunID[-(len(RunID)-9):] + \"/\"\n",
        "  dirs = RunID[:6] + \"/\" + dir2 + RunID\n",
        "  \n",
        "  return \"ftp://ftp.sra.ebi.ac.uk/vol1/fastq/\" + dirs + \"/\" + RunID + \"_1.fastq.gz\", \"ftp://ftp.sra.ebi.ac.uk/vol1/fastq/\" + dirs + \"/\" + RunID + \".fastq.gz\"\n",
        "\n",
        "def get_RNA_seq_data(RunID, indexName):\n",
        "  print('Getting expression values for ', RunID)\n",
        "  paired, unpaired = ENA_url(RunID)  \n",
        "  go = 0\n",
        "  if RunID not in os.listdir(path): #If folder does not exist, set go to 1\n",
        "    go = 1\n",
        "  else:\n",
        "    print(RunID, ' has already been done')\n",
        "\n",
        "  if go == 1: \n",
        "    for i in range(3): \n",
        "        print('Attempt: ',i)\n",
        "\n",
        "        !kallisto quant -i $indexName -o $RunID --single -l 200 -s 20 -t 2 <(curl -L -r 0-1000000000 -m 600 --speed-limit 1000000 --speed-time 30 $unpaired 2> $RunID+'.log')\n",
        "        if '\"n_processed\": 0' in open(RunID+'/run_info.json','r').read():\n",
        "            !kallisto quant -i $indexName -o $RunID --single -l 200 -s 20 -t 2 <(curl -L -r 0-1000000000 -m 600 --speed-limit 1000000 --speed-time 30  $paired 2> $RunID+'.log')\n",
        "            \n",
        "        if 'curl: (28)' not in open(RunID+'+.log','r').read(): \n",
        "            print('All OK! ')\n",
        "            os.remove(RunID+'+.log')\n",
        "            break\n",
        "\n",
        "        if RunID+'+.log' in os.listdir(path):\n",
        "            print('Download too slow. Will try ', 2-i, ' more times')\n",
        "            os.remove(RunID+'+.log')"
      ],
      "execution_count": 15,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AK1qEd1_Tbdd",
        "outputId": "bbd478b6-d551-4865-b934-18532755f8d4"
      },
      "source": [
        "count = 0\n",
        "for i in ecoliacc:\n",
        "  get_RNA_seq_data(i, indexName)\n",
        "  count+=1\n",
        "  print(count)"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "274\n",
            "Getting expression values for  SRR1583071\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "275\n",
            "Getting expression values for  SRR1583074\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 13,442,065 reads, 6,409,558 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "276\n",
            "Getting expression values for  SRR1583075\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,362,599 reads, 5,547,857 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "277\n",
            "Getting expression values for  SRR1583077\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 15,161,354 reads, 5,946,237 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "278\n",
            "Getting expression values for  SRR1583079\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,126,493 reads, 1,889,050 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "279\n",
            "Getting expression values for  SRR5282299\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,912,046 reads, 6,960,886 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "280\n",
            "Getting expression values for  SRR5282300\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,195,713 reads, 4,476,970 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "281\n",
            "Getting expression values for  SRR5282301\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,897,130 reads, 5,102,790 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "282\n",
            "Getting expression values for  SRR5282302\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,924,524 reads, 4,472,263 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "283\n",
            "Getting expression values for  SRR5282303\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,919,321 reads, 4,412,795 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "284\n",
            "Getting expression values for  SRR5282304\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,553,768 reads, 3,858,053 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "285\n",
            "Getting expression values for  SRR5282305\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,783,613 reads, 5,162,473 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "286\n",
            "Getting expression values for  SRR5282306\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "287\n",
            "Getting expression values for  SRR5282307\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,227,665 reads, 3,504,493 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "288\n",
            "Getting expression values for  SRR5282308\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 295,447 reads, 182,918 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 9,961,469 reads, 6,144,592 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "289\n",
            "Getting expression values for  SRR5282309\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 11,435,308 reads, 6,756,670 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "290\n",
            "Getting expression values for  SRR5282310\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,891,187 reads, 3,840,708 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,317,808 reads, 4,078,399 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "291\n",
            "Getting expression values for  SRR5282311\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,767,235 reads, 4,628,652 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "292\n",
            "Getting expression values for  SRR5282312\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 11,043,237 reads, 5,545,818 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "293\n",
            "Getting expression values for  SRR8909304\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "294\n",
            "Getting expression values for  SRR8909314\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,610,229 reads, 3,754,375 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "295\n",
            "Getting expression values for  SRR8909353\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 9,485,977 reads, 6,226,648 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "296\n",
            "Getting expression values for  SRR8909437\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,467,331 reads, 4,370,951 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 53 rounds\n",
            "\n",
            "All OK! \n",
            "297\n",
            "Getting expression values for  SRR8909312\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 955,953 reads, 510,167 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,055,377 reads, 5,365,985 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "298\n",
            "Getting expression values for  SRR8909313\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,374,359 reads, 1,072,860 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "299\n",
            "Getting expression values for  SRR8909347\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "300\n",
            "Getting expression values for  SRR8909480\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 24,735,186 reads, 10,312,559 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "301\n",
            "Getting expression values for  SRR8909317\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,134,610 reads, 3,226,247 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "302\n",
            "Getting expression values for  SRR8909469\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,323,435 reads, 2,642,581 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "303\n",
            "Getting expression values for  SRR8909328\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 504,543 reads, 268,202 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "304\n",
            "Getting expression values for  SRR8909466\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "305\n",
            "Getting expression values for  SRR8909332\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,801,584 reads, 3,788,179 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "306\n",
            "Getting expression values for  SRR8909371\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "307\n",
            "Getting expression values for  SRR8909375\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,161,462 reads, 2,606,736 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "308\n",
            "Getting expression values for  SRR8909333\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "309\n",
            "Getting expression values for  SRR8909334\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 1,646,753 reads, 1,009,050 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "310\n",
            "Getting expression values for  SRR8909407\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,766,567 reads, 5,043,840 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "311\n",
            "Getting expression values for  SRR8909478\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,038,663 reads, 5,571,932 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "312\n",
            "Getting expression values for  SRR8909339\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,613,816 reads, 4,715,517 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,058,732 reads, 5,032,810 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "313\n",
            "Getting expression values for  SRR8909345\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,374,099 reads, 5,363,710 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "314\n",
            "Getting expression values for  SRR8909470\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,491,402 reads, 5,256,407 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "315\n",
            "Getting expression values for  SRR8909341\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "316\n",
            "Getting expression values for  SRR8909453\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "317\n",
            "Getting expression values for  SRR8909473\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,041,418 reads, 4,660,863 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "318\n",
            "Getting expression values for  SRR8909359\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 13,687,953 reads, 8,450,985 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "319\n",
            "Getting expression values for  SRR8909387\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "320\n",
            "Getting expression values for  SRR8909398\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 4,985,483 reads, 1,821,431 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "321\n",
            "Getting expression values for  SRR8909399\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "322\n",
            "Getting expression values for  SRR8909190\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 4,826,990 reads, 3,189,172 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "323\n",
            "Getting expression values for  SRR8909409\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 474,826 reads, 323,194 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 19,201,616 reads, 13,021,722 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "324\n",
            "Getting expression values for  SRR8909432\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,386,971 reads, 3,527,331 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "325\n",
            "Getting expression values for  SRR8909450\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,837,201 reads, 3,126,679 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "326\n",
            "Getting expression values for  SRR8909440\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "327\n",
            "Getting expression values for  SRR8909461\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,943,657 reads, 3,334,766 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 13,253,314 reads, 5,564,871 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "328\n",
            "Getting expression values for  SRR8909464\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,079,964 reads, 2,638,308 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "329\n",
            "Getting expression values for  SRR8909479\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "330\n",
            "Getting expression values for  SRR9613401\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 1,755,620 reads, 794,792 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,295,601 reads, 4,677,287 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "331\n",
            "Getting expression values for  SRR9613406\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "332\n",
            "Getting expression values for  SRR9613404\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "333\n",
            "Getting expression values for  SRR9613405\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,811,773 reads, 1,214,480 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,368,037 reads, 4,487,382 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "334\n",
            "Getting expression values for  SRR8844842\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,737,567 reads, 8,553,524 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "335\n",
            "Getting expression values for  SRR8844843\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 11,064,899 reads, 7,438,657 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "336\n",
            "Getting expression values for  SRR8844845\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 1,961,950 reads, 1,151,225 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "337\n",
            "Getting expression values for  SRR8844849\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "338\n",
            "Getting expression values for  SRR8844850\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 6,555,901 reads, 3,979,961 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "339\n",
            "Getting expression values for  SRR8844851\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "340\n",
            "Getting expression values for  SRR8844853\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 11,776,286 reads, 6,132,197 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "341\n",
            "Getting expression values for  SRR8844856\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "342\n",
            "Getting expression values for  SRR8844857\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,429,606 reads, 4,927,507 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "343\n",
            "Getting expression values for  SRR2982421\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,330,901 reads, 6,098,066 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "344\n",
            "Getting expression values for  SRR2982422\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 9,130,628 reads, 4,251,475 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "345\n",
            "Getting expression values for  SRR2982423\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "346\n",
            "Getting expression values for  SRR2982424\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "347\n",
            "Getting expression values for  SRR2982429\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "348\n",
            "Getting expression values for  SRR2982430\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "349\n",
            "Getting expression values for  SRR2982431\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 22,332,954 reads, 9,916,155 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "350\n",
            "Getting expression values for  SRR2982432\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 21,015,477 reads, 8,077,555 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 68 rounds\n",
            "\n",
            "All OK! \n",
            "351\n",
            "Getting expression values for  SRR2982437\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "352\n",
            "Getting expression values for  SRR2982438\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 22,633,989 reads, 11,877,535 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "353\n",
            "Getting expression values for  SRR2982439\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 22,624,667 reads, 10,568,853 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "354\n",
            "Getting expression values for  SRR2982440\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,520,654 reads, 4,074,114 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "355\n",
            "Getting expression values for  SRR2982441\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "356\n",
            "Getting expression values for  SRR2982442\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 21,088,326 reads, 9,263,381 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "357\n",
            "Getting expression values for  SRR2982445\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "358\n",
            "Getting expression values for  SRR2982446\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 11,778,947 reads, 6,256,261 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "359\n",
            "Getting expression values for  SRR2982447\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,214 reads, 1,123 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,178,240 reads, 6,283,241 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  1  more times\n",
            "Attempt:  2\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 54,238 reads, 27,897 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  0  more times\n",
            "360\n",
            "Getting expression values for  SRR2982448\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "361\n",
            "Getting expression values for  SRR2982449\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "362\n",
            "Getting expression values for  SRR2982450\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 13,830,763 reads, 6,866,780 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 80,995 reads, 40,187 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  1  more times\n",
            "Attempt:  2\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 20,456,092 reads, 10,151,902 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "363\n",
            "Getting expression values for  SRR2982451\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 18,624,212 reads, 8,254,445 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "364\n",
            "Getting expression values for  SRR8521434\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "365\n",
            "Getting expression values for  SRR8521435\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 918,111 reads, 667,040 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  1  more times\n",
            "Attempt:  2\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,984,481 reads, 1,911,242 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "366\n",
            "Getting expression values for  SRR8521436\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "367\n",
            "Getting expression values for  SRR8521437\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "368\n",
            "Getting expression values for  SRR8521438\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,957,673 reads, 1,857,700 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "369\n",
            "Getting expression values for  SRR8521439\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,104,480 reads, 1,404,259 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "370\n",
            "Getting expression values for  SRR8521440\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,732,071 reads, 3,849,251 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "371\n",
            "Getting expression values for  SRR8521441\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "372\n",
            "Getting expression values for  SRR8521442\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "373\n",
            "Getting expression values for  SRR8521443\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "374\n",
            "Getting expression values for  SRR8521455\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "375\n",
            "Getting expression values for  SRR8521456\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 3,089,583 reads, 1,806,012 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "376\n",
            "Getting expression values for  SRR8521457\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 3,696,530 reads, 2,325,182 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "377\n",
            "Getting expression values for  SRR8521458\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 3,703,459 reads, 2,186,189 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "378\n",
            "Getting expression values for  SRR8521459\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 631,233 reads, 416,288 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 4,178,273 reads, 2,624,857 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "379\n",
            "Getting expression values for  SRR8521460\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 3,898,972 reads, 2,455,630 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "380\n",
            "Getting expression values for  SRR8521461\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "381\n",
            "Getting expression values for  SRR8521462\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "382\n",
            "Getting expression values for  ERR2226558\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,299,550 reads, 6,531,590 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "383\n",
            "Getting expression values for  ERR2226559\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 9,052,181 reads, 5,786,929 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "384\n",
            "Getting expression values for  ERR2226560\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,899,140 reads, 5,690,397 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "385\n",
            "Getting expression values for  ERR2226561\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,314,761 reads, 4,690,778 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "386\n",
            "Getting expression values for  ERR2226562\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "387\n",
            "Getting expression values for  ERR2226563\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "388\n",
            "Getting expression values for  ERR2226564\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "389\n",
            "Getting expression values for  ERR2226565\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 12,460,047 reads, 8,501,151 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "390\n",
            "Getting expression values for  ERR2226566\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "391\n",
            "Getting expression values for  ERR2226567\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,296,885 reads, 5,691,687 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 10,322,199 reads, 7,081,908 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "392\n",
            "Getting expression values for  ERR2226568\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,288,008 reads, 4,555,280 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "393\n",
            "Getting expression values for  ERR2226569\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "394\n",
            "Getting expression values for  ERR2226570\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 8,529,435 reads, 5,349,838 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "395\n",
            "Getting expression values for  ERR2226571\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,039,426 reads, 4,432,175 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "396\n",
            "Getting expression values for  ERR2226572\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 7,014,384 reads, 4,418,962 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "All OK! \n",
            "397\n",
            "Getting expression values for  ERR2226573\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "398\n",
            "Getting expression values for  ERR2226574\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "399\n",
            "Getting expression values for  ERR2226575\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 5,789,404 reads, 3,841,985 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "400\n",
            "Getting expression values for  ERR2226576\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "401\n",
            "Getting expression values for  ERR2226577\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "402\n",
            "Getting expression values for  ERR2226578\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "403\n",
            "Getting expression values for  ERR2226579\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "404\n",
            "Getting expression values for  ERR2226580\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "405\n",
            "Getting expression values for  ERR2226581\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "406\n",
            "Getting expression values for  ERR2226582\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "407\n",
            "Getting expression values for  ERR2780171\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "408\n",
            "Getting expression values for  ERR2780172\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "409\n",
            "Getting expression values for  ERR2780173\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 3,471,405 reads, 1,907,269 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 2,767,449 reads, 1,520,322 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  1  more times\n",
            "Attempt:  2\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "410\n",
            "Getting expression values for  ERR2780174\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 22,444 reads, 11,860 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "411\n",
            "Getting expression values for  ERR2780175\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "412\n",
            "Getting expression values for  ERR2780176\n",
            "Attempt:  0\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 9,925,935 reads, 5,581,051 reads pseudoaligned\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "\n",
            "Download too slow. Will try  2  more times\n",
            "Attempt:  1\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "\n",
            "[quant] fragment length distribution is truncated gaussian with mean = 200, sd = 20\n",
            "[index] k-mer length: 31\n",
            "[index] number of targets: 5,577\n",
            "[index] number of k-mers: 4,606,753\n",
            "[index] number of equivalence classes: 5,823\n",
            "[quant] running in single-end mode\n",
            "[quant] will process file 1: /dev/fd/63\n",
            "[quant] finding pseudoalignments for the reads ... done\n",
            "[quant] processed 0 reads, 0 reads pseudoaligned\n",
            "[~warn] no reads pseudoaligned.\n",
            "[   em] quantifying the abundances ... done\n",
            "[   em] the Expectation-Maximization algorithm ran for 52 rounds\n",
            "[~warn] Warning, zero reads pseudoaligned check your input files and index\n",
            "\n",
            "All OK! \n",
            "413\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gT-3q-9N1y0D",
        "outputId": "6b14c384-5ccc-4c96-96a1-c697eb2e5612"
      },
      "source": [
        "#Extractint n_pseudoaligned values - check\n",
        "import os\n",
        "all = os.listdir(\"/content/drive/MyDrive/c.jejuni\")\n",
        "print(len(all))\n",
        "exp0pseudo = []\n",
        "exps = []\n",
        "count = 0\n",
        "for i in all:\n",
        "  if \"SRR\" in i:\n",
        "    z = open(\"/content/drive/MyDrive/c.jejuni/\"+i+\"/run_info.json\", \"r\").readlines()[4].split(\":\")[1].split(\",\\n\")[0]\n",
        "    if z != \" 0.0\":\n",
        "      print(i, z)\n",
        "      exps.append(i)\n",
        "    count +=1\n",
        "print(exps)\n",
        "print(len(exps)) \n",
        "print(count)"
      ],
      "execution_count": 31,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "420\n",
            "SRR8909310  72.4\n",
            "SRR8909201  49.2\n",
            "SRR8909417  49.0\n",
            "SRR8909444  45.8\n",
            "SRR8909202  63.2\n",
            "SRR8909420  63.2\n",
            "SRR8909487  65.5\n",
            "SRR8909203  48.4\n",
            "SRR8909235  48.9\n",
            "SRR8909285  66.8\n",
            "SRR8909367  67.8\n",
            "SRR8909379  48.0\n",
            "SRR8909204  67.0\n",
            "SRR8909323  71.2\n",
            "SRR8909475  69.8\n",
            "SRR8909205  70.7\n",
            "SRR8909295  65.8\n",
            "SRR8909296  70.6\n",
            "SRR8909189  43.6\n",
            "SRR8909207  52.7\n",
            "SRR8909283  55.2\n",
            "SRR8909299  72.5\n",
            "SRR8909331  53.9\n",
            "SRR8909400  52.4\n",
            "SRR8909414  65.5\n",
            "SRR8909419  67.3\n",
            "SRR8909428  64.5\n",
            "SRR8909208  71.5\n",
            "SRR8909271  61.9\n",
            "SRR8909376  64.8\n",
            "SRR8909406  61.8\n",
            "SRR8909209  56.4\n",
            "SRR8909232  66.5\n",
            "SRR8909244  67.5\n",
            "SRR8909293  58.6\n",
            "SRR8909452  67.1\n",
            "SRR8517249  42.0\n",
            "SRR8517250  46.3\n",
            "SRR8517254  50.3\n",
            "SRR8517460  38.9\n",
            "SRR8517578  44.1\n",
            "SRR8909263  67.9\n",
            "SRR8909266  69.1\n",
            "SRR8909211  54.1\n",
            "SRR8909390  61.9\n",
            "SRR8836010  54.4\n",
            "SRR4896361  60.2\n",
            "SRR4896362  60.1\n",
            "SRR4896364  62.4\n",
            "SRR4896365  58.7\n",
            "SRR4896366  60.0\n",
            "SRR6111092  40.8\n",
            "SRR8909246  68.0\n",
            "SRR8909301  67.5\n",
            "SRR8909460  66.0\n",
            "SRR8909216  43.2\n",
            "SRR8909233  55.9\n",
            "SRR8909217  67.9\n",
            "SRR8909424  62.5\n",
            "SRR8909242  68.3\n",
            "SRR8909389  57.4\n",
            "SRR8909393  51.3\n",
            "SRR8909445  54.5\n",
            "SRR8909267  38.4\n",
            "SRR8909361  56.0\n",
            "SRR8909433  68.9\n",
            "SRR8909222  69.0\n",
            "SRR8909289  69.1\n",
            "SRR8909477  73.3\n",
            "SRR8909255  71.5\n",
            "SRR8909278  70.9\n",
            "SRR8909489  51.3\n",
            "SRR8909254  56.8\n",
            "SRR8909319  44.8\n",
            "SRR8909456  56.5\n",
            "SRR8909340  38.7\n",
            "SRR8909237  67.6\n",
            "SRR8909462  61.7\n",
            "SRR2914536  38.4\n",
            "SRR2914537  38.5\n",
            "SRR2914544  41.3\n",
            "SRR2914545  48.4\n",
            "SRR2914547  45.7\n",
            "SRR2914566  45.3\n",
            "SRR2914569  45.6\n",
            "SRR8909238  54.4\n",
            "SRR8909274  50.6\n",
            "SRR8909284  67.1\n",
            "SRR8909357  46.7\n",
            "SRR3171776  57.3\n",
            "SRR3171777  53.4\n",
            "SRR8909239  47.0\n",
            "SRR8909422  43.2\n",
            "SRR8909240  50.5\n",
            "SRR8909260  51.0\n",
            "SRR8909280  58.5\n",
            "SRR8909243  57.1\n",
            "SRR8909250  68.9\n",
            "SRR8909306  42.4\n",
            "SRR8909327  46.5\n",
            "SRR8909391  67.5\n",
            "SRR8909403  45.4\n",
            "SRR8909429  45.2\n",
            "SRR8909305  39.9\n",
            "SRR8909307  65.5\n",
            "SRR8909382  39.0\n",
            "SRR8909434  54.9\n",
            "SRR8521426  57.7\n",
            "SRR8521427  55.7\n",
            "SRR8521428  64.8\n",
            "SRR8521429  58.9\n",
            "SRR8521430  62.8\n",
            "SRR8521432  65.7\n",
            "SRR8521433  67.4\n",
            "SRR8521444  67.0\n",
            "SRR8521447  56.6\n",
            "SRR8521448  55.8\n",
            "SRR8521451  68.8\n",
            "SRR8521453  61.3\n",
            "SRR8521454  67.2\n",
            "SRR8909256  63.2\n",
            "SRR8909302  69.3\n",
            "SRR8909259  58.8\n",
            "SRR8909326  56.9\n",
            "SRR8909366  57.8\n",
            "SRR8909368  57.6\n",
            "SRR8909372  40.5\n",
            "SRR8909383  67.9\n",
            "SRR8909315  69.5\n",
            "SRR8909418  65.9\n",
            "SRR8909430  69.4\n",
            "SRR8909262  38.6\n",
            "SRR8909265  67.4\n",
            "SRR8909308  57.1\n",
            "SRR8909344  72.5\n",
            "SRR8909484  74.4\n",
            "SRR8909269  72.4\n",
            "SRR8909410  70.1\n",
            "SRR8909490  71.1\n",
            "SRR8909273  42.5\n",
            "SRR8909392  72.8\n",
            "SRR8909394  43.0\n",
            "SRR8909401  72.5\n",
            "SRR8909412  71.9\n",
            "SRR8909191  48.1\n",
            "SRR8909281  64.4\n",
            "SRR8909381  56.7\n",
            "SRR8909431  46.4\n",
            "SRR8909476  68.3\n",
            "SRR1582908  43.6\n",
            "SRR1583074  47.7\n",
            "SRR1583075  44.9\n",
            "SRR1583077  39.2\n",
            "SRR1583079  36.8\n",
            "SRR5282299  63.8\n",
            "SRR5282301  57.4\n",
            "SRR5282302  56.4\n",
            "SRR5282303  55.7\n",
            "SRR5282304  45.1\n",
            "SRR5282305  66.3\n",
            "SRR5282307  42.6\n",
            "SRR5282308  61.7\n",
            "SRR5282309  59.1\n",
            "SRR5282310  55.7\n",
            "SRR5282311  52.8\n",
            "SRR5282312  50.2\n",
            "SRR8909314  66.9\n",
            "SRR8909353  65.6\n",
            "SRR8909437  51.6\n",
            "SRR8909312  53.4\n",
            "SRR8909480  41.7\n",
            "SRR8909317  52.6\n",
            "SRR8909469  49.6\n",
            "SRR8909332  48.6\n",
            "SRR8909375  42.3\n",
            "SRR8909334  61.3\n",
            "SRR8909407  64.9\n",
            "SRR8909478  46.3\n",
            "SRR8909339  71.3\n",
            "SRR8909345  72.7\n",
            "SRR8909470  70.2\n",
            "SRR8909473  66.2\n",
            "SRR8909359  61.7\n",
            "SRR8909398  36.5\n",
            "SRR8909190  66.1\n",
            "SRR8909409  67.8\n",
            "SRR8909432  65.5\n",
            "SRR8909450  53.6\n",
            "SRR8909461  42.0\n",
            "SRR8909464  43.4\n",
            "SRR9613401  45.4\n",
            "SRR9613405  43.3\n",
            "SRR8844842  67.2\n",
            "SRR8844843  67.2\n",
            "SRR8844845  58.7\n",
            "SRR8844850  60.7\n",
            "SRR8844853  52.1\n",
            "SRR8844857  66.3\n",
            "SRR2982421  49.5\n",
            "SRR2982422  46.6\n",
            "SRR2982431  44.4\n",
            "SRR2982432  38.4\n",
            "SRR2982438  52.5\n",
            "SRR2982439  46.7\n",
            "SRR2982442  43.9\n",
            "SRR2982446  53.1\n",
            "SRR2982447  51.4\n",
            "SRR2982450  49.6\n",
            "SRR2982451  44.3\n",
            "SRR8521435  64.0\n",
            "SRR8521438  62.8\n",
            "SRR8521439  66.7\n",
            "SRR8521440  67.2\n",
            "SRR8521456  58.5\n",
            "SRR8521457  62.9\n",
            "SRR8521458  59.0\n",
            "SRR8521459  62.8\n",
            "SRR8521460  63.0\n",
            "['SRR8909310', 'SRR8909201', 'SRR8909417', 'SRR8909444', 'SRR8909202', 'SRR8909420', 'SRR8909487', 'SRR8909203', 'SRR8909235', 'SRR8909285', 'SRR8909367', 'SRR8909379', 'SRR8909204', 'SRR8909323', 'SRR8909475', 'SRR8909205', 'SRR8909295', 'SRR8909296', 'SRR8909189', 'SRR8909207', 'SRR8909283', 'SRR8909299', 'SRR8909331', 'SRR8909400', 'SRR8909414', 'SRR8909419', 'SRR8909428', 'SRR8909208', 'SRR8909271', 'SRR8909376', 'SRR8909406', 'SRR8909209', 'SRR8909232', 'SRR8909244', 'SRR8909293', 'SRR8909452', 'SRR8517249', 'SRR8517250', 'SRR8517254', 'SRR8517460', 'SRR8517578', 'SRR8909263', 'SRR8909266', 'SRR8909211', 'SRR8909390', 'SRR8836010', 'SRR4896361', 'SRR4896362', 'SRR4896364', 'SRR4896365', 'SRR4896366', 'SRR6111092', 'SRR8909246', 'SRR8909301', 'SRR8909460', 'SRR8909216', 'SRR8909233', 'SRR8909217', 'SRR8909424', 'SRR8909242', 'SRR8909389', 'SRR8909393', 'SRR8909445', 'SRR8909267', 'SRR8909361', 'SRR8909433', 'SRR8909222', 'SRR8909289', 'SRR8909477', 'SRR8909255', 'SRR8909278', 'SRR8909489', 'SRR8909254', 'SRR8909319', 'SRR8909456', 'SRR8909340', 'SRR8909237', 'SRR8909462', 'SRR2914536', 'SRR2914537', 'SRR2914544', 'SRR2914545', 'SRR2914547', 'SRR2914566', 'SRR2914569', 'SRR8909238', 'SRR8909274', 'SRR8909284', 'SRR8909357', 'SRR3171776', 'SRR3171777', 'SRR8909239', 'SRR8909422', 'SRR8909240', 'SRR8909260', 'SRR8909280', 'SRR8909243', 'SRR8909250', 'SRR8909306', 'SRR8909327', 'SRR8909391', 'SRR8909403', 'SRR8909429', 'SRR8909305', 'SRR8909307', 'SRR8909382', 'SRR8909434', 'SRR8521426', 'SRR8521427', 'SRR8521428', 'SRR8521429', 'SRR8521430', 'SRR8521432', 'SRR8521433', 'SRR8521444', 'SRR8521447', 'SRR8521448', 'SRR8521451', 'SRR8521453', 'SRR8521454', 'SRR8909256', 'SRR8909302', 'SRR8909259', 'SRR8909326', 'SRR8909366', 'SRR8909368', 'SRR8909372', 'SRR8909383', 'SRR8909315', 'SRR8909418', 'SRR8909430', 'SRR8909262', 'SRR8909265', 'SRR8909308', 'SRR8909344', 'SRR8909484', 'SRR8909269', 'SRR8909410', 'SRR8909490', 'SRR8909273', 'SRR8909392', 'SRR8909394', 'SRR8909401', 'SRR8909412', 'SRR8909191', 'SRR8909281', 'SRR8909381', 'SRR8909431', 'SRR8909476', 'SRR1582908', 'SRR1583074', 'SRR1583075', 'SRR1583077', 'SRR1583079', 'SRR5282299', 'SRR5282301', 'SRR5282302', 'SRR5282303', 'SRR5282304', 'SRR5282305', 'SRR5282307', 'SRR5282308', 'SRR5282309', 'SRR5282310', 'SRR5282311', 'SRR5282312', 'SRR8909314', 'SRR8909353', 'SRR8909437', 'SRR8909312', 'SRR8909480', 'SRR8909317', 'SRR8909469', 'SRR8909332', 'SRR8909375', 'SRR8909334', 'SRR8909407', 'SRR8909478', 'SRR8909339', 'SRR8909345', 'SRR8909470', 'SRR8909473', 'SRR8909359', 'SRR8909398', 'SRR8909190', 'SRR8909409', 'SRR8909432', 'SRR8909450', 'SRR8909461', 'SRR8909464', 'SRR9613401', 'SRR9613405', 'SRR8844842', 'SRR8844843', 'SRR8844845', 'SRR8844850', 'SRR8844853', 'SRR8844857', 'SRR2982421', 'SRR2982422', 'SRR2982431', 'SRR2982432', 'SRR2982438', 'SRR2982439', 'SRR2982442', 'SRR2982446', 'SRR2982447', 'SRR2982450', 'SRR2982451', 'SRR8521435', 'SRR8521438', 'SRR8521439', 'SRR8521440', 'SRR8521456', 'SRR8521457', 'SRR8521458', 'SRR8521459', 'SRR8521460']\n",
            "218\n",
            "382\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Kf_gUY2T5EfV",
        "outputId": "5fd696b4-eccf-4a4f-8d43-34796f8b3659"
      },
      "source": [
        "#Used to extract tpm and est_counts columns for the construction of exp matrix\n",
        "import pandas as pd\n",
        "transcripts = []\n",
        "oriexp = open(\"/content/drive/MyDrive/e.coli/ERR2226558/abundance.tsv\", \"r\").readlines() #changed to c.jejuni files\n",
        "for o in oriexp[1:]:\n",
        "  transcript = o.split(\"\\t\")[0]\n",
        "  transcripts.append(transcript)\n",
        "print(len(transcripts))\n",
        "df = pd.DataFrame({\"target_id\":transcripts})\n",
        "for i in exps:\n",
        "  dt = pd.read_csv(\"/content/drive/MyDrive/e.coli/\"+i+\"/abundance.tsv\", sep = \"\\t\")\n",
        "  df[i] = dt[\"tpm\"]\n",
        "print(df)\n",
        "df.to_csv('expmat(e.coli_tpm).csv', index = False)"
      ],
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "5577\n",
            "     target_id    SRR8909310  SRR8909201  ...  SRR8521458  SRR8521459  SRR8521460\n",
            "0     ESA80851  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "1     ESA80852  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "2     ESA80853  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "3     ESA80854  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "4     ESA80855  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "...        ...           ...         ...  ...         ...         ...         ...\n",
            "5572  ESA60472  0.000000e+00    0.000000  ...   12.344600    0.000000    0.000000\n",
            "5573  ESA60473  0.000000e+00    0.000000  ...    0.000000    0.000000    0.000000\n",
            "5574  ESA60457  0.000000e+00    0.170379  ...    1.434460    1.288000    0.988118\n",
            "5575  ESA60456  6.908920e-07    3.157670  ...    1.159490    0.292811    0.898544\n",
            "5576  ESA60399  2.021840e+00    2.319160  ...    0.306617    2.152160    2.051110\n",
            "\n",
            "[5577 rows x 219 columns]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TFj2k3pl-Nob",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "726f96f7-cd3d-40cf-b045-2b0b8b26762b"
      },
      "source": [
        "#Used to construct exp_annot \n",
        "exp_annot = open(\"exp_annot_c.jejuni.txt\",\"w\")\n",
        "s = open(\"/content/drive/MyDrive/c.jejuni/c,jejuniannot.csv\", \"r\").readlines()\n",
        "exp_annot.writelines(s[0])\n",
        "for i in s:\n",
        "  run = i.split(\",\")[0]\n",
        "  print(run)\n",
        "  if run in exps:\n",
        "    exp_annot.writelines(i)\n",
        "exp_annot.close()"
      ],
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "SRR8909192\n",
            "SRR8909206\n",
            "SRR8909408\n",
            "SRR8909193\n",
            "SRR8909195\n",
            "SRR8909197\n",
            "SRR8909226\n",
            "SRR8909249\n",
            "SRR8909270\n",
            "SRR8909303\n",
            "SRR8909310\n",
            "SRR8909318\n",
            "SRR8909346\n",
            "SRR8909348\n",
            "SRR8909404\n",
            "SRR8909413\n",
            "SRR8909447\n",
            "SRR8909454\n",
            "SRR8909472\n",
            "SRR8909488\n",
            "SRR8909492\n",
            "SRR8909194\n",
            "SRR8909231\n",
            "SRR8909276\n",
            "SRR8909287\n",
            "SRR8909415\n",
            "SRR8909448\n",
            "SRR8909474\n",
            "SRR8909196\n",
            "SRR8909219\n",
            "SRR8909290\n",
            "SRR8909386\n",
            "SRR8909411\n",
            "SRR8909459\n",
            "SRR8909467\n",
            "SRR8909198\n",
            "SRR8909300\n",
            "SRR8909362\n",
            "SRR8909199\n",
            "SRR8909245\n",
            "SRR8909363\n",
            "SRR8909378\n",
            "SRR8909425\n",
            "SRR8909455\n",
            "SRR8909493\n",
            "SRR8909200\n",
            "SRR8909264\n",
            "SRR8909282\n",
            "SRR8909465\n",
            "SRR8909201\n",
            "SRR8909417\n",
            "SRR8909444\n",
            "SRR8909202\n",
            "SRR8909420\n",
            "SRR8909487\n",
            "SRR8909203\n",
            "SRR8909235\n",
            "SRR8909285\n",
            "SRR8909367\n",
            "SRR8909379\n",
            "SRR8909204\n",
            "SRR8909323\n",
            "SRR8909475\n",
            "SRR8909205\n",
            "SRR8909295\n",
            "SRR8909296\n",
            "SRR8909189\n",
            "SRR8909207\n",
            "SRR8909283\n",
            "SRR8909299\n",
            "SRR8909331\n",
            "SRR8909400\n",
            "SRR8909414\n",
            "SRR8909419\n",
            "SRR8909428\n",
            "SRR8909208\n",
            "SRR8909271\n",
            "SRR8909376\n",
            "SRR8909406\n",
            "SRR8909209\n",
            "SRR8909232\n",
            "SRR8909244\n",
            "SRR8909293\n",
            "SRR8909452\n",
            "SRR8517249\n",
            "SRR8517250\n",
            "SRR8517254\n",
            "SRR8517255\n",
            "SRR8517421\n",
            "SRR8517424\n",
            "SRR8517425\n",
            "SRR8517458\n",
            "SRR8517459\n",
            "SRR8517460\n",
            "SRR8517563\n",
            "SRR8517578\n",
            "SRR8909210\n",
            "SRR8909263\n",
            "SRR8909266\n",
            "SRR8909211\n",
            "SRR8909325\n",
            "SRR8909374\n",
            "SRR8909390\n",
            "SRR8836010\n",
            "SRR8836016\n",
            "SRR8836017\n",
            "SRR8836018\n",
            "SRR8836019\n",
            "SRR8836020\n",
            "SRR4896361\n",
            "SRR4896362\n",
            "SRR4896363\n",
            "SRR4896364\n",
            "SRR4896365\n",
            "SRR4896366\n",
            "SRR6111092\n",
            "SRR6111094\n",
            "SRR8909212\n",
            "SRR8909246\n",
            "SRR8909395\n",
            "SRR8909215\n",
            "SRR8909301\n",
            "SRR8909460\n",
            "SRR8909216\n",
            "SRR8909233\n",
            "SRR8909258\n",
            "SRR8909485\n",
            "SRR8909217\n",
            "SRR8909424\n",
            "SRR8909481\n",
            "SRR8909218\n",
            "SRR8909242\n",
            "SRR8909389\n",
            "SRR8909393\n",
            "SRR8909445\n",
            "SRR8909221\n",
            "SRR8909267\n",
            "SRR8909361\n",
            "SRR8909433\n",
            "SRR8909222\n",
            "SRR8909289\n",
            "SRR8909477\n",
            "SRR8909482\n",
            "SRR8909223\n",
            "SRR8909277\n",
            "SRR8909451\n",
            "SRR8909224\n",
            "SRR8909255\n",
            "SRR8909278\n",
            "SRR8909489\n",
            "SRR8909225\n",
            "SRR8909254\n",
            "SRR8909316\n",
            "SRR8909319\n",
            "SRR8909373\n",
            "SRR8909402\n",
            "SRR8909456\n",
            "SRR8909483\n",
            "SRR8909228\n",
            "SRR8909230\n",
            "SRR8909337\n",
            "SRR8909340\n",
            "SRR8909229\n",
            "SRR8909321\n",
            "SRR8909458\n",
            "SRR8909237\n",
            "SRR8909462\n",
            "SRR2914536\n",
            "SRR2914537\n",
            "SRR2914540\n",
            "SRR2914541\n",
            "SRR2914542\n",
            "SRR2914543\n",
            "SRR2914544\n",
            "SRR2914545\n",
            "SRR2914546\n",
            "SRR2914547\n",
            "SRR2914563\n",
            "SRR2914564\n",
            "SRR2914565\n",
            "SRR2914566\n",
            "SRR2914567\n",
            "SRR2914568\n",
            "SRR2914569\n",
            "SRR2914570\n",
            "SRR2914571\n",
            "SRR8909238\n",
            "SRR8909274\n",
            "SRR8909284\n",
            "SRR8909357\n",
            "SRR3171776\n",
            "SRR3171777\n",
            "SRR3146206\n",
            "SRR3146207\n",
            "SRR8909239\n",
            "SRR8909422\n",
            "SRR8909240\n",
            "SRR8909260\n",
            "SRR8909280\n",
            "SRR8909355\n",
            "SRR8909463\n",
            "SRR8909243\n",
            "SRR8909250\n",
            "SRR8909306\n",
            "SRR8909327\n",
            "SRR8909391\n",
            "SRR8909403\n",
            "SRR8909429\n",
            "SRR8909441\n",
            "SRR8909251\n",
            "SRR8909291\n",
            "SRR8909305\n",
            "SRR8909307\n",
            "SRR8909382\n",
            "SRR8909434\n",
            "SRR8909494\n",
            "SRR8521424\n",
            "SRR8521425\n",
            "SRR8521426\n",
            "SRR8521427\n",
            "SRR8521428\n",
            "SRR8521429\n",
            "SRR8521430\n",
            "SRR8521431\n",
            "SRR8521432\n",
            "SRR8521433\n",
            "SRR8521444\n",
            "SRR8521445\n",
            "SRR8521447\n",
            "SRR8521448\n",
            "SRR8521451\n",
            "SRR8521452\n",
            "SRR8521453\n",
            "SRR8521454\n",
            "SRR8909256\n",
            "SRR8909302\n",
            "SRR8909257\n",
            "SRR8909322\n",
            "SRR8909384\n",
            "SRR8909259\n",
            "SRR8909326\n",
            "SRR8909366\n",
            "SRR8909368\n",
            "SRR8909372\n",
            "SRR8909383\n",
            "SRR8909261\n",
            "SRR8909315\n",
            "SRR8909418\n",
            "SRR8909430\n",
            "SRR8909262\n",
            "SRR8909265\n",
            "SRR8909292\n",
            "SRR8909324\n",
            "SRR8909268\n",
            "SRR8909308\n",
            "SRR8909344\n",
            "SRR8909484\n",
            "SRR8909269\n",
            "SRR8909410\n",
            "SRR8909490\n",
            "SRR8909273\n",
            "SRR8909392\n",
            "SRR8909394\n",
            "SRR8909401\n",
            "SRR8909412\n",
            "SRR8909191\n",
            "SRR8909281\n",
            "SRR8909381\n",
            "SRR8909431\n",
            "SRR8909476\n",
            "SRR8909288\n",
            "SRR8909343\n",
            "SRR1582908\n",
            "SRR1583069\n",
            "SRR1583070\n",
            "SRR1583071\n",
            "SRR1583074\n",
            "SRR1583075\n",
            "SRR1583077\n",
            "SRR1583079\n",
            "SRR5282299\n",
            "SRR5282300\n",
            "SRR5282301\n",
            "SRR5282302\n",
            "SRR5282303\n",
            "SRR5282304\n",
            "SRR5282305\n",
            "SRR5282306\n",
            "SRR5282307\n",
            "SRR5282308\n",
            "SRR5282309\n",
            "SRR5282310\n",
            "SRR5282311\n",
            "SRR5282312\n",
            "SRR8909304\n",
            "SRR8909314\n",
            "SRR8909353\n",
            "SRR8909437\n",
            "SRR8909312\n",
            "SRR8909313\n",
            "SRR8909347\n",
            "SRR8909480\n",
            "SRR8909317\n",
            "SRR8909469\n",
            "SRR8909328\n",
            "SRR8909466\n",
            "SRR8909332\n",
            "SRR8909371\n",
            "SRR8909375\n",
            "SRR8909333\n",
            "SRR8909334\n",
            "SRR8909407\n",
            "SRR8909478\n",
            "SRR8909339\n",
            "SRR8909345\n",
            "SRR8909470\n",
            "SRR8909341\n",
            "SRR8909453\n",
            "SRR8909473\n",
            "SRR8909359\n",
            "SRR8909387\n",
            "SRR8909398\n",
            "SRR8909399\n",
            "SRR8909190\n",
            "SRR8909409\n",
            "SRR8909432\n",
            "SRR8909450\n",
            "SRR8909440\n",
            "SRR8909461\n",
            "SRR8909464\n",
            "SRR8909479\n",
            "SRR9613401\n",
            "SRR9613406\n",
            "SRR9613404\n",
            "SRR9613405\n",
            "SRR8844842\n",
            "SRR8844843\n",
            "SRR8844845\n",
            "SRR8844849\n",
            "SRR8844850\n",
            "SRR8844851\n",
            "SRR8844853\n",
            "SRR8844856\n",
            "SRR8844857\n",
            "SRR2982421\n",
            "SRR2982422\n",
            "SRR2982423\n",
            "SRR2982424\n",
            "SRR2982429\n",
            "SRR2982430\n",
            "SRR2982431\n",
            "SRR2982432\n",
            "SRR2982437\n",
            "SRR2982438\n",
            "SRR2982439\n",
            "SRR2982440\n",
            "SRR2982441\n",
            "SRR2982442\n",
            "SRR2982445\n",
            "SRR2982446\n",
            "SRR2982447\n",
            "SRR2982448\n",
            "SRR2982449\n",
            "SRR2982450\n",
            "SRR2982451\n",
            "SRR8521434\n",
            "SRR8521435\n",
            "SRR8521436\n",
            "SRR8521437\n",
            "SRR8521438\n",
            "SRR8521439\n",
            "SRR8521440\n",
            "SRR8521441\n",
            "SRR8521442\n",
            "SRR8521443\n",
            "SRR8521455\n",
            "SRR8521456\n",
            "SRR8521457\n",
            "SRR8521458\n",
            "SRR8521459\n",
            "SRR8521460\n",
            "SRR8521461\n",
            "SRR8521462\n",
            "ERR2226558\n",
            "ERR2226559\n",
            "ERR2226560\n",
            "ERR2226561\n",
            "ERR2226562\n",
            "ERR2226563\n",
            "ERR2226564\n",
            "ERR2226565\n",
            "ERR2226566\n",
            "ERR2226567\n",
            "ERR2226568\n",
            "ERR2226569\n",
            "ERR2226570\n",
            "ERR2226571\n",
            "ERR2226572\n",
            "ERR2226573\n",
            "ERR2226574\n",
            "ERR2226575\n",
            "ERR2226576\n",
            "ERR2226577\n",
            "ERR2226578\n",
            "ERR2226579\n",
            "ERR2226580\n",
            "ERR2226581\n",
            "ERR2226582\n",
            "ERR2780171\n",
            "ERR2780172\n",
            "ERR2780173\n",
            "ERR2780174\n",
            "ERR2780175\n",
            "ERR2780176\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "qeS-oS-nADap",
        "outputId": "dbc07e84-cbd0-4d02-cfb0-452be3da3ab7"
      },
      "source": [
        "annotofinterest = open(\"/content/annot_list_ecoli.txt\", \"r\").readlines()\n",
        "keywords = []\n",
        "for i in annotofinterest:\n",
        "  keywords.append(i.split(\"\\n\")[0].split(\" NA\")[0])\n",
        "print(keywords)\n",
        "newdf = []\n",
        "all = open(\"all_expannot(ecoli).txt\", \"w\")\n",
        "df = pd.read_csv(\"/content/drive/MyDrive/Biohackathon2021/Escherichia coli_exp_anno.txt\", sep = \"\\t\").astype(str)\n",
        "df2 = df.values.tolist()\n",
        "print(df2)\n",
        "for z in keywords:\n",
        "  for s in df2:\n",
        "    for o in s:\n",
        "      if z != \"nan\":\n",
        "        if z in o or z == o:\n",
        "          if s not in newdf:\n",
        "            newdf.append(s)\n",
        "print(len(newdf))\n",
        "df = pd.DataFrame(newdf)\n",
        "df.to_csv('new_ecoliannot.csv', index=False, header=False)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "['treatment genotype strain', 'phosphomycin K12 C3000 K-12 BW25113', 'teicoplanin resistant strain', 'streptomycin susceptible strain', 'water-7', 'teixobactin', 'doxycycline', 'teixobactin', 'vancomycin', 'teixobactin', 'naoh-6', 'polymyxin', 'monactin', 'linezolid', 'thiostrepton', 'levofloxacin', 'water-8a', 'water-10', 'water-7', 'phosphomycin', 'triclosan', 'meropenem', 'dmso-9', 'ciprofloxacin', 'sulfamethoxazole', 'dmso-7', 'retapamulin', 'water-8b', 'penicillin', 'vancomycin', 'colistin', 'd-cycloserine', 'water-1a', 'fidaxomicin', 'novobiocin', 'teixobactin', 'trimethoprim', 'flavomycin', 'trimethoprim', 'doxycycline', 'meropenem', 'retapamulin', 'levofloxacin', 'norfloxacin', 'tetracycline', 'water-6', 'zeocin', 'penicillin', 'nalidixic-acid', 'meropenem', 'polymyxin', 'sulfamethoxazole', 'teixobactin', 'nalidixic-acid', 'chloramphenicol', 'novobiocin', 'fidaxomicin', 'cerulenin', 'erythromycin', 'retapamulin', 'bacitracin', 'zeocin', 'cefotaxime', 'metronidazole', 'dmso-9', 'monactin', 'ethanol-8', 'dmso-9', 'colistin', 'isoniazid', 'ramoplanin', 'teixobactin', 'triclosan', 'dmso-1', 'tetracycline', 'doxycycline', 'water-1a', 'fidaxomicin', 'zeocin', 'streptomycin', 'monactin', 'phosphomycin', 'tetracycline', 'levofloxacin', 'doxycycline', 'rifampicin', 'd-cycloserine', 'vancomycin', 'chloramphenicol', 'ethanol-8', 'meropenem', 'water-10', 'water-10', 'phosphomycin', 'naoh-6', 'dmso-7', 'cerulenin', 'teixobactin', 'rachelmycin', 'chloramphenicol', 'nalidixic-acid', 'chloramphenicol', 'isoniazid', 'teixobactin', 'actinomycin-d', 'valinomycin', 'rachelmycin', 'cefotaxime', 'novobiocin', 'naoh-2', 'teixobactin', 'novobiocin', 'flavomycin', 'erythromycin', 'water-8a', 'ethanol-8', 'ciprofloxacin', 'bacitracin', 'nalidixic-acid', 'sulfacetamide', 'phosphomycin', 'gentamicin', 'ceftriaxone', 'ceftriaxone', 'trimethoprim', 'ethanol-10', 'trimethoprim', 'doxorubicin', 'rifampicin', 'isoniazid', 'ethanol-10', 'teixobactin', 'valinomycin', 'teixobactin', 'rachelmycin', 'zeocin', 'tetracycline', 'water-1b', 'colistin', 'naoh-6', 'polymyxin', 'bacitracin', 'levofloxacin', 'bacitracin', 'gentamicin', 'bacitracin', 'novobiocin', 'ciprofloxacin', 'gentamicin', 'triclosan', 'polymyxin', 'levofloxacin', 'streptomycin', 'chloramphenicol', 'bacitracin', 'erythromycin', 'vancomycin', 'dmso-2', 'penicillin', 'ciprofloxacin', 'nalidixic-acid', 'dmso-10', 'penicillin', 'dmso-1', 'sulfamethoxazole', 'rifapentine', 'dmso-6', 'phosphomycin', 'dmso-10', 'novobiocin', 'nalidixic-acid', 'teixobactin', 'triclosan', 'ceftriaxone', 'water-7', 'teicoplanin', 'ramoplanin', 'vancomycin', 'dmso-10', 'teixobactin', 'phosphomycin', 'doxycycline', 'linezolid', 'cefotaxime', 'phosphomycin', 'thiostrepton', 'water-6', 'water-8b', 'polymyxin', 'phosphomycin', 'nalidixic-acid', 'cefotaxime', 'streptomycin', 'teicoplanin', 'colistin', 'chloramphenicol', 'rachelmycin', 'dmso-5', 'nalidixic-acid', 'linezolid', 'penicillin', 'teixobactin', 'doxycycline', 'teicoplanin', 'water-1a', 'meropenem', 'doxorubicin', 'teixobactin', 'polymyxin', 'novobiocin', 'flavomycin', 'vancomycin', 'dmso-7', 'naoh-9', 'norfloxacin', 'zeocin', 'water-2', 'monactin', 'sulfacetamide', 'vancomycin', 'naoh-2', 'ethanol-10', 'teixobactin', 'doxorubicin', 'doxycycline', 'water-8a', 'streptomycin', 'd-cycloserine', 'ceftriaxone', 'water-2', 'valinomycin', 'water-8b', 'd-cycloserine', 'novobiocin', 'isoniazid', 'retapamulin', 'thiostrepton', 'teixobactin', 'fidaxomicin', 'ramoplanin', 'teixobactin', 'polymyxin', 'chloramphenicol', 'dmso', 'kirromycin', 'kirromycin', 'dmso', 'Darobactin', 'ampicilin', 'kasugamycin', 'norfloxacin', 'kirromycin', 'chloramphenicol', 'Kasugamycin', 'imipenem', 'meropenem', 'ciprofloxacin']\n",
            "[['ERR3013213', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['ERR3013214', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['SRR7361755', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', '1/2 trans-C adapted', '10.1128/AEM.01616-18'], ['SRR7361761', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1128/AEM.01616-18'], ['SRR7361762', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1128/AEM.01616-18'], ['SRR7361764', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1128/AEM.01616-18'], ['SRR7361765', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', '1/2 trans-C adapted', '10.1128/AEM.01616-18'], ['SRR7361766', 'O157:H7 (ATCC 35150)', 'nan', 'Culture', 'nan', 'nan', '1/2 trans-C adapted', '10.1128/AEM.01616-18'], ['SRR8580014', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1038/s41598-019-51046-7'], ['SRR8580015', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1038/s41598-019-51046-7'], ['SRR8580016', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', 'control', '10.1038/s41598-019-51046-7'], ['SRR8580017', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', '5 hours Wi-Fi exposure', '10.1038/s41598-019-51046-7'], ['SRR8580018', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', '5 hours Wi-Fi exposure', '10.1038/s41598-019-51046-7'], ['SRR8580019', 'K12 DH5{alpha}', 'nan', 'Culture', 'nan', 'nan', '5 hours Wi-Fi exposure', '10.1038/s41598-019-51046-7'], ['ERR2686025', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'logarithmic phase culture (control)', 'nan'], ['ERR2686027', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '2 hours of MazF induction', 'nan'], ['ERR2686029', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '30 minutes recovery from MazF induction', 'nan'], ['ERR2686030', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '30 minutes recovery from MazF induction', 'nan'], ['ERR2686031', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '2 hours of MqsR induction', 'nan'], ['ERR2686032', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '2 hours of MqsR induction', 'nan'], ['ERR2686033', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '30 minutes recovery from MqsR induction', 'nan'], ['ERR2686034', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', '30 minutes recovery from MqsR induction', 'nan'], ['SRR10517474', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517475', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517476', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517477', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517478', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517479', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Optimised culture medium', '10.3389/fgene.2021.633073'], ['SRR10517480', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Evolved strain', '10.3389/fgene.2021.633073'], ['SRR10517481', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Evolved strain', '10.3389/fgene.2021.633073'], ['SRR10517482', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'Evolved strain', '10.3389/fgene.2021.633073'], ['ERR2521823', 'PI-7', 'nan', 'Culture', 'nan', 'nan', 'Phage infection', '10.1021/acs.est.8b04501'], ['ERR2521855', 'PI-7', 'nan', 'Culture', 'nan', 'nan', 'Phage infection', '10.1021/acs.est.8b04501'], ['SRR10018129', 'Sakai', 'O157:H7', 'Homo sapiens', 'nan', 'nan', 'nan', 'nan'], ['SRR10018136', 'Sakai', 'O157:H7', 'Homo sapiens', 'nan', 'nan', 'nan', 'nan'], ['SRR10018138', 'TW16501', 'O26:H11', 'Homo sapiens', 'nan', 'nan', 'nan', 'nan'], ['SRR10022790', 'nan', 'rpsL-K88R\\\\, {delta}tssM vipA-sfGFPclpV-mCherry2', 'nan', 'nan', 'nan', 'T6SS- A. baylyi vs E. coli', '10.1016/j.celrep.2019.09.083'], ['SRR10022792', 'nan', 'rpsL-K88R vipA-sfGFP clpV-mCherry2 {delta}ACIAD0053 {delta}ACIAD1790 {delta}ACIAD3114 {delta}ACIAD3425', 'nan', 'nan', 'nan', 'tae1 expressed A. baylyi vs E. coli', '10.1016/j.celrep.2019.09.083'], ['SRR8909189', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909190', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teicoplanin', '10.1128/AAC.01207-19'], ['SRR8909191', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'streptomycin', '10.1128/AAC.01207-19'], ['SRR8909192', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-7', '10.1128/AAC.01207-19'], ['SRR8909193', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909194', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909195', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909196', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909197', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909198', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-6', '10.1128/AAC.01207-19'], ['SRR8909199', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909200', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'monactin', '10.1128/AAC.01207-19'], ['SRR8909201', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'linezolid', '10.1128/AAC.01207-19'], ['SRR8909202', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'thiostrepton', '10.1128/AAC.01207-19'], ['SRR8909203', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'levofloxacin', '10.1128/AAC.01207-19'], ['SRR8909204', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8a', '10.1128/AAC.01207-19'], ['SRR8909205', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-10', '10.1128/AAC.01207-19'], ['SRR8909206', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-7', '10.1128/AAC.01207-19'], ['SRR8909207', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909208', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'triclosan', '10.1128/AAC.01207-19'], ['SRR8909209', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'meropenem', '10.1128/AAC.01207-19'], ['SRR8909210', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-9', '10.1128/AAC.01207-19'], ['SRR8909211', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ciprofloxacin', '10.1128/AAC.01207-19'], ['SRR8909212', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'sulfamethoxazole', '10.1128/AAC.01207-19'], ['SRR8909215', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-7', '10.1128/AAC.01207-19'], ['SRR8909216', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'retapamulin', '10.1128/AAC.01207-19'], ['SRR8909217', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8b', '10.1128/AAC.01207-19'], ['SRR8909218', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'penicillin', '10.1128/AAC.01207-19'], ['SRR8909219', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909221', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'colistin', '10.1128/AAC.01207-19'], ['SRR8909222', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'd-cycloserine', '10.1128/AAC.01207-19'], ['SRR8909223', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-1a', '10.1128/AAC.01207-19'], ['SRR8909224', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'fidaxomicin', '10.1128/AAC.01207-19'], ['SRR8909225', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909226', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909228', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'trimethoprim', '10.1128/AAC.01207-19'], ['SRR8909229', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'flavomycin', '10.1128/AAC.01207-19'], ['SRR8909230', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'trimethoprim', '10.1128/AAC.01207-19'], ['SRR8909231', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909232', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'meropenem', '10.1128/AAC.01207-19'], ['SRR8909233', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'retapamulin', '10.1128/AAC.01207-19'], ['SRR8909235', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'levofloxacin', '10.1128/AAC.01207-19'], ['SRR8909237', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'norfloxacin', '10.1128/AAC.01207-19'], ['SRR8909238', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'tetracycline', '10.1128/AAC.01207-19'], ['SRR8909239', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-6', '10.1128/AAC.01207-19'], ['SRR8909240', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'zeocin', '10.1128/AAC.01207-19'], ['SRR8909242', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'penicillin', '10.1128/AAC.01207-19'], ['SRR8909243', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909244', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'meropenem', '10.1128/AAC.01207-19'], ['SRR8909245', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909246', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'sulfamethoxazole', '10.1128/AAC.01207-19'], ['SRR8909249', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909250', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909251', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909254', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909255', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'fidaxomicin', '10.1128/AAC.01207-19'], ['SRR8909256', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cerulenin', '10.1128/AAC.01207-19'], ['SRR8909257', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'erythromycin', '10.1128/AAC.01207-19'], ['SRR8909258', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'retapamulin', '10.1128/AAC.01207-19'], ['SRR8909259', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909260', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'zeocin', '10.1128/AAC.01207-19'], ['SRR8909261', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cefotaxime', '10.1128/AAC.01207-19'], ['SRR8909262', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'metronidazole', '10.1128/AAC.01207-19'], ['SRR8909263', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-9', '10.1128/AAC.01207-19'], ['SRR8909264', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'monactin', '10.1128/AAC.01207-19'], ['SRR8909265', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-8', '10.1128/AAC.01207-19'], ['SRR8909266', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-9', '10.1128/AAC.01207-19'], ['SRR8909267', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'colistin', '10.1128/AAC.01207-19'], ['SRR8909268', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'isoniazid', '10.1128/AAC.01207-19'], ['SRR8909269', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ramoplanin', '10.1128/AAC.01207-19'], ['SRR8909270', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909271', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'triclosan', '10.1128/AAC.01207-19'], ['SRR8909273', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-1', '10.1128/AAC.01207-19'], ['SRR8909274', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'tetracycline', '10.1128/AAC.01207-19'], ['SRR8909276', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909277', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-1a', '10.1128/AAC.01207-19'], ['SRR8909278', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'fidaxomicin', '10.1128/AAC.01207-19'], ['SRR8909280', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'zeocin', '10.1128/AAC.01207-19'], ['SRR8909281', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'streptomycin', '10.1128/AAC.01207-19'], ['SRR8909282', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'monactin', '10.1128/AAC.01207-19'], ['SRR8909283', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909284', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'tetracycline', '10.1128/AAC.01207-19'], ['SRR8909285', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'levofloxacin', '10.1128/AAC.01207-19'], ['SRR8909287', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909288', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rifampicin', '10.1128/AAC.01207-19'], ['SRR8909289', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'd-cycloserine', '10.1128/AAC.01207-19'], ['SRR8909290', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909291', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909292', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-8', '10.1128/AAC.01207-19'], ['SRR8909293', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'meropenem', '10.1128/AAC.01207-19'], ['SRR8909295', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-10', '10.1128/AAC.01207-19'], ['SRR8909296', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-10', '10.1128/AAC.01207-19'], ['SRR8909299', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909300', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-6', '10.1128/AAC.01207-19'], ['SRR8909301', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-7', '10.1128/AAC.01207-19'], ['SRR8909302', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cerulenin', '10.1128/AAC.01207-19'], ['SRR8909303', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909304', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rachelmycin', '10.1128/AAC.01207-19'], ['SRR8909305', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909306', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909307', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909308', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'isoniazid', '10.1128/AAC.01207-19'], ['SRR8909310', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909312', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'actinomycin-d', '10.1128/AAC.01207-19'], ['SRR8909313', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'valinomycin', '10.1128/AAC.01207-19'], ['SRR8909314', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rachelmycin', '10.1128/AAC.01207-19'], ['SRR8909315', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cefotaxime', '10.1128/AAC.01207-19'], ['SRR8909316', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909317', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-2', '10.1128/AAC.01207-19'], ['SRR8909318', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909319', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909321', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'flavomycin', '10.1128/AAC.01207-19'], ['SRR8909322', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'erythromycin', '10.1128/AAC.01207-19'], ['SRR8909323', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8a', '10.1128/AAC.01207-19'], ['SRR8909324', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-8', '10.1128/AAC.01207-19'], ['SRR8909325', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ciprofloxacin', '10.1128/AAC.01207-19'], ['SRR8909326', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909327', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909328', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'sulfacetamide', '10.1128/AAC.01207-19'], ['SRR8909331', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909332', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'gentamicin', '10.1128/AAC.01207-19'], ['SRR8909333', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ceftriaxone', '10.1128/AAC.01207-19'], ['SRR8909334', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ceftriaxone', '10.1128/AAC.01207-19'], ['SRR8909337', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'trimethoprim', '10.1128/AAC.01207-19'], ['SRR8909339', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-10', '10.1128/AAC.01207-19'], ['SRR8909340', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'trimethoprim', '10.1128/AAC.01207-19'], ['SRR8909341', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxorubicin', '10.1128/AAC.01207-19'], ['SRR8909343', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rifampicin', '10.1128/AAC.01207-19'], ['SRR8909344', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'isoniazid', '10.1128/AAC.01207-19'], ['SRR8909345', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-10', '10.1128/AAC.01207-19'], ['SRR8909346', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909347', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'valinomycin', '10.1128/AAC.01207-19'], ['SRR8909348', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909353', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rachelmycin', '10.1128/AAC.01207-19'], ['SRR8909355', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'zeocin', '10.1128/AAC.01207-19'], ['SRR8909357', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'tetracycline', '10.1128/AAC.01207-19'], ['SRR8909359', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-1b', '10.1128/AAC.01207-19'], ['SRR8909361', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'colistin', '10.1128/AAC.01207-19'], ['SRR8909362', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-6', '10.1128/AAC.01207-19'], ['SRR8909363', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909366', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909367', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'levofloxacin', '10.1128/AAC.01207-19'], ['SRR8909368', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909371', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'gentamicin', '10.1128/AAC.01207-19'], ['SRR8909372', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909373', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909374', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ciprofloxacin', '10.1128/AAC.01207-19'], ['SRR8909375', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'gentamicin', '10.1128/AAC.01207-19'], ['SRR8909376', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'triclosan', '10.1128/AAC.01207-19'], ['SRR8909378', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909379', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'levofloxacin', '10.1128/AAC.01207-19'], ['SRR8909381', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'streptomycin', '10.1128/AAC.01207-19'], ['SRR8909382', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909383', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'bacitracin', '10.1128/AAC.01207-19'], ['SRR8909384', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'erythromycin', '10.1128/AAC.01207-19'], ['SRR8909386', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909387', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-2', '10.1128/AAC.01207-19'], ['SRR8909389', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'penicillin', '10.1128/AAC.01207-19'], ['SRR8909390', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ciprofloxacin', '10.1128/AAC.01207-19'], ['SRR8909391', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909392', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-10', '10.1128/AAC.01207-19'], ['SRR8909393', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'penicillin', '10.1128/AAC.01207-19'], ['SRR8909394', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-1', '10.1128/AAC.01207-19'], ['SRR8909395', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'sulfamethoxazole', '10.1128/AAC.01207-19'], ['SRR8909398', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rifapentine', '10.1128/AAC.01207-19'], ['SRR8909399', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-6', '10.1128/AAC.01207-19'], ['SRR8909400', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909401', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-10', '10.1128/AAC.01207-19'], ['SRR8909402', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909403', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909404', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909406', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'triclosan', '10.1128/AAC.01207-19'], ['SRR8909407', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ceftriaxone', '10.1128/AAC.01207-19'], ['SRR8909408', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-7', '10.1128/AAC.01207-19'], ['SRR8909409', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teicoplanin', '10.1128/AAC.01207-19'], ['SRR8909410', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ramoplanin', '10.1128/AAC.01207-19'], ['SRR8909411', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909412', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-10', '10.1128/AAC.01207-19'], ['SRR8909413', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909414', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909415', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909417', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'linezolid', '10.1128/AAC.01207-19'], ['SRR8909418', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cefotaxime', '10.1128/AAC.01207-19'], ['SRR8909419', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909420', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'thiostrepton', '10.1128/AAC.01207-19'], ['SRR8909422', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-6', '10.1128/AAC.01207-19'], ['SRR8909424', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8b', '10.1128/AAC.01207-19'], ['SRR8909425', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909428', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'phosphomycin', '10.1128/AAC.01207-19'], ['SRR8909429', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909430', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'cefotaxime', '10.1128/AAC.01207-19'], ['SRR8909431', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'streptomycin', '10.1128/AAC.01207-19'], ['SRR8909432', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teicoplanin', '10.1128/AAC.01207-19'], ['SRR8909433', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'colistin', '10.1128/AAC.01207-19'], ['SRR8909434', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR8909437', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'rachelmycin', '10.1128/AAC.01207-19'], ['SRR8909440', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-5', '10.1128/AAC.01207-19'], ['SRR8909441', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'nalidixic-acid', '10.1128/AAC.01207-19'], ['SRR8909444', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'linezolid', '10.1128/AAC.01207-19'], ['SRR8909445', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'penicillin', '10.1128/AAC.01207-19'], ['SRR8909447', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909448', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909450', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teicoplanin', '10.1128/AAC.01207-19'], ['SRR8909451', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-1a', '10.1128/AAC.01207-19'], ['SRR8909452', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'meropenem', '10.1128/AAC.01207-19'], ['SRR8909453', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxorubicin', '10.1128/AAC.01207-19'], ['SRR8909454', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909455', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909456', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909458', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'flavomycin', '10.1128/AAC.01207-19'], ['SRR8909459', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909460', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso-7', '10.1128/AAC.01207-19'], ['SRR8909461', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-9', '10.1128/AAC.01207-19'], ['SRR8909462', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'norfloxacin', '10.1128/AAC.01207-19'], ['SRR8909463', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'zeocin', '10.1128/AAC.01207-19'], ['SRR8909464', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-2', '10.1128/AAC.01207-19'], ['SRR8909465', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'monactin', '10.1128/AAC.01207-19'], ['SRR8909466', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'sulfacetamide', '10.1128/AAC.01207-19'], ['SRR8909467', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'vancomycin', '10.1128/AAC.01207-19'], ['SRR8909469', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'naoh-2', '10.1128/AAC.01207-19'], ['SRR8909470', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ethanol-10', '10.1128/AAC.01207-19'], ['SRR8909472', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909473', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxorubicin', '10.1128/AAC.01207-19'], ['SRR8909474', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'doxycycline', '10.1128/AAC.01207-19'], ['SRR8909475', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8a', '10.1128/AAC.01207-19'], ['SRR8909476', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'streptomycin', '10.1128/AAC.01207-19'], ['SRR8909477', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'd-cycloserine', '10.1128/AAC.01207-19'], ['SRR8909478', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ceftriaxone', '10.1128/AAC.01207-19'], ['SRR8909479', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-2', '10.1128/AAC.01207-19'], ['SRR8909480', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'valinomycin', '10.1128/AAC.01207-19'], ['SRR8909481', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'water-8b', '10.1128/AAC.01207-19'], ['SRR8909482', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'd-cycloserine', '10.1128/AAC.01207-19'], ['SRR8909483', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'novobiocin', '10.1128/AAC.01207-19'], ['SRR8909484', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'isoniazid', '10.1128/AAC.01207-19'], ['SRR8909485', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'retapamulin', '10.1128/AAC.01207-19'], ['SRR8909487', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'thiostrepton', '10.1128/AAC.01207-19'], ['SRR8909488', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909489', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'fidaxomicin', '10.1128/AAC.01207-19'], ['SRR8909490', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'ramoplanin', '10.1128/AAC.01207-19'], ['SRR8909492', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'teixobactin', '10.1128/AAC.01207-19'], ['SRR8909493', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'polymyxin', '10.1128/AAC.01207-19'], ['SRR8909494', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1128/AAC.01207-19'], ['SRR9163184', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163185', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163186', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163187', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163188', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163189', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't01', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163190', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't02', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163191', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't02', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163192', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't03', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163193', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't03', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163194', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't04', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163196', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't04', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163198', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't04', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163199', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't05', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163200', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't05', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163201', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't05', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163203', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't06', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163204', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't06', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163205', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't07', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163206', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't07', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163207', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't07', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163208', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't08', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163209', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't08', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163210', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't08', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163211', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't09', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163212', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't09', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163213', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't09', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163214', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't10', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163215', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't10', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163216', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't10', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163217', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't11', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163218', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't11', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163219', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't12', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163220', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't12', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163221', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't13', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163222', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't13', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163223', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't13', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163224', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't14', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163225', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't14', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163226', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't15', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163227', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't15', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163228', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't15', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163229', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't16', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163230', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't16', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163231', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't17', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163232', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't17', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163235', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't18', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163237', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't18', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163238', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't19', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163239', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't19', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163240', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't20', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163241', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't21', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163242', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't21', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163243', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't22', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163244', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't22', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163245', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't23', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163246', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't23', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163247', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't23', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163248', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't24', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163249', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't24', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163250', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't24', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163251', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't24', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163252', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't25', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163253', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't26', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163254', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't26', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163255', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't27', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163256', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't27', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163257', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't28', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163258', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't28', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163259', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't28', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163260', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't28', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163261', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't29', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163262', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't29', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163263', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't29', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9163264', 'K-12 BW25113', 'nan', 'Culture', 'nan', 't29', 'nan', '10.1038/s41467-019-12474-1'], ['SRR9204648', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR9204649', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR9204650', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose and 1 mM cytidine', '10.1038/s41467-019-13483-w'], ['SRR9204651', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose and 1 mM cytidine', '10.1038/s41467-019-13483-w'], ['SRR9204652', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose and 100 mg/L adenine', '10.1038/s41467-019-13483-w'], ['SRR9204653', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose and 100 mg/L adenine', '10.1038/s41467-019-13483-w'], ['SRR8242101', 'K-12 MG1655', 'WT', 'Culture', 'nan', '3 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242102', 'K-12 MG1655', 'ryeA -10 mutation; ryeA promoter mutant', 'Culture', 'nan', '3 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242103', 'K-12 MG1655', 'WT', 'Culture', 'nan', '8 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242104', 'K-12 MG1655', 'ryeA -10 mutation; ryeA promoter mutant', 'Culture', 'nan', '8 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242105', 'K-12 MG1655', 'WT', 'Culture', 'nan', '3 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242106', 'K-12 MG1655', 'sdsR -10 mutation; sdsR promoter mutant', 'Culture', 'nan', '3 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242107', 'K-12 MG1655', 'WT', 'Culture', 'nan', '8 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8242108', 'K-12 MG1655', 'sdsR -10 mutation; sdsR promoter mutant', 'Culture', 'nan', '8 h post-inoculation', 'nan', '10.1038/s41598-019-45998-z'], ['SRR8867849', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867850', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867852', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867853', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867854', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867855', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867856', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867857', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867858', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867860', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867861', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867865', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867866', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867868', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867871', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867872', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867873', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867874', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867875', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867876', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867877', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867878', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867879', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867880', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867881', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867882', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867884', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867885', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867887', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867888', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867889', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867890', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867893', 'O157', 'O157:H7 Sakai', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867897', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867898', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8867906', 'O157', 'O157:H_ EC2422', 'Meat', 'nan', 'nan', 'nan', '10.1016/j.fm.2019.03.008'], ['SRR8844840', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844841', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844842', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844843', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844844', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844845', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844846', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844847', 'K-12 BW25113', 'nan', 'Culture', 'nan', '15min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844848', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844849', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844850', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844851', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844852', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844853', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844854', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844855', 'K-12 BW25113', 'nan', 'Culture', 'nan', '1h', 'Solvent (Control)', '10.1038/s41586-019-1791-1'], ['SRR8844856', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8844857', 'K-12 BW25113', 'nan', 'Culture', 'nan', '30min', 'Darobactin', '10.1038/s41586-019-1791-1'], ['SRR8282453', 'Rosetta2 DE3', 'nan', 'Culture', 'nan', 'nan', 'LB broth Kanamycin Chloramphenicol 0.4 mM ZnSO4', '10.1038/s42003-019-0328-3'], ['SRR9096026', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.molcel.2019.10.022'], ['SRR10259334', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', '5FU', '10.1101/820084'], ['SRR10259335', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'LB', '10.1101/820084'], ['SRR10259336', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'LB', '10.1101/820084'], ['SRR10259337', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'LB', '10.1101/820084'], ['SRR10259338', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'CAP', '10.1101/820084'], ['SRR10259339', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'CAP', '10.1101/820084'], ['SRR10259340', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', 'CAP', '10.1101/820084'], ['SRR10259341', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', '5FU', '10.1101/820084'], ['SRR10259342', 'K-12 MG1655', 'nan', 'Homo sapiens', 'nan', 'nan', '5FU', '10.1101/820084'], ['SRR8130752', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Untreated', '10.1093/nar/gkz736'], ['SRR8130753', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Untreated', '10.1093/nar/gkz736'], ['SRR8130754', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Untreated', '10.1093/nar/gkz736'], ['SRR8130755', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'NaBH4 treated', '10.1093/nar/gkz736'], ['SRR8130756', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'NaBH4 treated', '10.1093/nar/gkz736'], ['SRR8130757', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'NaBH4 treated', '10.1093/nar/gkz736'], ['SRR7261718', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261720', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261722', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261724', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261726', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261728', 'K-12 BW25113', 'Diesel exhaust atmosphere-adapted', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['ERR366427', '910/00', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366428', '910/00', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366429', '910/00', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366430', '910/00', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366431', '910/00', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366432', '910/00', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366433', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366434', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366435', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366436', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366437', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366438', 'Mu/07/0324/', 'nan', 'nan', 'nan', 'nan', 'Infected with phi3538_delta[stx2]', 'nan'], ['ERR366440', 'IMT8073', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366441', 'IMT8073', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366442', 'IMT8073', 'delta[yjjQ]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366443', 'IMT8073', 'delta[yjjQ]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR366444', 'IMT8073', 'delta[yjjQ]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862917', 'ETEC E36', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862919', 'ETEC E616', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862921', 'ETEC E806', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862923', 'ETEC E1057', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862925', 'ETEC E1189', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862927', 'ETEC E1441', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862929', 'ETEC E1556', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862931', 'ETEC E1600', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862933', 'ETEC E1649', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862934', 'ETEC E1679', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862936', 'ETEC E1752', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862938', 'ETEC E2980', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR1910358', 'nan', 'WT DHFR', 'Culture', 'nan', 'nan', 'No treatment', '10.1016/j.celrep.2015.03.051'], ['SRR1910359', 'nan', 'WT DHFR', 'Culture', 'nan', 'nan', 'No treatment', '10.1016/j.celrep.2015.03.051'], ['SRR1910360', 'nan', 'DHFR mutant M133V', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910361', 'nan', 'DHFR mutant M133V', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910362', 'nan', 'DHFR mutant V75H+I155A', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910363', 'nan', 'DHFR mutant V75H+I155A', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910364', 'nan', 'DHFR mutant I91L+W133V', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910365', 'nan', 'DHFR mutant I91L+W133V', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910375', 'nan', 'DHFR mutant V75H+I91L+I155A', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910377', 'nan', 'DHFR mutant V75H+I91L+I155A', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.celrep.2015.03.051'], ['SRR1910381', 'nan', 'WT DHFR', 'Culture', 'nan', 'nan', 'TMP 0.5 ug/mL treatment', '10.1016/j.celrep.2015.03.051'], ['SRR1910383', 'nan', 'WT DHFR', 'Culture', 'nan', 'nan', 'TMP 0.5 ug/mL treatment', '10.1016/j.celrep.2015.03.051'], ['ERR862918', 'ETEC E562', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862920', 'ETEC E626', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862922', 'ETEC E925', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862924', 'ETEC E1167', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862926', 'ETEC E1373', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862928', 'ETEC E1533', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862930', 'ETEC E1574', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862935', 'ETEC E1739', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR862937', 'ETEC E1779', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3194456', 'Crooks', 'nan', 'Culture', 'nan', 'nan', 'Anaerobic env', '10.1016/j.cels.2016.08.013'], ['SRR1124840', 'K-12 JM83', 'nan', 'Culture', 'nan', 'nan', 'Untreated E. coli cells', '10.1016/j.cell.2014.01.068'], ['SRR1124841', 'K-12 JM83', 'nan', 'Culture', 'nan', 'nan', 'Untreated E. coli cells', '10.1016/j.cell.2014.01.068'], ['SRR1124843', 'K-12 JM83', 'nan', 'Culture', 'nan', 'nan', 'Trimethoprim-treated E. coli cells', '10.1016/j.cell.2014.01.068'], ['SRR441582', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR441587', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR2135666', 'K-12 W3110', 'nan', 'Culture', 'nan', '2 min', 'nan', '10.3390/genes8030090'], ['SRR2135676', 'K-12 W3110', 'nan', 'Culture', 'nan', '0.5 min', 'nan', '10.3390/genes8030090'], ['SRR2135677', 'K-12 W3110', 'nan', 'Culture', 'nan', '1 min', 'nan', '10.3390/genes8030090'], ['SRR2135678', 'K-12 W3110', 'nan', 'Culture', 'nan', '2 min', 'nan', '10.3390/genes8030090'], ['SRR3171772', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'No pressure sample (media only)', '10.1128/mSphere.00009-17'], ['SRR3171774', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to ampicillin', '10.1128/mSphere.00009-17'], ['SRR3171776', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to tetracycline', '10.1128/mSphere.00009-17'], ['SRR3171777', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to tetracycline', '10.1128/mSphere.00009-17'], ['SRR3171780', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to n-butanol', '10.1128/mSphere.00009-17'], ['SRR3194455', 'Crooks', 'nan', 'Culture', 'nan', 'nan', 'Anaerobic env', '10.1016/j.cels.2016.08.013'], ['SRR2982421', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982422', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982423', 'K-12 MG1655', 'WT', 'Culture', 'nan', '4 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982424', 'K-12 MG1655', 'WT', 'Culture', 'nan', '6 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982429', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982430', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982431', 'K-12 MG1655', 'WT', 'Culture', 'nan', '4 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982432', 'K-12 MG1655', 'WT', 'Culture', 'nan', '6 min', 'Without kasugamycin', '10.7554/eLife.13065'], ['SRR2982437', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982438', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982439', 'K-12 MG1655', 'WT', 'Culture', 'nan', '4 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982440', 'K-12 MG1655', 'WT', 'Culture', 'nan', '6 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982441', 'K-12 MG1655', 'WT', 'Culture', 'nan', '8 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982442', 'K-12 MG1655', 'WT', 'Culture', 'nan', '10 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982445', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982446', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982447', 'K-12 MG1655', 'WT', 'Culture', 'nan', '4 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982448', 'K-12 MG1655', 'WT', 'Culture', 'nan', '6 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982449', 'K-12 MG1655', 'WT', 'Culture', 'nan', '8 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982450', 'K-12 MG1655', 'WT', 'Culture', 'nan', '10 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982451', 'K-12 MG1655', 'WT', 'Culture', 'nan', '15 min', 'With kasugamycin', '10.7554/eLife.13065'], ['SRR2982453', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '0 min', 'nan', '10.7554/eLife.13065'], ['SRR2982454', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '2 min', 'nan', '10.7554/eLife.13065'], ['SRR2982455', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '4 min', 'nan', '10.7554/eLife.13065'], ['SRR2982456', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '6 min', 'nan', '10.7554/eLife.13065'], ['SRR2982457', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '8 min', 'nan', '10.7554/eLife.13065'], ['SRR2982458', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '10 min', 'nan', '10.7554/eLife.13065'], ['SRR2982461', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '0 min', 'nan', '10.7554/eLife.13065'], ['SRR2982462', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '2 min', 'nan', '10.7554/eLife.13065'], ['SRR2982463', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '4 min', 'nan', '10.7554/eLife.13065'], ['SRR2982464', 'K-12 MG1655', 'delta[A] mutant', 'Culture', 'nan', '6 min', 'nan', '10.7554/eLife.13065'], ['SRR2914536', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '50ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914537', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '50ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914540', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '62.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914541', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '62.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914542', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914543', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914544', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914545', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914546', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914547', 'K-12 MG1655', 'delta[mutS]', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914563', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '62.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914564', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '62.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914565', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '62.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914566', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914567', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914568', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '75ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914569', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914570', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR2914571', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '87.5ng/ml norfloxacin', '10.1073/pnas.1601208113'], ['SRR1927169', 'K-12 MG1655', 'nan', 'nan', '37C', 'nan', 'LB', 'nan'], ['SRR1927170', 'K-12 MG1655', 'nan', 'nan', '37C', 'nan', 'LB', 'nan'], ['SRR2864016', 'Avian Pathogenic', 'nan', 'duck', 'nan', 'nan', 'nan', '10.1038/srep25085'], ['SRR5115645', 'RP437-L2', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115646', 'RP437-R4', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115647', 'RP437-L7', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115648', 'RP437-R6', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115650', 'RP437-L4', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115651', 'RP437-L5', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115653', 'RP437-L6', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115655', 'RP437-L3', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115657', 'RP437-R2', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115658', 'RP437-R5', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115659', 'RP437-R1', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115661', 'RP437-R3', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115662', 'RP437-L1', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR5115664', 'RP437', 'nan', 'warm-blooded organisms', 'nan', 'nan', 'nan', '10.1016/j.celrep.2016.12.088'], ['SRR4047463', 'K-12 BW25113', 'WT (Keio mutant)', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047464', 'K-12 BW25113', 'delta[zwf]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047465', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047466', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047467', 'K-12 BW25113', 'delta[ppk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047468', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047469', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047470', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047471', 'K-12 BW25113', 'delta[dapF]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047472', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047473', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047474', 'K-12 BW25113', 'delta[entC]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047475', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047476', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047477', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047478', 'K-12 BW25113', 'delta[dgk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047479', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047480', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047481', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047482', 'K-12 BW25113', 'WT (Keio mutant)', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047483', 'K-12 BW25113', 'delta[zwf]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047484', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047485', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047486', 'K-12 BW25113', 'delta[ppk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047487', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047488', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047489', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047490', 'K-12 BW25113', 'delta[dapF]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047491', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047492', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047493', 'K-12 BW25113', 'delta[entC]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047494', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047495', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047496', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047497', 'K-12 BW25113', 'delta[dgk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047498', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047499', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047500', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047501', 'K-12 BW25113', 'WT (Keio mutant)', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047502', 'K-12 BW25113', 'delta[zwf]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047503', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047504', 'K-12 BW25113', 'zwf suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047505', 'K-12 BW25113', 'delta[ppk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047506', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047507', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047508', 'K-12 BW25113', 'ppk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047509', 'K-12 BW25113', 'delta[dapF]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047510', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047511', 'K-12 BW25113', 'dapF suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047512', 'K-12 BW25113', 'delta[entC]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047513', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047514', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047515', 'K-12 BW25113', 'entC suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047516', 'K-12 BW25113', 'delta[dgk]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047517', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047518', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR4047519', 'K-12 BW25113', 'dgk suppressor', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1007284'], ['SRR5121109', 'K-12 MG1655', 'ORF1', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121110', 'K-12 MG1655', 'ORF1', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121111', 'K-12 MG1655', 'ORF1', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121112', 'K-12 MG1655', 'ORF1', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121113', 'K-12 MG1655', 'Svi3-3', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121114', 'K-12 MG1655', 'Svi3-3', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121115', 'K-12 MG1655', 'Svi3-3', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121116', 'K-12 MG1655', 'Svi3-3', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121117', 'K-12 MG1655', 'pCA24N, -gfp', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121118', 'K-12 MG1655', 'pCA24N, -gfp', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41559-018-0568-5'], ['SRR5121784', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121785', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121786', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121787', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121788', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121789', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121790', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121791', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121792', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121793', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121794', 'K-12', 'H9-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121795', 'K-12', 'H9-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121796', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121797', 'K-12', 'H9-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121798', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121799', 'K-12', 'H9-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121800', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121858', 'K-12', 'F11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121859', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121860', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121861', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121862', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121863', 'K-12', 'H9-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121864', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121865', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121866', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121868', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121869', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121870', 'K-12', 'F9-2', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121871', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121872', 'K-12', 'B11-1', '96-well Microtiter plate', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['SRR5121873', 'K-12', 'D8', 'Stock Culture', 'nan', 'nan', 'nan', '10.1128/AEM.00442-17'], ['ERR519409', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR588854', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR588856', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR1613487', 'O157:H7 NCTC 12900', 'nan', 'Culture', 'nan', 'nan', 'None', '10.1128/mBio.00961-16'], ['SRR1613488', 'O157:H7 NCTC 12900', 'nan', 'Culture', 'nan', 'nan', 'Treated by HPCD at 5 MPa and 25C for 40 min', '10.1128/mBio.00961-16'], ['SRR3948601', 'Crooks (ATCC 8739)', 'NZ-502 (RpoB mutant)', 'Culture', 'nan', 'nan', 'normal condtion (5% w/v glucose)', '10.1186/s12896-017-0337-6'], ['SRR3948602', 'Crooks (ATCC 8739)', 'NZ-502 (RpoB mutant)', 'Culture', 'nan', 'nan', 'osmotic stress condtion (12% w/v glucose)', '10.1186/s12896-017-0337-6'], ['SRR3948603', 'Crooks (ATCC 8739)', 'Suc-T110 (parental strain)', 'Culture', 'nan', 'nan', 'normal condtion (5% w/v glucose)', '10.1186/s12896-017-0337-6'], ['SRR3948604', 'Crooks (ATCC 8739)', 'Suc-T110 (parental strain)', 'Culture', 'nan', 'nan', 'osmotic stress condtion (12% w/v glucose)', '10.1186/s12896-017-0337-6'], ['SRR5143858', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '0min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR5143860', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '5min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR5143862', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '5min', '0.5 µg/ml Carolacton', '10.1128/mSphereDirect.00375-17'], ['SRR5143863', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '15min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR5143864', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '15min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR5143865', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '15min', '0.5 µg/ml Carolacton', '10.1128/mSphereDirect.00375-17'], ['SRR5143866', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '15min', '0.5 µg/ml Carolacton', '10.1128/mSphereDirect.00375-17'], ['SRR5143868', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '30min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR5143869', 'nan', 'TolC defective (TolC-)', 'planktonic cells', 'nan', '30min', 'none', '10.1128/mSphereDirect.00375-17'], ['SRR8756989', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0min', 'nan', '10.1128/mBio.01330-19'], ['SRR8756990', 'K-12 MG1655', 'WT', 'Culture', 'nan', '15min', 'nan', '10.1128/mBio.01330-19'], ['SRR8756991', 'K-12 MG1655', 'RNAP 1_2_', 'Culture', 'nan', '0min', 'nan', '10.1128/mBio.01330-19'], ['SRR8756992', 'K-12 MG1655', 'RNAP 1_2_', 'Culture', 'nan', '15min', 'nan', '10.1128/mBio.01330-19'], ['SRR8757041', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757042', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757043', 'K-12 MG1655', 'pair8', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757044', 'K-12 MG1655', 'pair14', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757045', 'K-12 MG1655', 'pair15', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757046', 'K-12 MG1655', 'pair13', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757047', 'K-12 MG1655', 'pair1', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757048', 'K-12 MG1655', 'pair4', 'Culture', 'nan', 'nan', '10 _M MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757049', 'K-12 MG1655', 'PhoQwt-PhoP4', 'Culture', 'nan', 'nan', '0 _M tZ', '10.1038/s41586-019-1639-8'], ['SRR8757050', 'K-12 MG1655', 'AQ4-PhoP4', 'Culture', 'nan', 'nan', '0 _M tZ', '10.1038/s41586-019-1639-8'], ['SRR8757051', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757052', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757053', 'K-12 MG1655', 'pair8', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757054', 'K-12 MG1655', 'pair14', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757055', 'K-12 MG1655', 'pair15', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757056', 'K-12 MG1655', 'pair13', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757057', 'K-12 MG1655', 'pair1', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757058', 'K-12 MG1655', 'pair4', 'Culture', 'nan', 'nan', '50 mM MgSO4', '10.1038/s41586-019-1639-8'], ['SRR8757059', 'K-12 MG1655', 'PhoQwt-PhoP4', 'Culture', 'nan', 'nan', '1 _M tZ', '10.1038/s41586-019-1639-8'], ['SRR8757060', 'K-12 MG1655', 'AQ4-PhoP4', 'Culture', 'nan', 'nan', '1 _M tZ', '10.1038/s41586-019-1639-8'], ['DRR089610', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089611', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089612', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089613', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089614', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089615', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089616', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089617', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089618', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089619', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089620', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089621', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089622', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089623', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089625', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089626', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089627', 'MY205', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089628', 'KT101', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['DRR089629', 'KT101', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR2287635', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'N-', '10.1099/mic.0.000683'], ['ERR2287636', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'N-', '10.1099/mic.0.000683'], ['ERR2287638', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'N+', '10.1099/mic.0.000683'], ['ERR2287640', 'K-12 BW25113', 'delta[yeaG]', 'Culture', 'nan', 'nan', 'N-24', '10.1099/mic.0.000683'], ['ERR2287642', 'K-12 BW25113', 'delta[yeaG]', 'Culture', 'nan', 'nan', 'N-', '10.1099/mic.0.000683'], ['ERR2287643', 'K-12 BW25113', 'delta[yeaG]', 'Culture', 'nan', 'nan', 'N-', '10.1099/mic.0.000683'], ['ERR2287645', 'K-12 BW25113', 'delta[yeaG]', 'Culture', 'nan', 'nan', 'N+', '10.1099/mic.0.000683'], ['SRR7454724', 'CBJ1041', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454725', 'CBJ1055', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454726', 'CBJ1041', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454727', 'CBJ1041', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454728', 'CBJ1055', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454729', 'CBJ1055', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-12022-x'], ['SRR7454850', 'K-12 MG1655', 'containing an IPTG-inducible pHM4T RNA expression vector', 'Culture', 'nan', 'nan', 'nan', '10.1080/15476286.2018.1532252'], ['SRR7454851', 'K-12 MG1655', 'containing a pHM4T derivative carrying the full sdsR sequence', 'Culture', 'nan', 'nan', 'nan', '10.1080/15476286.2018.1532252'], ['SRR7454852', 'K-12 MG1655', 'containing a pHM4T derivative carrying the 45-103 sequence of sdsR', 'Culture', 'nan', 'nan', 'nan', '10.1080/15476286.2018.1532252'], ['SRR7454859', 'ATCC 8739', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1534/genetics.118.301345'], ['SRR8100576', 'APEC O2-211', 'nan', 'Pigeon', 'nan', 'nan', 'nan', '10.3382/ps/pez205'], ['SRR8100577', 'APEC 40', 'nan', 'Pigeon', 'nan', 'nan', 'nan', '10.3382/ps/pez205'], ['SRR8179881', 'SJ_XTL219', 'ELSA-Stress control', 'Culture', 'nan', 'nan', 'grown in induced conditions', '10.1038/s41587-019-0286-9'], ['SRR8179882', 'SJ_XTL219', 'ELSA-Stress control', 'Culture', 'nan', 'nan', 'grown in induced conditions', '10.1038/s41587-019-0286-9'], ['SRR8179883', 'SJ_XTL219', 'ELSA-Stress', 'Culture', 'nan', 'nan', 'grown in induced conditions', '10.1038/s41587-019-0286-9'], ['SRR8179884', 'SJ_XTL219', 'ELSA-Stress', 'Culture', 'nan', 'nan', 'grown in induced conditions', '10.1038/s41587-019-0286-9'], ['SRR8179885', 'SJ_XTL219', 'ELSA-MultiAux control', 'Culture', 'nan', 'nan', 'grown in induced conditions with supplemented amino acids', '10.1038/s41587-019-0286-9'], ['SRR8179886', 'SJ_XTL219', 'ELSA-MultiAux control', 'Culture', 'nan', 'nan', 'grown in induced conditions with supplemented amino acids', '10.1038/s41587-019-0286-9'], ['SRR8179887', 'SJ_XTL219', 'ELSA-MultiAux', 'Culture', 'nan', 'nan', 'grown in induced conditions with supplemented amino acids', '10.1038/s41587-019-0286-9'], ['SRR8179888', 'SJ_XTL219', 'ELSA-MultiAux', 'Culture', 'nan', 'nan', 'grown in induced conditions with supplemented amino acids', '10.1038/s41587-019-0286-9'], ['SRR8180413', 'K-12', 'WT with delta[relA]', 'Culture', 'nan', '5min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180416', 'K-12', 'WT with delta[relA]', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180418', 'K-12', 'WT with delta[relA]', 'Culture', 'nan', '10min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180422', 'K-12', 'RNAP 1-2- mutant with delta[relA]', 'Culture', 'nan', '5min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180423', 'K-12', 'RNAP 1-2- mutant with delta[relA]', 'Culture', 'nan', '10min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180424', 'K-12', 'WT with relA', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180426', 'K-12', 'WT with relA', 'Culture', 'nan', '5min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180433', 'K-12', 'RNAP 1-2- mutant with relA', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180434', 'K-12', 'RNAP 1-2- mutant with relA', 'Culture', 'nan', '5min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180437', 'K-12', 'RNAP 1-2- mutant with relA', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180439', 'K-12', 'RNAP 1-2- mutant with relA', 'Culture', 'nan', '10min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180440', 'K-12', 'RNAP 1-2- mutant with relA', 'Culture', 'nan', '5min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180442', 'K-12', 'WT with delta[relA]', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8180443', 'K-12', 'WT with delta[relA]', 'Culture', 'nan', '0min', 'nan', '10.1073/pnas.1819682116'], ['SRR8164476', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose w/o trace elements', '10.1038/s41467-019-13483-w'], ['SRR8164477', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose w/o trace elements', '10.1038/s41467-019-13483-w'], ['SRR8164478', 'K-12 BW25113', 'delta[cbl]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164479', 'K-12 BW25113', 'delta[cbl]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164480', 'K-12 BW25113', 'delta[fabR]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164481', 'K-12 BW25113', 'delta[fabR]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164482', 'K-12 BW25113', 'delta[fabR]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164483', 'K-12 BW25113', 'delta[fabR]', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose', '10.1038/s41467-019-13483-w'], ['SRR8164484', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 0.2% glucose and 10mM glutamate', '10.1038/s41467-019-13483-w'], ['SRR9919224', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919225', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919226', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919227', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919228', 'K-12 MG1655', 'GMOS', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919229', 'K-12 MG1655', 'GMOS', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919230', 'K-12 MG1655', 'GMOS', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919231', 'K-12 MG1655', 'GMOS', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919232', 'K-12 MG1655', 'EC ALE-1', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919233', 'K-12 MG1655', 'EC ALE-1', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919234', 'K-12 MG1655', 'EC ALE-1', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919235', 'K-12 MG1655', 'EC ALE-1', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919236', 'K-12 MG1655', 'EC ALE-2', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919237', 'K-12 MG1655', 'EC ALE-2', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919238', 'K-12 MG1655', 'EC ALE-2', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919239', 'K-12 MG1655', 'EC ALE-2', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919240', 'K-12 MG1655', 'EC ALE-3', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919241', 'K-12 MG1655', 'EC ALE-3', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919242', 'K-12 MG1655', 'EC ALE-3', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919243', 'K-12 MG1655', 'EC ALE-3', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919244', 'K-12 MG1655', 'EC ALE-4', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919245', 'K-12 MG1655', 'EC ALE-4', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose', '10.1093/molbev/msz251'], ['SRR9919246', 'K-12 MG1655', 'EC ALE-4', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR9919247', 'K-12 MG1655', 'EC ALE-4', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucose + 2 mM Hydrogen peroxide', '10.1093/molbev/msz251'], ['SRR8173218', 'K-12 W3110', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173227', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173228', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173229', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173230', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173231', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-sorbitol', '10.1038/s41467-019-13483-w'], ['SRR8173232', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-sorbitol', '10.1038/s41467-019-13483-w'], ['SRR8173233', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-ribose', '10.1038/s41467-019-13483-w'], ['SRR8173234', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-ribose', '10.1038/s41467-019-13483-w'], ['SRR8173235', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173236', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173238', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L glucarate', '10.1038/s41467-019-13483-w'], ['SRR8173239', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173240', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173241', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'anaerobic M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173242', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'anaerobic M9 minimal media w/ 2g/L glucose', '10.1038/s41467-019-13483-w'], ['SRR8173243', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L N-acetylglucosamine', '10.1038/s41467-019-13483-w'], ['SRR8173244', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L N-acetylglucosamine', '10.1038/s41467-019-13483-w'], ['SRR8173245', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L galactose', '10.1038/s41467-019-13483-w'], ['SRR8173246', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L galactose', '10.1038/s41467-019-13483-w'], ['SRR8173247', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-gluconate', '10.1038/s41467-019-13483-w'], ['SRR8173248', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 4g/L D-gluconate', '10.1038/s41467-019-13483-w'], ['SRR8173249', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 3.3g/L pyruvate', '10.1038/s41467-019-13483-w'], ['SRR8173250', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'M9 minimal media w/ 3.3g/L pyruvate', '10.1038/s41467-019-13483-w'], ['ERR2534339', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534340', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534341', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534342', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2534343', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2534344', 'K-12 MG1655', 'drnc14::Tn10', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2534345', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534346', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534347', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'logarithmic phase culture', '10.1093/nar/gky684'], ['ERR2534348', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2534349', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2534350', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'stationary phase culture', '10.1093/nar/gky684'], ['ERR2097558', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['ERR2097559', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['ERR2097560', 'K-12 MG1655', 'eMS57', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['ERR2097561', 'K-12 MG1655', 'eMS57', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-019-08888-6'], ['SRR7749991', 'AE17', 'nan', 'Duck', 'nan', 'nan', 'nan', 'nan'], ['SRR7749992', 'KdpDE', 'nan', 'Duck', 'nan', 'nan', 'nan', 'nan'], ['SRR7750422', 'J-KAFS', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7750486', 'pGETS118', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7750487', 'J-KAFS', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280079', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280080', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280081', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280082', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280083', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280084', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280085', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280086', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280087', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280088', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280089', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7280090', 'BL21(DE3)', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR8607533', 'K-12 W3110', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8607534', 'K-12 W3110', 'delta[mgrB]', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8607535', 'K-12 W3110', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8607536', 'K-12 W3110', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8607537', 'K-12 W3110', 'delta[mgrB]', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8607538', 'K-12 W3110', 'delta[mgrB]', 'Culture', 'nan', 'nan', 'nan', '10.1093/femsle/fnz123'], ['SRR8587784', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'naive (wild type)', '10.1186/s12864-019-5892-x'], ['SRR8587785', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'naive (wild type)', '10.1186/s12864-019-5892-x'], ['SRR8587786', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'naive (wild type)', '10.1186/s12864-019-5892-x'], ['SRR8587787', 'K-12 MG1655', 'MG1655::phiO104', 'Culture', 'nan', 'nan', 'phageO104 in the wrbA gene', '10.1186/s12864-019-5892-x'], ['SRR8587788', 'K-12 MG1655', 'MG1655::phiO104', 'Culture', 'nan', 'nan', 'phageO104 in the wrbA gene', '10.1186/s12864-019-5892-x'], ['SRR8587789', 'K-12 MG1655', 'MG1655::phiO104', 'Culture', 'nan', 'nan', 'phageO104 in the wrbA gene', '10.1186/s12864-019-5892-x'], ['SRR8587790', 'K-12 MG1655', 'MG1655::phiPA8', 'Culture', 'nan', 'nan', 'phagePA8 in the argW gene', '10.1186/s12864-019-5892-x'], ['SRR8587791', 'K-12 MG1655', 'MG1655::phiPA8', 'Culture', 'nan', 'nan', 'phagePA8 in the argW gene', '10.1186/s12864-019-5892-x'], ['SRR8587792', 'K-12 MG1655', 'MG1655::phiPA8', 'Culture', 'nan', 'nan', 'phagePA8 in the argW gene', '10.1186/s12864-019-5892-x'], ['SRR7261719', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261721', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261723', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261725', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261727', 'K-12 BW25113', 'Diesel exhaust atmosphere-adapted', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7261729', 'K-12 BW25113', 'Diesel exhaust atmosphere-adapted', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-47427-7'], ['SRR7216844', 'APEC O2-211', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7216845', 'APEC O2-211', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7217923', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'High pressure', '10.1371/journal.pone.0200660'], ['SRR7217924', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'High pressure', '10.1371/journal.pone.0200660'], ['SRR7217925', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'High pressure', '10.1371/journal.pone.0200660'], ['SRR7217926', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'High pressure', '10.1371/journal.pone.0200660'], ['SRR7217927', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Low pressure', '10.1371/journal.pone.0200660'], ['SRR7217928', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Low pressure', '10.1371/journal.pone.0200660'], ['SRR7217929', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Low pressure', '10.1371/journal.pone.0200660'], ['SRR7217930', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Low pressure', '10.1371/journal.pone.0200660'], ['SRR7905329', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', '2% ethanol', '10.1007/s00253-018-9509-0'], ['SRR7905330', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', 'blank control', '10.1007/s00253-018-9509-0'], ['SRR7905331', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', 'blank control', '10.1007/s00253-018-9509-0'], ['SRR7905332', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', 'blank control', '10.1007/s00253-018-9509-0'], ['SRR7905333', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', '2% ethanol', '10.1007/s00253-018-9509-0'], ['SRR7905334', 'BL21(pEt26b-katA)', 'nan', 'Culture', 'nan', 'nan', '2% ethanol', '10.1007/s00253-018-9509-0'], ['SRR7878514', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'with pT7-7 encoding NinH', 'nan'], ['SRR7878515', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'with pT7-7 encoding NinH', 'nan'], ['SRR7878516', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'Control', 'nan'], ['SRR7878517', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'Control', 'nan'], ['SRR7878518', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'Control', 'nan'], ['SRR7878519', 'K-12 BW25113', 'nan', 'Homo sapiens', 'nan', 'nan', 'with pT7-7 encoding NinH', 'nan'], ['SRR9600395', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Gamma irradiation', '10.1016/j.micpath.2020.104342'], ['SRR9600396', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Control', '10.1016/j.micpath.2020.104342'], ['SRR9600397', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Control', '10.1016/j.micpath.2020.104342'], ['SRR9600398', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Control', '10.1016/j.micpath.2020.104342'], ['SRR9600401', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Gamma irradiation', '10.1016/j.micpath.2020.104342'], ['SRR9600402', 'O157:H7', 'nan', 'Animal intestine', 'nan', 'nan', 'Gamma irradiation', '10.1016/j.micpath.2020.104342'], ['SRR7175042', 'LP-4 mutant', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-018-05830-0'], ['SRR7175043', 'LP-7 mutant', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-018-05830-0'], ['SRR9613401', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso', '10.1128/AAC.01207-19'], ['SRR9613404', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'kirromycin', '10.1128/AAC.01207-19'], ['SRR9613405', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'kirromycin', '10.1128/AAC.01207-19'], ['SRR9613406', 'WO153', 'nan', 'Culture', 'nan', 'nan', 'dmso', '10.1128/AAC.01207-19'], ['SRR7140738', '2429', 'nan', 'Culture', 'nan', 'nan', 'SB broth', '10.1073/pnas.1901730116'], ['SRR7140739', '3780', 'nan', 'Culture', 'nan', 'nan', 'SB broth', '10.1073/pnas.1901730116'], ['SRR7140740', '3780', 'nan', 'Culture', 'nan', 'nan', 'SB broth', '10.1073/pnas.1901730116'], ['SRR8521063', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521064', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521065', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521066', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521067', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521068', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'none', '10.1021/acsbiomaterials.9b01087'], ['SRR8521069', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521070', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521071', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521072', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521073', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521074', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521075', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521076', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521077', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521078', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521079', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521080', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdTe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521081', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521082', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521083', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521084', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521085', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521086', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521087', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521088', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521089', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521090', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521091', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521092', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 nM CdSe-2.4 quantum dot\\\\, no LED illumination', '10.1021/acsbiomaterials.9b01087'], ['SRR8521424', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521425', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521426', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521427', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521428', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521429', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521430', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521431', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521432', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521433', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521434', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521435', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521436', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521437', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521438', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521439', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521440', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521441', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521442', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521443', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521444', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521445', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521447', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521448', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521450', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'sucrose', '10.1016/j.molcel.2019.08.013'], ['SRR8521451', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521452', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521453', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521454', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no chloramphenicol', '10.1016/j.molcel.2019.08.013'], ['SRR8521455', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521456', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521457', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521458', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521459', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521460', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521461', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521462', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Kasugamycin', '10.1016/j.molcel.2019.08.013'], ['SRR8521464', 'X1488', 'nan', 'Culture', 'nan', 'nan', 'sucrose', '10.1016/j.molcel.2019.08.013'], ['SRR7166251', 'DH5', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7166252', 'DH5', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7866023', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866024', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866025', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866026', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866027', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866028', 'K-12 W3110', 'rpoA mutant', 'A1-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866029', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866030', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866031', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866032', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866033', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866034', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866035', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866036', 'K-12 W3110', 'rob mutant', 'G5-2 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866037', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866038', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866039', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866040', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866041', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866042', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866043', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866044', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866045', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866046', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866047', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866048', 'K-12 W3110', 'WT', 'D13 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 0 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866049', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866050', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866051', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866052', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866053', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866054', 'K-12 W3110', 'emrA mutant', 'G5-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866055', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866056', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866057', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR7866058', 'K-12 W3110', 'rfaY mutant', 'C3-1 culture', 'nan', 'nan', 'exponential phase at pH 6.5 with 5 mM benzoate', '10.1128/AEM.00966-19'], ['SRR8085109', 'K-12 MG1655', 'no expression of small RNA RydC', 'Culture', 'nan', '10min', 'Vector control', '10.1128/mSphere.00561-18'], ['SRR8085110', 'K-12 MG1655', 'no expression of small RNA RydC', 'Culture', 'nan', '10min', 'Vector control', '10.1128/mSphere.00561-18'], ['SRR8085111', 'K-12 MG1655', 'no expression of small RNA RydC', 'Culture', 'nan', '10min', 'Vector control', '10.1128/mSphere.00561-18'], ['SRR8085112', 'K-12 MG1655', 'overexpression of small RNA RydC', 'Culture', 'nan', '10min', 'Plac-rydC plasmids', '10.1128/mSphere.00561-18'], ['SRR8085113', 'K-12 MG1655', 'overexpression of small RNA RydC', 'Culture', 'nan', '10min', 'Plac-rydC plasmids', '10.1128/mSphere.00561-18'], ['SRR8085114', 'K-12 MG1655', 'overexpression of small RNA RydC', 'Culture', 'nan', '10min', 'Plac-rydC plasmids', '10.1128/mSphere.00561-18'], ['SRR8088330', 'EDL931', 'nan', 'Mouse fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR8088331', 'EDL931', 'nan', 'Mouse fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR8088332', 'EDL931', 'nan', 'Human fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR8088333', 'EDL931', 'nan', 'Human fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR8088334', 'EDL931', 'nan', 'Mouse fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR8088335', 'EDL931', 'nan', 'Human fecal matter', 'nan', 'nan', 'nan', '10.1186/s40168-019-0650-5'], ['SRR9855372', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'CL-IBG3', 'nan'], ['SRR9855373', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'KB2', 'nan'], ['SRR9855374', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'KB1', 'nan'], ['SRR9855375', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'CL1', 'nan'], ['SRR9855376', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'KB3', 'nan'], ['SRR9855377', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'CL3', 'nan'], ['SRR9855378', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'CL2', 'nan'], ['SRR9855379', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'IBG2', 'nan'], ['SRR9855380', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'IBG1', 'nan'], ['SRR9855382', 'ATCC 25922', 'nan', 'Homo sapiens', 'nan', 'nan', 'IBG3', 'nan'], ['SRR8182978', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182979', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182990', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182991', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182992', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182993', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182994', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182995', 'K-12 BW25113', 'delta[adhE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182996', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182997', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182998', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8182999', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183000', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183001', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183002', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183003', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183004', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183005', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183006', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183007', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'total RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183008', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183009', 'K-12 BW25113', 'delta[rpoS]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183010', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183011', 'K-12 BW25113', 'delta[relE]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183012', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183013', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183014', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183015', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183016', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183017', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183018', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183019', 'K-12 BW25113', 'delta[relA]', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183032', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183033', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183038', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8183039', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'polysomal RNA-seq', '10.1016/j.celrep.2018.08.093'], ['SRR8799594', 'HM01', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799595', 'HM03', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799596', 'HM06', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799597', 'HM07', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799598', 'HM14', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799600', 'HM43', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799601', 'HM54', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799602', 'HM56', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799606', 'HM68', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799607', 'HM86', 'nan', 'UTI patient urine', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799609', 'HM03', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799610', 'HM06', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799611', 'HM07', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799612', 'HM14', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799613', 'HM17', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799614', 'HM43', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799615', 'HM54', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799616', 'HM56', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799620', 'HM68', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8799621', 'HM86', 'nan', 'in vitro urine culture', 'nan', 'nan', 'nan', '10.7554/eLife.49748'], ['SRR8836010', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836011', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8836012', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8836013', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836014', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836015', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836016', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8836017', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8836018', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8836019', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836020', 'K12 C3000', 'ciprofloxacin resistant strain', 'Culture', 'nan', 'nan', 'co-exposure', '10.1021/acs.est.0c01155'], ['SRR8836031', 'K12 C3000', 'ampicillin resistant strain', 'Culture', 'nan', 'nan', 'amp-exposure', '10.1021/acs.est.0c01155'], ['SRR8664715', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', '1/2MIC OLA treated', '10.3390/antibiotics9110791'], ['SRR8664716', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', '1/2MIC OLA treated', '10.3390/antibiotics9110791'], ['SRR8664719', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', '1/10MIC OLA treated', '10.3390/antibiotics9110791'], ['SRR8664720', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', 'untreated', '10.3390/antibiotics9110791'], ['SRR8664721', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', '1/2MIC OLA treated', '10.3390/antibiotics9110791'], ['SRR8664722', 'ATCC 25922', 'nan', 'Culture', 'nan', 'nan', '1/10MIC OLA treated', '10.3390/antibiotics9110791'], ['ERR3048193', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048194', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048196', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048200', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048201', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048202', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048203', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['ERR3048204', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-019-0641-0'], ['SRR8457199', 'K-12 MG1655', 'rph+', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz616'], ['SRR8457200', 'K-12 MG1655', 'rph+', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz616'], ['SRR8457201', 'K-12 MG1655', 'rph+ delta[pnp]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz616'], ['SRR8457202', 'K-12 MG1655', 'rph+ delta[pnp]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz616'], ['SRR6869344', 'EDL933', 'O157:H7', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12864-018-5167-y'], ['SRR7536294', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12918-018-0653-z'], ['SRR7536295', 'K-12 BW25113', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12918-018-0653-z'], ['SRR7536296', 'K-12 BW25113', 'delta[thrA]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536297', 'K-12 BW25113', 'delta[thrA]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536298', 'K-12 BW25113', 'delta[serB]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536299', 'K-12 BW25113', 'delta[serB]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536300', 'K-12 BW25113', 'delta[ptsI]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536301', 'K-12 BW25113', 'delta[ptsI]', 'Culture', 'nan', 'nan', 'Flask 5, Clone', '10.1186/s12918-018-0653-z'], ['SRR7536302', 'K-12 BW25113', 'delta[ptsI]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7536303', 'K-12 BW25113', 'delta[ptsI]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7536304', 'K-12 BW25113', 'delta[thrA]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7536305', 'K-12 BW25113', 'delta[thrA]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7536306', 'K-12 BW25113', 'delta[serB]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7536307', 'K-12 BW25113', 'delta[serB]', 'Culture', 'nan', 'nan', 'Flask 1, Population', '10.1186/s12918-018-0653-z'], ['SRR7537403', 'JW3367', 'nan', 'nan', 'nan', 'nan', 'LED 1/600Hz', 'nan'], ['SRR7537404', 'JW3367', 'nan', 'nan', 'nan', 'nan', 'LED 1/900Hz', 'nan'], ['SRR7537405', 'JW3367', 'nan', 'nan', 'nan', 'nan', 'LED 1/1200Hz', 'nan'], ['SRR7537406', 'JW3367', 'nan', 'nan', 'nan', 'nan', 'LED 1/1800Hz', 'nan'], ['SRR6841089', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'MOPS minimal media + glycerol', '10.1128/JB.00411-18'], ['SRR6841094', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841096', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841097', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841107', 'K-12 MG1655', 'Lrp::kanR', 'Culture', 'nan', 'nan', 'MOPS minimal media + glycerol', '10.1128/JB.00411-18'], ['SRR6841113', 'K-12 MG1655', 'Lrp::kanR', 'Culture', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841118', 'K-12 MG1655', 'Lrp::kanR', 'Culture', 'nan', 'nan', 'MOPS rich defined media + glycerol', '10.1128/JB.00411-18'], ['SRR9286024', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Control', 'nan'], ['SRR9286025', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Control', 'nan'], ['SRR9286026', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Treated with PBT2', 'nan'], ['SRR9286028', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Treated with PBT2', 'nan'], ['SRR9286023', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Control', 'nan'], ['SRR9286027', 'MS8345', 'nan', 'nan', 'nan', 'nan', 'Treated with PBT2', 'nan'], ['ERR2729622', 'nan', 'Temperature sensitive RNase E mutant', 'nan', 'nan', 'nan', 'nan', '10.1128/mBio.01730-18'], ['SRR8485399', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', 'Control', '10.1039/c9sc03532j'], ['SRR8485403', 'Sakai', 'O157:H7', 'Culture', 'nan', 'nan', \"Treated with delta[5b']\", '10.1039/c9sc03532j'], ['SRR7154624', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'D-lyxose', '10.15252/msb.20188462'], ['SRR7154625', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-lyxose', '10.15252/msb.20188462'], ['SRR7154626', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-lyxose', '10.15252/msb.20188462'], ['SRR7154627', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-lyxose', '10.15252/msb.20188462'], ['SRR7154628', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-lyxose', '10.15252/msb.20188462'], ['SRR7154629', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'D-2-deoxyribose', '10.15252/msb.20188462'], ['SRR7154631', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'D-2-deoxyribose', '10.15252/msb.20188462'], ['SRR7154632', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-2-deoxyribose', '10.15252/msb.20188462'], ['SRR7154633', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154634', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154635', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154636', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154637', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154638', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'D-arabinose', '10.15252/msb.20188462'], ['SRR7154639', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154640', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154641', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154642', 'K-12 MG1655', 'Evolved - early/innovation', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154643', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154644', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154645', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['SRR7154646', 'K-12 MG1655', 'Evolved - endpoint/optimization', 'Culture', 'nan', 'nan', 'm-tartrate', '10.15252/msb.20188462'], ['ERR2780171', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['ERR2780172', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['ERR2780173', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['ERR2780174', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['ERR2780175', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['ERR2780176', 'K-12 MG1655', 'pBIC1a', 'Culture', 'nan', 'nan', 'imipenem exposure', '10.3389/fmicb.2018.02929'], ['SRR7851671', 'DT900', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR8485400', 'O157:H7 str. Sakai', 'nan', 'Culture', 'nan', 'nan', 'methanol', 'nan'], ['SRR8485401', 'O157:H7 str. Sakai', 'nan', 'Culture', 'nan', 'nan', '0.5 ug/ml of lambda-5b', 'nan'], ['SRR7057985', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057986', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057987', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057988', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057989', 'K-12 MG1655', 'delta[ybaO]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057990', 'K-12 MG1655', 'delta[ybaO]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057991', 'K-12 MG1655', 'delta[ybaQ]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057992', 'K-12 MG1655', 'delta[ybaQ]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057993', 'K-12 MG1655', 'delta[ybiH]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057994', 'K-12 MG1655', 'delta[ybiH]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057995', 'K-12 MG1655', 'delta[ydcI]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057996', 'K-12 MG1655', 'delta[ydcI]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057997', 'K-12 MG1655', 'delta[ydcI]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057998', 'K-12 MG1655', 'delta[ydcI]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7057999', 'K-12 MG1655', 'delta[yddM]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7058000', 'K-12 MG1655', 'delta[yddM]', 'Culture', 'nan', 'nan', 'nan', 'https://doi.org/10.1093/nar/gky752'], ['SRR7058001', 'K-12 MG1655', 'delta[yheO]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky752'], ['SRR7058002', 'K-12 MG1655', 'delta[yheO]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky752'], ['SRR4427755', 'NEB 10-beta', 'nan', 'Culture grown in 14 ml tube', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427756', 'NEB 10-beta', 'nan', 'Culture grown in 14 ml tube', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427757', 'NEB 10-beta', 'nan', 'Culture grown in 14 ml tube', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427758', 'NEB 10-beta', 'nan', 'Culture grown in 14 ml tube', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427760', 'NEB 10-beta', 'nan', 'Culture grown in 14 ml tube', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427763', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427764', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427765', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427766', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427767', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427768', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427769', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4427770', 'NEB 10-beta', 'nan', 'Culture grown in 250 ml Erlenmeyer flask', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR4436361', 'O157:H7 strain 6564', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1186/s12866-017-0966-x'], ['SRR7473187', 'S17-3', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7473188', 'S17-3', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7473189', 'S17-3', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7473190', 'S17-3', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR7474255', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR7474256', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR7474257', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR7474258', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR7474259', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR7474260', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/s13036-019-0159-2'], ['SRR9207138', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR9207139', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR9207140', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR9207141', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR9207142', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR9207143', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.3390/ijms21155590'], ['SRR8315492', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', 'control (ground lab)', '10.2217/fmb-2018-0273'], ['SRR8315493', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315494', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315495', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315496', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315497', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315498', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315499', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315500', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315501', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315502', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315503', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315504', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315505', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315506', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315509', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315511', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315512', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315513', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315517', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315527', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR8315529', 'T1_13', 'nan', 'Feces of patients with severe pneumonia', 'nan', 'nan', '64 Days Space', '10.2217/fmb-2018-0273'], ['SRR7630962', 'K-12 MG1655', 'rpos', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12859-019-2704-x'], ['SRR7630963', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12859-019-2704-x'], ['SRR8545712', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with WT RM (p18)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8545713', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with WT RM (p18)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8545714', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with C+R- RM (p24)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8545715', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with C+R- RM (p24)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8545716', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with C-R+ RM (p30)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8545717', 'K-12 MG1655', 'pMCsp231 with additional protective MTase gene on ACYC177 derivative\\\\, and with plasmid with C-R+ RM (p30)', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkz665'], ['SRR8485402', 'O157:H7 str. Sakai', 'nan', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['SRR8517249', 'AR0069', 'resistant strain', 'Culture', 'nan', '60min', 'no meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517250', 'AR0048', 'resistant strain', 'Culture', 'nan', '60min', 'no meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517254', 'RB002', 'susceptible strain', 'Culture', 'nan', '60min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517255', 'RB001', 'susceptible strain', 'Culture', 'nan', '60min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517421', 'RB002', 'susceptible strain', 'Culture', 'nan', '30min', 'no meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517424', 'RB001', 'susceptible strain', 'Culture', 'nan', '120min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517425', 'RB002', 'susceptible strain', 'Culture', 'nan', '120min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517458', 'AR0069', 'resistant strain', 'Culture', 'nan', '60min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517459', 'AR0048', 'resistant strain', 'Culture', 'nan', '60min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517460', 'AR0069', 'resistant strain', 'Culture', 'nan', '30min', 'meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517563', 'RB002', 'susceptible strain', 'Culture', 'nan', '60min', 'no meropenem', '10.1038/s41591-019-0650-9'], ['SRR8517578', 'AR0069', 'resistant strain', 'Culture', 'nan', '30min', 'no meropenem', '10.1038/s41591-019-0650-9'], ['SRR10353326', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR9994135', 'Nissle 1917', 'nan', 'nan', 'nan', 'nan', 'EcN_incubated_with_T4_phages_2 h', 'nan'], ['SRR8218063', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218064', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218065', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218066', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218067', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218068', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218069', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218070', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218071', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218072', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Ferrous sulfate and sodium citrate', '10.1038/s41564-018-0340-2'], ['SRR8218073', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218074', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218075', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218076', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218077', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR8218078', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41564-018-0340-2'], ['SRR1573435', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1128/AEM.02246-14'], ['SRR1573441', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1128/AEM.02246-14'], ['SRR1582908', 'K-12 MG1655', 'nan', 'Culture', 'nan', '0 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583069', 'K-12 MG1657', 'nan', 'Culture', 'nan', '1 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583070', 'K-12 MG1658', 'nan', 'Culture', 'nan', '1.5 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583071', 'K-12 MG1659', 'nan', 'Culture', 'nan', '2 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583074', 'K-12 MG1661', 'nan', 'Culture', 'nan', '4 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583075', 'K-12 MG1662', 'nan', 'Culture', 'nan', '5 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583077', 'K-12 MG1664', 'nan', 'Culture', 'nan', '7 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR1583079', 'K-12 MG1666', 'nan', 'Culture', 'nan', '9 min post rifampicin', 'nan', '10.15252/msb.20145794'], ['SRR5071589', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '5min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071590', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '10min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071591', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071592', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '45min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071593', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '75min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071594', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071595', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '210min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071596', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '330min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071597', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071598', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '26h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071599', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071600', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-16h', 'long term, ref 1', '10.1111/1751-7915.12713'], ['SRR5071601', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-15h', 'long term, ref 2', '10.1111/1751-7915.12713'], ['SRR5071602', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-2h', 'long term, ref 3', '10.1111/1751-7915.12713'], ['SRR5071603', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '5min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071604', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '10min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071605', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071606', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071607', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071608', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '45min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071609', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071610', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071611', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071612', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '210min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071613', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25h', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071614', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071615', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071616', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071617', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '5min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071618', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '10min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071619', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071620', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '45min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071621', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '75min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071622', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071623', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '210min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071624', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '330min', 'long term', '10.1111/1751-7915.12713'], ['SRR5071625', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071626', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '26h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071627', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'long term', '10.1111/1751-7915.12713'], ['SRR5071628', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-16h', 'long term, ref 1', '10.1111/1751-7915.12713'], ['SRR5071629', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-15h', 'long term, ref 2', '10.1111/1751-7915.12713'], ['SRR5071630', 'K-12 W3110', 'nan', 'chemostat STR culture', 'nan', '0-2h', 'long term, ref 3', '10.1111/1751-7915.12713'], ['SRR5071631', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '5min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071632', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '10min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071633', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071634', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071635', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25min', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071637', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '75min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071638', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P5', '10.1111/1751-7915.12713'], ['SRR5071639', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071640', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '120min', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071641', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '210min', 'short term, P1', '10.1111/1751-7915.12713'], ['SRR5071642', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '330min', 'short term,P5', '10.1111/1751-7915.12713'], ['SRR5071643', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '25h', 'short term,P5', '10.1111/1751-7915.12713'], ['SRR5071644', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '26h', 'short term,P5 (most ammonia starved', '10.1111/1751-7915.12713'], ['SRR5071645', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term,P5', '10.1111/1751-7915.12713'], ['SRR5071646', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term, P3', '10.1111/1751-7915.12713'], ['SRR5071647', 'K-12 W3110', 'nan', 'chemostat STR-PFR culture', 'nan', '28h', 'short term, P1 (little ammonia starvation)', '10.1111/1751-7915.12713'], ['SRR1571006', 'LCT-ECG001', 'nan', 'sputum', 'nan', 'nan', 'nan', 'nan'], ['SRR1573434', 'K-12 MG1655', 'WT', 'Total RNA', 'nan', 'nan', 'nan', '10.1128/AEM.02246-14'], ['SRR1573442', 'K-12 MG1655', 'Evolved', 'Total RNA', 'nan', 'nan', 'nan', '10.1128/AEM.02246-14'], ['SRR5051298', 'BW27785', 'Rho prion-containing cells', 'Total RNA ', 'nan', 'nan', 'nan', '10.1126/science.aai7776'], ['SRR5051299', 'BW27785', 'Rho prion-containing cells', 'Total RNA', 'nan', 'nan', 'nan', '10.1126/science.aai7776'], ['SRR5051300', 'BW27785', 'Rho prion-lacking cells', 'Total RNA ', 'nan', 'nan', 'nan', '10.1126/science.aai7776'], ['SRR5051301', 'BW27785', 'Rho prion-lacking cells', 'Total RNA', 'nan', 'nan', 'nan', '10.1126/science.aai7776'], ['SRR5051302', 'BW27785', 'Rho prion-lacking cells', 'Total RNA ', 'nan', 'nan', 'nan', '10.1126/science.aai7776'], ['SRR1301051', 'K-12 MG1655', 'WT', 'whole cell lysate', 'nan', 'nan', 'nan', '10.1016/j.cell.2014.10.043'], ['SRR345260', 'O139:H28 str. E24377A', 'nan', 'Culture', 'nan', 'nan', 'LB only', '10.1128/IAI.06138-11'], ['SRR3473904', 'K-12 ', 'EcsR2 WT Compliment plasmid', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3473905', 'K-12', 'EcsR2 WT Compliment plasmid', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491481', 'K-12 MG1655', 'mutS_ancestral', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491482', 'K-12 MG1655', 'mutS_ancestral', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491483', 'K-12 MG1655', 'mutS_ancestral', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491484', 'K-12 MG1656', 'mutS_1_t200', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491485', 'K-12 MG1656', 'mutS_1_t200', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491486', 'K-12 MG1656', 'mutS_1_t200', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491487', 'K-12 MG1657', 'mutS_1_t250', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3491488', 'K-12 MG1657', 'mutS_1_t250', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3033275', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033276', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033277', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033278', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033280', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033281', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033282', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033283', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033284', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033285', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033286', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033287', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Control', '10.1016/j.dib.2016.08.069'], ['SRR3033288', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033289', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033290', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033291', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033293', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033294', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033295', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033296', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033297', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033298', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033299', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR3033300', 'K-12 W3110', 'nan', 'nan', 'nan', 'nan', 'Nickel', '10.1016/j.dib.2016.08.069'], ['SRR2131268', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2131285', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132336', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132337', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132373', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132377', 'B REL1206', 'REL1206', 'Culture', '37C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132392', 'B REL1206', 'REL1206', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132401', 'B REL1206', 'REL1206 with mutation rpoB I572F', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132458', 'B REL1206', 'REL1206 with mutation rpoB I572F', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132462', 'B REL1206', 'REL1206 with mutation rpoB I572F', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132465', 'B REL1206', 'REL1206 with mutation rpoB I572F', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132489', 'B REL1206', 'REL1206 with mutation rpoB I572L', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132506', 'B REL1206', 'REL1206 with mutation rpoB I572L', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132526', 'B REL1206', 'REL1206 with mutation rpoB I572L', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132539', 'B REL1206', 'REL1206 with mutation rpoB I572N', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132542', 'B REL1206', 'REL1206 with mutation rpoB I572N', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132563', 'B REL1206', 'REL 1206 with mutation rpoB I966S', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132565', 'B REL1206', 'REL 1206 with mutation rpoB I966S', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132574', 'B REL1206', 'REL 1206 with mutation rpoB I966S', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132578', 'B REL1206', 'REL 1206 with mutation rpoB I966S', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132594', 'B REL1206', 'Clone 27', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132597', 'B REL1206', 'Clone 27', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132602', 'B REL1206', 'Clone 97', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132605', 'B REL1206', 'Clone 97', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2132614', 'B REL1206', 'Clone 97', 'Culture', '42C', 'nan', 'nan', '10.1093/molbev/msx216'], ['SRR2167766', 'K-12 MG1655', 'nan', 'Biofilm culture', 'nan', 'nan', 'Aerobic', '10.1093/femsle/fnx006'], ['SRR2167767', 'K-12 MG1655', 'nan', 'Biofilm culture', 'nan', 'nan', 'Aerobic', '10.1093/femsle/fnx006'], ['SRR2167769', 'K-12 MG1655', 'nan', 'Biofilm culture', 'nan', 'nan', 'Anaerobic', '10.1093/femsle/fnx006'], ['SRR2167770', 'K-12 MG1655', 'nan', 'Planktonic culture', 'nan', 'nan', 'Aerobic', '10.1093/femsle/fnx006'], ['ERR951779', 'K-12 MG1655', 'delta[rpoN] + pBAD24', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1005552'], ['ERR951780', 'K-12 MG1655', 'delta[rpoN] + pBAD24', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1005552'], ['ERR951781', 'K-12 MG1655', 'delta[rpoN] + pRpoN', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1005552'], ['ERR951782', 'K-12 MG1655', 'delta[rpoN] + pRpoN', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1005552'], ['SRR2153541', 'K-12 MG1655', 'delta[mutS] line A100', 'Culture', 'nan', 'nan', 'nan', '10.1093/gbe/evw205'], ['SRR1181647', 'K-12 MG1655', '3xFlag Hfq', 'Culture', 'nan', 'nan', 'nan', '10.4161/rna.29299'], ['SRR1181648', 'K-12 MG1655', 'HA Hfq', 'Culture', 'nan', 'nan', 'nan', '10.4161/rna.29299'], ['SRR3144708', 'K12f', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3146202', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'No pressure sample (media only)', '10.1128/mSphere.00009-17'], ['SRR3146204', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to ampicillin', '10.1128/mSphere.00009-17'], ['SRR3146206', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to tetracycline', '10.1128/mSphere.00009-17'], ['SRR3146207', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to tetracycline', '10.1128/mSphere.00009-17'], ['SRR3146210', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Adapted to n-butanol', '10.1128/mSphere.00009-17'], ['SRR3103785', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'vector control (vector)', '10.1128/mSystems.00021-15'], ['SRR3103786', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'vector control (vector)', '10.1128/mSystems.00021-15'], ['SRR3103787', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'vector control (vector)', '10.1128/mSystems.00021-15'], ['SRR3103788', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'plasmid expressing dicF (DicF)', '10.1128/mSystems.00021-15'], ['SRR3103789', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'plasmid expressing dicF (DicF)', '10.1128/mSystems.00021-15'], ['SRR3103790', 'K-12 MG1655', 'delta[dicF, lacIq]', 'Culture', 'nan', 'nan', 'plasmid expressing dicF (DicF)', '10.1128/mSystems.00021-15'], ['ERR1218301', 'ETEC E873', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1218302', 'ETEC E1101', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1223488', 'K-12 MG1655', 'delta[thyA] delta[nusB]::thyA', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['ERR1223489', 'K-12 MG1655', 'delta[thyA] delta[nusB]::thyA', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['ERR1223490', 'K-12 MG1655', 'delta[thyA] delta[suhB]::thyA', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['ERR1223491', 'K-12 MG1655', 'delta[thyA] delta[suhB]::thyA', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['ERR1223492', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['ERR1223493', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1128/mBio.00114-16'], ['SRR1168135', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'DPD', '10.1038/ncomms5910'], ['SRR1168136', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'DPD', '10.1038/ncomms5910'], ['SRR1168140', 'K-12 MG1655', 'delta[fur]', 'Culture', 'nan', 'nan', 'DPD', '10.1038/ncomms5910'], ['SRR2097724', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '10min', 'Gentamicin', 'nan'], ['SRR2097908', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2098356', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '120min', 'Trimethoprim', 'nan'], ['SRR2098632', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '0min', 'No treatment of antibiotic', 'nan'], ['SRR2098633', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '10min', 'Trimethoprim', 'nan'], ['SRR2098793', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '120min', 'No treatment of antibiotic', 'nan'], ['SRR2098803', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2098819', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '10min', 'Trimethoprim', 'nan'], ['SRR2098912', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2098933', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '120min', 'No treatment of antibiotic', 'nan'], ['SRR2098967', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'Trimethoprim', 'nan'], ['SRR2099308', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2099415', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '0min', 'Trimethoprim', 'nan'], ['SRR2099429', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '120min', 'Trimethoprim', 'nan'], ['SRR2099433', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '0min', 'No treatment of antibiotic', 'nan'], ['SRR2099477', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '10min', 'No treatment of antibiotic', 'nan'], ['SRR2099478', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'Trimethoprim', 'nan'], ['SRR2099496', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '30min', 'Trimethoprim', 'nan'], ['SRR2099525', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '10min', 'No treatment of antibiotic', 'nan'], ['SRR2099539', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2099552', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '30min', 'No treatment of antibiotic', 'nan'], ['SRR2099615', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '10min', 'Trimethoprim', 'nan'], ['SRR2099706', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '0min', 'No treatment of antibiotic', 'nan'], ['SRR2099720', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '0min', 'Trimethoprim', 'nan'], ['SRR2099910', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '10min', 'No treatment of antibiotic', 'nan'], ['SRR2101831', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '0min', 'No treatment of antibiotic', 'nan'], ['SRR2101843', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '120min', 'Trimethoprim', 'nan'], ['SRR2101867', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '0min', 'Trimethoprim', 'nan'], ['SRR2101871', 'nan', 'susceptible strain', 'Homo sapiens', 'nan', '10min', 'No treatment of antibiotic', 'nan'], ['SRR2102261', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '0min', 'Trimethoprim', 'nan'], ['SRR2102366', 'nan', 'resistant strain', 'Homo sapiens', 'nan', '120min', 'No treatment of antibiotic', 'nan'], ['SRR2062331', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate-', '10.1261/rna.052886.115'], ['SRR2062332', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate-', '10.1261/rna.052886.115'], ['SRR2062333', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate-', '10.1261/rna.052886.115'], ['SRR2062334', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate-', '10.1261/rna.052886.115'], ['SRR2062335', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate+', '10.1261/rna.052886.115'], ['SRR2062336', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate+', '10.1261/rna.052886.115'], ['SRR2062337', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate+', '10.1261/rna.052886.115'], ['SRR2062338', 'K-12 MG1655', 'pnpFLAG', 'Culture', 'nan', 'nan', 'Tungstate+', '10.1261/rna.052886.115'], ['SRR2062523', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Butanol', '10.1038/srep19899'], ['SRR2062524', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'N,N-dimethylacetamide', '10.1038/srep19899'], ['SRR2062533', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'DMS', '10.1038/srep19899'], ['SRR5097611', 'K-12 BW25113', 'nan', 'intestine', 'nan', '30min', 'Aerosolised', '10.1007/s00253-018-9083-5'], ['SRR5097614', 'K-12 BW25113', 'nan', 'intestine', 'nan', '30min', 'Aerosolised', '10.1007/s00253-018-9083-5'], ['SRR5097615', 'K-12 BW25113', 'nan', 'intestine', 'nan', '30min', 'Aerosolised', '10.1007/s00253-018-9083-5'], ['SRR5097618', 'K-12 BW25113', 'nan', 'intestine', 'nan', '30min', 'Aerosolised', '10.1007/s00253-018-9083-5'], ['SRR5097620', 'K-12 BW25113', 'nan', 'intestine', 'nan', '30min', 'Aerosolised', '10.1007/s00253-018-9083-5'], ['SRR3729390', 'ECC-Z', 'nan', 'Stool', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR3729499', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swimming', '10.1128/mSystems.00064-16'], ['SRR3729500', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swarming', '10.1128/mSystems.00064-17'], ['SRR3729574', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'planktonic', '10.1128/mSystems.00064-18'], ['SRR3729575', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swimming', '10.1128/mSystems.00064-19'], ['SRR3729576', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swarming', '10.1128/mSystems.00064-20'], ['SRR3729624', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'planktonic', '10.1128/mSystems.00064-21'], ['SRR3729625', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swimming', '10.1128/mSystems.00064-22'], ['SRR3729626', 'ECC-Z', 'nan', 'Bos taurus', 'nan', 'nan', 'swarming', '10.1128/mSystems.00064-23'], ['SRR5626392', 'nan', 'nan', 'Sewage and sediment', 'nan', 'nan', 'nan', 'nan'], ['SRR1563011', 'nan', 'nan', 'Homo Sapiens (Sputum)', 'nan', 'nan', 'nan', 'nan'], ['SRR1564968', 'nan', 'nan', 'Homo Sapiens (Sputum)', 'nan', 'nan', 'nan', 'nan'], ['SRR2299393', 'O157:H7', '43894', 'Stool', 'nan', 'nan', 'nan', '10.1128/MRA.01143-20'], ['SRR2299394', 'O157:H7', '43894OR', 'Culture', 'nan', 'nan', 'nan', '10.1128/MRA.01143-20'], ['SRR2299395', 'O157:H7', 'PA20', 'Stool', 'nan', 'nan', 'nan', '10.1128/MRA.01143-20'], ['SRR2299396', 'O157:H7', '20R2R', 'Culture', 'nan', 'nan', 'nan', '10.1128/MRA.01143-20'], ['SRR3403686', 'REL4536', 'nan', 'Culture', 'nan', 'nan', 'Aerobic', '10.1371/journal.pgen.1006570'], ['SRR3403687', 'REL4536', 'nan', 'Culture', 'nan', 'nan', 'Aerobic', '10.1371/journal.pgen.1006570'], ['SRR3403688', 'REL4536', 'nan', 'Culture', 'nan', 'nan', 'Aerobic', '10.1371/journal.pgen.1006570'], ['SRR794827', 'K-12 MG1655', 'pHDB3', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794828', 'K-12 MG1655', 'pHDB3', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794829', 'K-12 MG1655', 'pHDB3', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794830', 'K-12 MG1655', 'pLCV1', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794831', 'K-12 MG1655', 'pLCV1', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794832', 'K-12 MG1655', 'pLCV1', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.2% Glucose', '10.1093/nar/gkt444'], ['SRR794833', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB', '10.1093/nar/gkt444'], ['SRR794834', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB', '10.1093/nar/gkt444'], ['SRR794835', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB', '10.1093/nar/gkt444'], ['SRR794836', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794837', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794838', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794839', 'K-12 MG1655', 'SgrR', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794840', 'K-12 MG1655', 'SgrR', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794841', 'K-12 MG1655', 'SgrR', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794842', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794843', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794844', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794845', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794846', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794847', 'K-12 MG1655', 'sgrS', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794848', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794849', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794850', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'LB + 0.5% aMG', '10.1093/nar/gkt444'], ['SRR794851', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794852', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Rich + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794854', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794855', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794856', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Rich+ 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794863', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794864', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794865', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794866', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794867', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794868', 'K-12 MG1655', 'MG1655', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% aMG', '10.1093/nar/gkt444'], ['SRR794869', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794870', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794871', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol', '10.1093/nar/gkt444'], ['SRR794872', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% 2-DG', '10.1093/nar/gkt444'], ['SRR794873', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% 2-DG', '10.1093/nar/gkt444'], ['SRR794874', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Defined MOPS Minimal + 0.4% Glycerol +0.5% 2-DG', '10.1093/nar/gkt444'], ['SRR1514290', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'glucose', '10.1016/j.cels.2016.04.003'], ['SRR1514291', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514292', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514296', 'K-12 MG1655', 'rpoBE546V mutant', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514297', 'K-12 MG1655', 'rpoBE546V mutant', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514300', 'K-12 MG1655', 'rpoBE672K mutant', 'Culture', 'nan', 'nan', 'glucose', '10.1016/j.cels.2016.04.003'], ['SRR1514301', 'K-12 MG1655', 'rpoBE672K mutant', 'Culture', 'nan', 'nan', 'glucose', '10.1016/j.cels.2016.04.003'], ['SRR1514302', 'K-12 MG1655', 'rpoBE672K mutant', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514303', 'K-12 MG1655', 'rpoBE672K mutant', 'Culture', 'nan', 'nan', 'LB complex media', '10.1016/j.cels.2016.04.003'], ['SRR1514602', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Exp (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514603', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Exp (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514604', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Exp (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514605', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Exp (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514606', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Exp (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514609', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Trans (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514610', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Trans (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514611', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Trans (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514612', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Trans (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514613', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Trans (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1514617', 'GLBRCE1', 'nan', 'Culture', 'nan', 'nan', 'Stat1 (synthetic hydrolysate medium)', '10.3389/fmicb.2014.00402'], ['SRR1515276', 'CFT073', 'nan', 'Culture', 'nan', 'nan', 'MOPS media ', '10.1073/pnas.1413272111'], ['SRR1515279', 'CFT073', 'nan', 'Culture', 'nan', 'nan', 'Urine ', '10.1073/pnas.1413272111'], ['SRR1515280', 'CFT073', 'nan', 'Culture', 'nan', 'nan', 'Urine ', '10.1073/pnas.1413272111'], ['SRR1515282', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'MOPS media ', '10.1073/pnas.1413272111'], ['SRR1515284', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'MOPS media ', '10.1073/pnas.1413272111'], ['SRR1515287', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'Urine ', '10.1073/pnas.1413272111'], ['ERR1450560', 'K-12 MG1655', 'delta[hns], kanamycin', 'Culture', '30C', 'nan', 'Not spiked w Salmonella RNA', 'nan'], ['ERR1450562', 'K-12 MG1655', 'delta[hns], kanamycin', 'Culture', '30C', 'nan', 'Not spiked w Salmonella RNA', 'nan'], ['ERR1450564', 'K-12 MG1655', 'WT', 'Culture', '30C', 'nan', 'Not spiked w Salmonella RNA', 'nan'], ['ERR1450565', 'K-12 MG1655', 'WT', 'Culture', '37C', 'nan', 'Spiked w Salmonella RNA', 'nan'], ['ERR1450566', 'K-12 MG1655', 'WT', 'Culture', '30C', 'nan', 'Not spiked w Salmonella RNA', 'nan'], ['SRR3584193', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584194', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584195', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584196', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584197', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584198', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584199', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584200', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584201', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584202', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584203', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584204', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584205', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584206', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584207', 'K-12 MG1655', 'delta[mfd]', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584208', 'K-12 MG1655', 'WT + vector', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584210', 'K-12 MG1655', 'WT + vector', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR3584211', 'K-12 MG1655', 'WT + vector', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR1507250', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['SRR1507252', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['SRR1507253', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571750', 'K-12 MG1655', 'delta[flhD]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571751', 'K-12 MG1655', 'delta[fliA]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571752', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571753', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571754', 'K-12 MG1655', 'delta[flhD]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571755', 'K-12 MG1655', 'delta[flhA]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571756', 'K-12 MG1655', 'delta[flhC]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['ERR571757', 'K-12 MG1655', 'delta[flhC]', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['SRR5002509', 'O157:H7 (EDL933)', 'Esr055 mutant', 'Homo sapiens', 'nan', 'nan', 'nan', 'nan'], ['SRR5002510', 'O157:H7 (EDL933)', 'WT', 'Homo sapiens', 'nan', 'nan', 'nan', 'nan'], ['SRR5016659', 'PA20', 'nan', 'Stool', 'nan', 'nan', 'nan', 'nan'], ['SRR5016662', '20R2R', 'nan', 'Culture', 'nan', 'nan', 'nan', 'nan'], ['SRR5576781', 'K-12 SMG 123 (ATCC PTA-7555)', 'nan', 'CNTRENE CL100LM unexposed', 'nan', 'nan', 'nan', '10.7717/peerj.3721'], ['SRR3879778', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR3879779', 'K-12 MG1655 + BCM', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR3879780', 'nan', 'delta[dsrA, rprA, arcZ]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR3879781', 'nan', 'delta[dsrA, rprA, arcZ]+BCM', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR3879782', 'nan', 'delta[hfq]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR1554187', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554188', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR1554189', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554190', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR1554191', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554192', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR1554193', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554194', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR1554195', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554196', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR1554197', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Control', '10.1016/j.ymben.2015.06.005'], ['SRR1554198', 'K-12 MG1655', 'delta[lpP]', 'Culture', 'nan', 'nan', 'Nitric oxide', '10.1016/j.ymben.2015.06.005'], ['SRR4896361', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin-', '10.3389/fmicb.2018.01438'], ['SRR4896362', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin+', '10.3389/fmicb.2018.01438'], ['SRR4896363', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin-', '10.3389/fmicb.2018.01438'], ['SRR4896364', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin+', '10.3389/fmicb.2018.01438'], ['SRR4896365', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin-', '10.3389/fmicb.2018.01438'], ['SRR4896366', 'O25b:H4-ST131', 'nan', 'Clinical', 'nan', 'nan', 'ciprofloxacin+', '10.3389/fmicb.2018.01438'], ['SRR5569405', 'ECA-B', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569406', 'ECC-M', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569407', 'ECC-M', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569409', 'ECC-1470', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569411', 'ECA-727', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569415', 'ECC-Z', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569416', 'ECC-Z', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569417', 'ECA-B', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569419', 'ECA-727', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569421', 'ECA-0157', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR5569422', 'ECA-0157', 'nan', 'Bos taurus (Milk)', 'nan', 'nan', 'nan', '10.1128/mSystems.00064-16'], ['SRR1525109', 'K-12 MG1655', 'WT', 'Total RNA from E.coli', 'nan', 'nan', '10 mM Glycine in Media', '10.1038/ncomms13091'], ['SRR1525110', 'K-12 MG1655', 'WT', 'Total RNA from E.coli', 'nan', 'nan', '10 mM L-Threonine in Media', '10.1038/ncomms13091'], ['SRR1525111', 'K-12 MG1655', 'WT', 'Total RNA from E.coli', 'nan', 'nan', '10 mM L-Threonine in Media', '10.1038/ncomms13091'], ['SRR3825304', 'ATCC 8739', 'CAR005', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2017.08.005'], ['SRR3825305', 'ATCC 8739', 'G4H14', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2017.08.005'], ['SRR3825306', 'ATCC 8739', 'IspG1', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2017.08.005'], ['SRR4885947', 'JJ1887', 'nan', 'urine', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['SRR4885958', 'JJ1887', 'nan', 'urine', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['SRR4885961', 'JJ1887', 'nan', 'blood', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['SRR4885971', 'JJ1887', 'nan', 'urine', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['SRR4885973', 'JJ1887', 'nan', 'blood', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['SRR4885976', 'JJ1887', 'nan', 'blood', 'nan', 'nan', 'nan', '10.1128/JCM.43.12.5860-5864.2005'], ['ERR2095503', 'K-12 MG1655', 'cas3 PcseI CRISPRI', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR2095504', 'K-12 MG1655', 'cas3 PcseI', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR5934784', 'DH1', 'DH1 harboring pBbA5c and pTrc99A', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934785', 'DH1', 'DH1 harboring pBbA5c and pTrc99A', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934786', 'DH1', 'DH1 harboring pBbA5c and pTrc99A', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934790', 'DH1', 'DH1 harboring pJBEI-6829 and pJBEI-6833', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934791', 'DH1', 'DH1 harboring pJBEI-6829 and pJBEI-6833', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934796', 'DH1', 'DH1 harboring pJBEI-6829 and pJBEI-6833', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR5934797', 'DH1', 'DH1 harboring pJBEI-6829 and pJBEI-6833', 'nan', 'nan', 'nan', 'nan', '10.1016/j.ymben.2018.03.004'], ['SRR4069349', 'B', 'A43T mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069350', 'B', 'A43T mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069351', 'B', 'I15F mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069352', 'B', 'I15F mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069353', 'B', 'I15N mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069354', 'B', 'I15N mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069355', 'B', 'T231A mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR4069356', 'B', 'T231A mutant', 'Laboratory strain', 'nan', 'nan', 'nan', 'nan'], ['SRR6304627', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304628', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304630', 'K-12 MG1655', 'AMX resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304631', 'K-12 MG1655', 'AMX resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304632', 'K-12 MG1655', 'AMX resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304633', 'K-12 MG1655', 'CHL resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304634', 'K-12 MG1655', 'CHL resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304635', 'K-12 MG1655', 'CHL resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304636', 'K-12 MG1655', 'Tet resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304637', 'K-12 MG1655', 'Tet resistant', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR6304638', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1016/j.envint.2018.06.004'], ['SRR4255369', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR1824557', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'pH 5.5', '10.1038/ncomms8970'], ['SRR6651996', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6651998', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6651999', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652000', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652003', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652004', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652005', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652006', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652007', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652008', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652009', 'K-12 BW25113', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652011', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652012', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6652013', 'K-12 BW25113', 'rpoD E575V mutant', 'nan', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR6757591', 'K-12 MG1655', 'carrying plasmid pCRISP_ISF', 'Culture', 'nan', 'nan', 'IS-knockdown ', '10.1093/synbio/ysz008'], ['SRR6757592', 'K-12 MG1655', 'carrying plasmid pCRISP_ISF', 'Culture', 'nan', 'nan', 'IS-knockdown ', '10.1093/synbio/ysz008'], ['SRR6757593', 'K-12 MG1655', 'carrying plasmid pCRISP_ISF', 'Culture', 'nan', 'nan', 'IS-knockdown ', '10.1093/synbio/ysz008'], ['SRR6757594', 'K-12 MG1655', 'carrying plasmid pCRISPath', 'Culture', 'nan', 'nan', 'Control', '10.1093/synbio/ysz008'], ['SRR6757595', 'K-12 MG1655', 'carrying plasmid pCRISPath', 'Culture', 'nan', 'nan', 'Control', '10.1093/synbio/ysz008'], ['SRR6757596', 'K-12 MG1655', 'carrying plasmid pCRISPath', 'Culture', 'nan', 'nan', 'Control', '10.1093/synbio/ysz008'], ['SRR1824558', 'K-12 MG1655', 'WT', 'nan', 'nan', 'nan', 'pH 5.5', '10.1038/ncomms8970'], ['SRR1824560', 'K-12 MG1655', 'delta[gadE]', 'nan', 'nan', 'nan', 'pH 5.5', '10.1038/ncomms8970'], ['ERR779269', 'Avian pathogenic E. coli', 'WT', 'LB broth', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779270', 'Avian pathogenic E. coli', 'WT', 'Serum', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779271', 'Avian pathogenic E. coli', 'WT', 'Serum', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779272', 'Avian pathogenic E. coli', 'WT', 'Serum', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779273', 'Avian pathogenic E. coli', 'WT', 'LB broth', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779275', 'Avian pathogenic E. coli', 'arcA mutant', 'LB broth', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779276', 'Avian pathogenic E. coli', 'arcA mutant', 'Serum', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['ERR779280', 'Avian pathogenic E. coli', 'arcA mutant', 'Serum', 'nan', 'nan', 'microaerobic conditions', '10.1111/mmi.13987'], ['SRR6020037', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020038', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020040', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020041', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020042', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020043', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020044', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020045', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020046', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020047', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020051', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020052', 'UR40', 'nan', 'Homo sapiens', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020053', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020054', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020055', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020056', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020057', 'UR40', 'nan', 'Homo sapiens', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020058', 'UR40', 'nan', 'Homo sapiens', 'nan', '90min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020063', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020064', 'R7AC', 'nan', 'Canis lupus familiaris', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020067', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020068', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020069', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020070', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020071', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020072', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020073', 'UR40', 'nan', 'Homo sapiens', 'nan', '30min', 'Cefotaxime', '10.1038/s41598-018-34191-3'], ['SRR6020074', 'UR40', 'nan', 'Homo sapiens', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020075', 'UR40', 'nan', 'Homo sapiens', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR6020076', 'UR40', 'nan', 'Homo sapiens', 'nan', '0min', 'Control', '10.1038/s41598-018-34191-3'], ['SRR5282299', 'MG1693', 'WT', 'bacterial cells', 'nan', '3.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282300', 'MG1693', 'WT', 'bacterial cells', 'nan', '6min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282301', 'MG1693', 'WT', 'bacterial cells', 'nan', '8.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282302', 'MG1693', 'WT', 'bacterial cells', 'nan', '11min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282303', 'MG1693', 'WT', 'bacterial cells', 'nan', '13.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282304', 'MG1693', 'WT', 'bacterial cells', 'nan', '23.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282305', 'MG1693', 'WT', 'bacterial cells', 'nan', '3.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282306', 'MG1693', 'WT', 'bacterial cells', 'nan', '6min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282307', 'MG1693', 'WT', 'bacterial cells', 'nan', '11min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282308', 'SK4455', 'delta[rnc]', 'bacterial cells', 'nan', '3.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282309', 'SK4455', 'delta[rnc]', 'bacterial cells', 'nan', '6min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282310', 'SK4455', 'delta[rnc]', 'bacterial cells', 'nan', '8.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282311', 'SK4455', 'delta[rnc]', 'bacterial cells', 'nan', '11min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR5282312', 'SK4455', 'delta[rnc]', 'bacterial cells', 'nan', '13.5min', 'rifampicin', '10.1128/mBio.00128-17'], ['SRR1796598', 'K-12 MG1655', 'WT', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796599', 'K-12 MG1655', 'WT', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796600', 'K-12 MG1655', 'delta[oxyR]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796601', 'K-12 MG1655', 'delta[oxyR]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796602', 'K-12 MG1655', 'delta[soxR]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796603', 'K-12 MG1655', 'delta[soxR]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796604', 'K-12 MG1655', 'delta[soxS]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796605', 'K-12 MG1655', 'delta[soxS]', 'nan', 'nan', '20min', 'paraquat', '10.1016/j.celrep.2015.07.043'], ['SRR1796826', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796827', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796828', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796830', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796831', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796832', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796833', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796834', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796835', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796837', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796838', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796840', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796841', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796843', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796846', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796847', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796848', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796851', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796852', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['SRR1796854', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.25585/1487510'], ['ERR1940647', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940648', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940649', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940650', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940651', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940652', 'ETEC H10407', 'E873', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940653', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940654', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940655', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940656', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940657', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940658', 'ETEC H10407', 'E3035', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940659', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940660', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940661', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940662', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940663', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940664', 'ETEC H10407', 'E821', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940665', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940666', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940667', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940668', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940669', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940670', 'ETEC H10407', 'E828', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940671', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940672', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940673', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940674', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940675', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940676', 'ETEC H10407', 'E885', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940677', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940678', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940679', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940680', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940681', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940682', 'ETEC H10407', 'E939', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940683', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940684', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940685', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940686', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940687', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940688', 'ETEC H10407', 'E1360', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940689', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940690', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940691', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940692', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940693', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940694', 'ETEC H10407', 'E1526', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940695', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940696', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940697', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940698', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940699', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940700', 'ETEC H10407', 'E1659', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940701', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940702', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940703', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940704', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940705', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['ERR1940706', 'ETEC H10407', 'E5087', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR6671736', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'AZ-LolCDE', '10.1038/s41598-019-39387-9'], ['SRR6671738', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'AZ-LolCDE', '10.1038/s41598-019-39387-9'], ['SRR6671741', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'B-01 (new hit)', '10.1038/s41598-019-39387-9'], ['SRR6671744', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Ceftriaxone', '10.1038/s41598-019-39387-9'], ['SRR6671746', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Chloramphenicol', '10.1038/s41598-019-39387-9'], ['SRR6671748', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Ciprofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671750', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Ciprofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671751', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Clarithromycin', '10.1038/s41598-019-39387-9'], ['SRR6671753', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Clarithromycin', '10.1038/s41598-019-39387-9'], ['SRR6671755', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Colistin', '10.1038/s41598-019-39387-9'], ['SRR6671756', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Colistin', '10.1038/s41598-019-39387-9'], ['SRR6671758', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671760', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671762', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671764', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Doxycycline', '10.1038/s41598-019-39387-9'], ['SRR6671766', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Globomycin', '10.1038/s41598-019-39387-9'], ['SRR6671769', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Levofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671770', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Levofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671772', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Mecillinam', '10.1038/s41598-019-39387-9'], ['SRR6671774', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Mecillinam', '10.1038/s41598-019-39387-9'], ['SRR6671776', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Meropenem', '10.1038/s41598-019-39387-9'], ['SRR6671778', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Nitrofurantoin', '10.1038/s41598-019-39387-9'], ['SRR6671781', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Nitroxolin', '10.1038/s41598-019-39387-9'], ['SRR6671783', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Nitroxolin', '10.1038/s41598-019-39387-9'], ['SRR6671784', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Norfloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671786', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Norfloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671787', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'PolymyxineB', '10.1038/s41598-019-39387-9'], ['SRR6671789', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'PolymyxineB', '10.1038/s41598-019-39387-9'], ['SRR6671792', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'Trimethoprim', '10.1038/s41598-019-39387-9'], ['SRR6710205', 'nan', 'nan', 'Culture', 'nan', 'nan', 'NaHCO3', '10.1111/lam.13092'], ['SRR6710206', 'nan', 'nan', 'Culture', 'nan', 'nan', 'NaHCO3', '10.1111/lam.13092'], ['SRR6710207', 'nan', 'nan', 'Culture', 'nan', 'nan', 'NaCl', '10.1111/lam.13092'], ['SRR6710208', 'nan', 'nan', 'Culture', 'nan', 'nan', 'NaCl', '10.1111/lam.13092'], ['SRR6188477', 'EAEC 042', 'hha hha2 mutant', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12864-019-5683-4'], ['SRR6188478', 'EAEC 042', 'hns mutant', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12864-019-5683-4'], ['SRR6188479', 'EAEC 042', 'hns2 mutant', 'Culture', 'nan', 'nan', 'nan', '10.1186/s12864-019-5683-4'], ['SRR6671737', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'AZ-LolCDE', '10.1038/s41598-019-39387-9'], ['SRR6671739', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'B-01 (new hit)', '10.1038/s41598-019-39387-9'], ['SRR6671740', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'B-01 (new hit)', '10.1038/s41598-019-39387-9'], ['SRR6671742', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Ceftriaxone', '10.1038/s41598-019-39387-9'], ['SRR6671743', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Ceftriaxone', '10.1038/s41598-019-39387-9'], ['SRR6671745', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Chloramphenicol', '10.1038/s41598-019-39387-9'], ['SRR6671747', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Chloramphenicol', '10.1038/s41598-019-39387-9'], ['SRR6671749', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Ciprofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671752', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Clarithromycin', '10.1038/s41598-019-39387-9'], ['SRR6671754', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Colistin', '10.1038/s41598-019-39387-9'], ['SRR6671757', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671759', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671761', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'DMSO (control)', '10.1038/s41598-019-39387-9'], ['SRR6671763', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Doxycycline', '10.1038/s41598-019-39387-9'], ['SRR6671765', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Doxycycline', '10.1038/s41598-019-39387-9'], ['SRR6671767', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Globomycin', '10.1038/s41598-019-39387-9'], ['SRR6671768', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Globomycin', '10.1038/s41598-019-39387-9'], ['SRR6671771', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Levofloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671773', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Mecillinam', '10.1038/s41598-019-39387-9'], ['SRR6671775', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Meropenem', '10.1038/s41598-019-39387-9'], ['SRR6671777', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Meropenem', '10.1038/s41598-019-39387-9'], ['SRR6671779', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Nitrofurantoin', '10.1038/s41598-019-39387-9'], ['SRR6671780', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Nitrofurantoin', '10.1038/s41598-019-39387-9'], ['SRR6671782', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Nitroxolin', '10.1038/s41598-019-39387-9'], ['SRR6671785', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Norfloxacin', '10.1038/s41598-019-39387-9'], ['SRR6671788', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'PolymyxineB', '10.1038/s41598-019-39387-9'], ['SRR6671790', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Trimethoprim', '10.1038/s41598-019-39387-9'], ['SRR6671791', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'Trimethoprim', '10.1038/s41598-019-39387-9'], ['SRR1751292', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751293', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751294', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751295', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751296', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751297', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751298', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751299', 'K-12 MG1655', 'Crp_delta[Ar1]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751301', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751302', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751303', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751304', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751305', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751306', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751307', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751308', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751309', 'K-12 MG1655', 'Crp_delta[Ar1]delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751310', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751311', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751312', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751313', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751314', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751315', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751316', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751317', 'K-12 MG1655', 'Crp_delta[Ar2]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751318', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751319', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751320', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751321', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751322', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751323', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751324', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751325', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751326', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751327', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751328', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751329', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751330', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751331', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751332', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751333', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751334', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR1751335', 'K-12 MG1655', 'delta[Crp]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.pone.0197272'], ['SRR4244377', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR4244378', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR4244385', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cell.2016.09.004'], ['SRR6111092', 'K-12', 'nan', 'Animal faeces', 'nan', 'nan', 'no ciprofloxacin', '10.1101/cshperspect.a025320'], ['SRR6111094', 'K-12', 'nan', 'Bacterial pure culture', 'nan', 'nan', 'no ciprofloxacin', '10.1101/cshperspect.a025320'], ['SRR6111095', 'K-12 WT', 'nan', 'Bacterial pure culture', 'nan', 'nan', 'with ciprofloxaxin', '10.1101/cshperspect.a025320'], ['SRR6048166', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glucose ', '10.1093/nar/gky118'], ['SRR6048167', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glucose ', '10.1093/nar/gky118'], ['SRR6048168', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glucose ', '10.1093/nar/gky118'], ['SRR6048169', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glycerol', '10.1093/nar/gky118'], ['SRR6048170', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glycerol', '10.1093/nar/gky118'], ['SRR6048171', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with glycerol', '10.1093/nar/gky118'], ['SRR6048172', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048173', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048174', 'K-12 WT', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048175', '{delta}rho', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048176', '{delta}rho', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048177', '{delta}rho', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048178', '{delta}nusG', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048179', '{delta}nusG', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6048180', '{delta}nusG', 'nan', 'Culture', 'nan', 'nan', 'with UvsW', '10.1093/nar/gky118'], ['SRR6029690', 'O157:H7 (EDL933)', 'WT', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR6029691', 'O157:H7 (EDL933)', 'WT', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR6029692', 'O157:H7 (EDL933)', 'CraY47D', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR6029693', 'O157:H7 (EDL933)', 'CraY47D', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR6029694', 'O157:H7 (EDL933)', 'CraY47F', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR6029695', 'O157:H7 (EDL933)', 'CraY47F', 'Culture', 'nan', 'nan', 'nan', 'private'], ['SRR5304286', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't0', 'MG1655 - unexposed at t0', '10.1186/s12864-017-4413-z'], ['SRR5304287', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 - unexposed at t10', '10.1186/s12864-017-4413-z'], ['SRR5304288', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 - unexposed at t30', '10.1186/s12864-017-4413-z'], ['SRR5304289', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 - unexposed at t60', '10.1186/s12864-017-4413-z'], ['SRR5304290', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 + 3M HgCl2 at t10', '10.1186/s12864-017-4413-z'], ['SRR5304291', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 + 3M HgCl2 at t30', '10.1186/s12864-017-4413-z'], ['SRR5304292', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M HgCl2 at t60', '10.1186/s12864-017-4413-z'], ['SRR5304293', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 + 3M PMA at t10', '10.1186/s12864-017-4413-z'], ['SRR5304295', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M PMA at t60', '10.1186/s12864-017-4413-z'], ['SRR5304296', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't0', 'MG1655 - unexposed at t0', '10.1186/s12864-017-4413-z'], ['SRR5304297', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 - unexposed at t10', '10.1186/s12864-017-4413-z'], ['SRR5304298', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 - unexposed at t30', '10.1186/s12864-017-4413-z'], ['SRR5304299', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 - unexposed at t60', '10.1186/s12864-017-4413-z'], ['SRR5304300', 'K-12 MG1655', 'nan', 'Culture', 'nan', ' t10', 'MG1655 + 3M HgCl2 at t10', '10.1186/s12864-017-4413-z'], ['SRR5304301', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 + 3M HgCl2 at t30', '10.1186/s12864-017-4413-z'], ['SRR5304302', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M HgCl2 at t60', '10.1186/s12864-017-4413-z'], ['SRR5304303', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 + 3M PMA at t10', '10.1186/s12864-017-4413-z'], ['SRR5304304', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 + 3M PMA at t30', '10.1186/s12864-017-4413-z'], ['SRR5304305', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M PMA at t60', '10.1186/s12864-017-4413-z'], ['SRR5304306', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't0', 'MG1655 - unexposed at t0', '10.1186/s12864-017-4413-z'], ['SRR5304307', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 - unexposed at t10', '10.1186/s12864-017-4413-z'], ['SRR5304308', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 - unexposed at t30', '10.1186/s12864-017-4413-z'], ['SRR5304309', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 - unexposed at t60', '10.1186/s12864-017-4413-z'], ['SRR5304310', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 + 3M HgCl2 at t10', '10.1186/s12864-017-4413-z'], ['SRR5304311', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 + 3M HgCl2 at t30', '10.1186/s12864-017-4413-z'], ['SRR5304312', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M HgCl2 at t60', '10.1186/s12864-017-4413-z'], ['SRR5304313', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't10', 'MG1655 + 3M PMA at t10', '10.1186/s12864-017-4413-z'], ['SRR5304314', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't30', 'MG1655 + 3M PMA at t30', '10.1186/s12864-017-4413-z'], ['SRR5304315', 'K-12 MG1655', 'nan', 'Culture', 'nan', 't60', 'MG1655 + 3M PMA at t60', '10.1186/s12864-017-4413-z'], ['SRR5456715', 'K-12 MG1655', 'Glc1', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456722', 'K-12 MG1655', 'wild-type', 'Culture', 'nan', 'nan', 'glycerol', '10.1128/AEM.00410-17'], ['SRR5456724', 'K-12 MG1655', 'Gly2', 'Culture', 'nan', 'nan', 'glycerol', '10.1128/AEM.00410-17'], ['SRR5456726', 'K-12 MG1655', 'wild-type', 'Culture', 'nan', 'nan', 'xylose', '10.1128/AEM.00410-17'], ['SRR5456728', 'K-12 MG1655', 'Xyl2', 'Culture', 'nan', 'nan', 'xylose', '10.1128/AEM.00410-17'], ['SRR5456730', 'K-12 MG1655', 'Glc/Xyl Generalist', 'Culture', 'nan', 'nan', 'xylose', '10.1128/AEM.00410-17'], ['SRR5456732', 'K-12 MG1655', 'Ac1', 'Culture', 'nan', 'nan', 'acetate', '10.1128/AEM.00410-17'], ['SRR5456733', 'K-12 MG1655', 'Ac2', 'Culture', 'nan', 'nan', 'acetate', '10.1128/AEM.00410-17'], ['SRR5456714', 'K-12 MG1655', 'wild-type', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456716', 'K-12 MG1655', 'Glc2', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456717', 'K-12 MG1655', 'Glc3', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456719', 'K-12 MG1655', 'Glc/Ac Ac Specialist', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456720', 'K-12 MG1655', 'Glc/Gly Generalist', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456721', 'K-12 MG1655', 'Glc/Xyl Generalist', 'Culture', 'nan', 'nan', 'glucose', '10.1128/AEM.00410-17'], ['SRR5456723', 'K-12 MG1655', 'Gly1', 'Culture', 'nan', 'nan', 'glycerol', '10.1128/AEM.00410-17'], ['SRR5456725', 'K-12 MG1655', 'Glc/Gly Generalist', 'Culture', 'nan', 'nan', 'glycerol', '10.1128/AEM.00410-17'], ['SRR5456731', 'K-12 MG1655', 'wild-type', 'Culture', 'nan', 'nan', 'acetate', '10.1128/AEM.00410-17'], ['SRR5433917', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433918', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433919', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433920', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433921', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433922', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 ackA::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433923', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433924', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433925', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433927', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433928', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 dnaA46 ts', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433929', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433930', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433931', 'K-12 MG1655', 'K-12 ilvG rfb-50 rph-1 pta::kan', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433933', 'K-12 MG1655', 'K12 ilvG rfb-50 rph-1', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5433934', 'K-12 MG1655', 'K12 ilvG rfb-50 rph-1', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gkt444'], ['SRR5435830', 'Top10', 'Overexpressing tadA', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1101/gr.222760.117'], ['SRR5435832', 'Top10', 'WT', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1101/gr.222760.117'], ['SRR5435833', 'K-12 MG1655', 'EcM2.1 tadA (D64E) mutant', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1101/gr.222760.117'], ['SRR5435835', 'K-12 MG1655', 'EcM2.1 WT', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1101/gr.222760.117'], ['SRR5912822', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912823', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2.5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912824', 'K-12 MG1655', 'WT', 'Culture', 'nan', '5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912827', 'K-12 MG1655', 'csrA::kan (Tn10-mini transposon inserted after codon 51)', 'Culture', 'nan', '0min', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912828', 'K-12 MG1655', 'csrA::kan (Tn10-mini transposon inserted after codon 51)', 'Culture', 'nan', '2.5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912832', 'K-12 MG1655', 'WT', 'Culture', 'nan', '0min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912833', 'K-12 MG1655', 'WT', 'Culture', 'nan', '2.5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912834', 'K-12 MG1655', 'WT', 'Culture', 'nan', '5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912837', 'K-12 MG1655', 'csrA::kan (Tn10-mini transposon inserted after codon 51)', 'Culture', 'nan', '0min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912838', 'K-12 MG1655', 'csrA::kan (Tn10-mini transposon inserted after codon 51)', 'Culture', 'nan', '2.5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5912839', 'K-12 MG1655', 'csrA::kan (Tn10-mini transposon inserted after codon 51)', 'Culture', 'nan', '5min ', 'nan', '10.1038/s41467-017-01613-1'], ['SRR5262404', 'ATCC25922', 'nan', 'Homo Sapiens', 'nan', 'nan', 'nan', '10.1038/s41598-017-07798-1'], ['SRR5262405', 'ATCC25922', 'nan', 'Homo Sapiens', 'nan', 'nan', 'nan', '10.1038/s41598-017-07798-1'], ['SRR5262406', 'Poultry E. coli Virulent and multidrug resistant', 'nan', 'Fecal', 'nan', 'nan', 'nan', '10.1038/s41598-017-07798-1'], ['SRR941894', 'K-12 W3110', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1073/pnas.1315974111'], ['SRR941896', 'K-12 W3110', 'Mutant ', 'Culture', 'nan', 'nan', 'nan', '10.1073/pnas.1315974111'], ['SRR942319', 'K-12 W3110', 'Mutant ', 'Culture', 'nan', 'nan', 'nan', '10.1073/pnas.1315974111'], ['SRR1391073', 'DS1', 'hipQ', 'Culture', 'nan', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1391074', 'DS1', 'hipQ', 'Culture', 'nan', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1391075', 'DS1', 'hipQ', 'Culture', 'nan', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1391076', 'DS1', 'hipQ', 'Culture', 'nan', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR639781', 'K-12', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR640159', 'K-12', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR2288779', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR2288780', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR2288781', 'K-12 BW25113', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR922260', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922261', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922262', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922264', 'K-12 MG1655', 'WT + L-trp', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922266', 'K-12 MG1655', 'WT -O2', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922267', 'K-12 MG1655', 'WT -O2', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922268', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922270', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR922272', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.15252/msb.20145243'], ['SRR942318', 'K-12 W3110', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1186/gb-2012-13-5-r37'], ['SRR835954', 'K-12 MG1655', 'error-prone in-vitro MnCl2', 'nan', 'nan', 'nan', 'nan', '10.4161/trns.28285'], ['SRR835956', 'K-12 MG1655', 'error-prone in-vitro MnCl2', 'nan', 'nan', 'nan', 'nan', '10.4161/trns.28285'], ['SRR835953', 'K-12 MG1655', 'error-prone in-vitro MnCl2', 'nan', 'nan', 'nan', 'nan', '10.4161/trns.28285'], ['SRR835955', 'K-12 MG1655', 'error-prone in-vitro MnCl2', 'nan', 'nan', 'nan', 'nan', '10.4161/trns.28285'], ['SRR835957', 'K-12 MG1655', 'error-prone in-vitro MnCl2', 'nan', 'nan', 'nan', 'nan', '10.4161/trns.28285'], ['SRR2637695', 'O157:H7 str. EDL933', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.ppat.1006246, 10.1016/j.gene.2016.05.007'], ['SRR2637696', 'O157:H7 str. EDL933', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.ppat.1006246, 10.1016/j.gene.2016.05.007'], ['SRR2637697', 'O157:H7 str. EDL933', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.ppat.1006246, 10.1016/j.gene.2016.05.007'], ['SRR2637698', 'O157:H7 str. EDL933', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1371/journal.ppat.1006246, 10.1016/j.gene.2016.05.007'], ['SRR638077', 'K-12', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan'], ['SRR957825', 'K-12', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1186/1471-2164-13-734'], ['SRR957828', 'K-12', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1186/1471-2164-13-734'], ['SRR1481492', 'TH1269', 'hipA7', 'Culture', '37', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1481494', 'TH1269', 'hipA7', 'Culture', '37', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1481495', 'TH1269', 'hipA7', 'Culture', '37', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR1481496', 'TH1269', 'hipA7', 'Culture', '37', 'nan', 'addition of osmotic lysis solution and lysosyme lytic solution', '10.1371/journal.pone.0088660'], ['SRR3643540', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643541', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643543', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643544', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643545', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643546', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643547', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643548', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR3643552', 'ATCC 4157', 'nan', 'Culture', 'nan', 'nan', 'Gentamicin Sulfate', '10.1371/journal.pone.0164359'], ['SRR5985582', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985583', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985584', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985585', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985586', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985587', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985588', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985589', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985590', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985591', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985592', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985593', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985594', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985595', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985596', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985597', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985599', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR5985600', 'NEB 10-beta', 'nan', 'Culture', 'nan', 'nan', 'IPTG, aTc and L-arabinose', '10.15252/msb.20167461'], ['SRR1787592', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky069'], ['SRR1787596', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky069'], ['SRR1787597', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky069'], ['SRR1787598', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky069'], ['SRR1787599', 'K-12 MG1655', 'delta[gene]', 'Culture', 'nan', 'nan', 'nan', '10.1093/nar/gky069'], ['SRR4255370', 'nan', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1371/journal.pgen.1008456'], ['SRR5333337', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333339', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333340', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333341', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333342', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333343', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333344', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333349', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5333350', 'CUS2B', 'nan', 'Homo sapiens', 'nan', 'nan', 'range of anitbiotics', '10.1101/2019.12.16.878389'], ['SRR5416991', 'K-12', 'AR1-/AR2-', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416992', 'K-12', 'AR1-/AR2-', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416993', 'K-12', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416994', 'K-12', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416995', 'K-12', 'WT', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416996', 'K-12', 'K100Q', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416998', 'K-12', 'K100Q', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5416999', 'K-12', 'K100R', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5417000', 'K-12', 'K100R', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['SRR5417001', 'K-12', 'K100R', 'nan', 'nan', 'nan', 'nan', '10.1111/mmi.13874'], ['ERR1952051', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952052', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952053', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952054', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952055', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952056', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['ERR1952057', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nan', '10.1021/acssynbio.7b00462'], ['SRR6781997', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6781998', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6781999', 'K-12 MG1655', 'delta[yafC]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782000', 'K-12 MG1655', 'delta[yafC]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782001', 'K-12 MG1655', 'delta[yeiE]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782002', 'K-12 MG1655', 'delta[yeiE]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782003', 'K-12 MG1655', 'delta[yiaJ]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782004', 'K-12 MG1655', 'delta[yiaJ]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782005', 'K-12 MG1655', 'delta[yieP]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['SRR6782006', 'K-12 MG1655', 'delta[yieP]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-020-76120-3'], ['DRR100423', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100424', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100425', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100426', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100427', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100428', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100429', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100430', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100431', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100432', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100433', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100434', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100435', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100436', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['DRR100437', 'K-12 MG1655', 'delta[cyaA]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.cels.2018.05.015'], ['SRR6675352', 'DH5alpha mutant 7', 'LP7', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-018-05830-0'], ['SRR6675353', 'DH5alpha', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-018-05830-0'], ['SRR6675354', 'DH5alpha mutant 4', 'LP4', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41467-018-05830-0'], ['SRR6795932', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'vehicle treatment (DMSO)', '10.1016/j.chembiol.2018.05.007'], ['SRR6795933', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'vehicle treatment (DMSO)', '10.1016/j.chembiol.2018.05.007'], ['SRR6795935', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no treatment', '10.1016/j.chembiol.2018.05.007'], ['SRR6795936', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no treatment', '10.1016/j.chembiol.2018.05.007'], ['SRR6795938', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no treatment', '10.1016/j.chembiol.2018.05.007'], ['SRR6795939', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no treatment', '10.1016/j.chembiol.2018.05.007'], ['SRR6795940', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', 'no treatment', '10.1016/j.chembiol.2018.05.007'], ['SRR6795941', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM Ss54-1', '10.1016/j.chembiol.2018.05.007'], ['SRR6795943', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM Ss54-2', '10.1016/j.chembiol.2018.05.007'], ['SRR6795945', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM Ss54-3', '10.1016/j.chembiol.2018.05.007'], ['SRR6795949', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM s54-WT', '10.1016/j.chembiol.2018.05.007'], ['SRR6795950', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM s54-WT', '10.1016/j.chembiol.2018.05.007'], ['SRR6795951', 'K-12 MG1655', 'nan', 'Culture', 'nan', 'nan', '10 mM s54-WT', '10.1016/j.chembiol.2018.05.007'], ['SRR6799508', 'AS19-RrmA-', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1128/genomeA.00385-18'], ['SRR6799509', 'ARM9', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1128/genomeA.00385-18'], ['SRR6322033', 'MG1655 ', 'pBAD30-empty', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.molcel.2018.04.026'], ['SRR6322034', 'MG1655 ', 'pBAD30-empty', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.molcel.2018.04.026'], ['SRR6322035', 'MG1655 ', 'pBAD30-MazF', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.molcel.2018.04.026'], ['SRR8309839', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.ygeno.2019.07.013'], ['SRR8309842', 'K-12 MG1655', 'delta[fis]', 'Culture', 'nan', 'nan', 'nan', '10.1016/j.ygeno.2019.07.013'], ['SRR4435464', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-017-02110-7'], ['SRR4435465', 'K-12 MG1655', 'WT', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-017-02110-7'], ['SRR4435466', 'K-12 MG1655', 'delta[ompR]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-017-02110-7'], ['SRR4435467', 'K-12 MG1655', 'delta[ompR]', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-017-02110-7'], ['SRR8334999', 'W1688', 'delta[fimH]', 'Culture', 'nan', 'nan', 'nan', '10.3389/fmicb.2019.01773'], ['SRR8335000', 'W1688', 'delta[fimH]', 'Culture', 'nan', 'nan', 'nan', '10.3389/fmicb.2019.01773'], ['SRR8335001', 'W1688', 'delta[fimH]', 'Culture', 'nan', 'nan', 'nan', '10.3389/fmicb.2019.01773'], ['SRR8335002', 'W1688', 'delta[fimH]', 'Culture', 'nan', 'nan', 'nan', '10.3389/fmicb.2019.01773'], ['SRR8335003', 'W1688', 'delta[fimH]', 'Culture', 'nan', 'nan', 'nan', '10.3389/fmicb.2019.01773'], ['SRR7524562', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524563', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524564', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524565', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524566', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524567', 'O104:H4 C227/11 phi cu', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524568', 'O157:H- 3072/96', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524569', 'O157:H- 3072/97', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524570', 'O157:H- 3072/98', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524571', 'O157:H- 3072/99', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524572', 'O157:H- 3072/100', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524573', 'O157:H- 3072/101', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524574', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524575', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524576', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524577', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524578', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR7524579', 'O157:H7 Sakai', 'nan', 'Homo sapiens', 'nan', 'nan', 'nan', '10.1186/s12866-019-1578-4'], ['SRR6919875', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919876', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919877', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919878', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919879', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919880', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919881', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR6919882', 'K-12 MG1655', 'nan', 'nan', 'nan', 'nan', 'nine fatty acid compund exposure during antibiotic treatment', '10.1007/s00253-018-9271-3'], ['SRR9304018', 'K-12 BW25113', 'nan', 'Culture', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304019', 'K-12 BW25113', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304020', 'K-12 BW25113', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304022', 'K-12 BW25113', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304087', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304021', 'K-12 BW25113', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304023', 'K-12 BW25113', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304088', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304089', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304090', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304091', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['SRR9304092', 'WO153', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1038/s41598-019-39492-9'], ['ERR2868174', 'K-12 MG1655', 'pNT3_bamC', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868175', 'K-12 MG1655', 'pNT3_bamC', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868176', 'K-12 MG1655', 'pNT3_cusA', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868177', 'K-12 MG1655', 'pNT3_cusA', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868178', 'K-12 MG1655', 'pNT3_hemX', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868179', 'K-12 MG1655', 'pNT3_hemX', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868181', 'K-12 MG1655', 'pNT3 empty vector', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868182', 'K-12 MG1655', 'pNT3_topA', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868183', 'K-12 MG1655', 'pNT3_topA', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868184', 'K-12 MG1655', 'pNT3_yicR', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868185', 'K-12 MG1655', 'pNT3_yicR', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['ERR2868186', 'K-12 MG1655', 'pNT3_yqjD', 'Culture ', 'nan', 'nan', 'Mutant or truncated genes were synthesised to introduce site-specific mutations or small deletions in pUC57 backbone', '10.1016/j.cell.2018.12.008'], ['SRR8449235', 'K-12', 'WT', 'Culture ', '37C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR8449236', 'K-12', 'WT', 'Culture ', '37C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR8449237', 'K-12', 'WT', 'Culture ', '37C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR8449238', 'K-12', 'WT', 'Culture ', '37C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR9257363', 'K-12 MG1655', 'WT', 'Culture ', 'nan', 'nan', 'nan', '10.1093/molbev/msu209'], ['SRR9257366', 'K-12 MG1655', 'Evolved', 'Culture ', 'nan', 'nan', 'nan', '10.1093/molbev/msu209'], ['SRR9257367', 'K-12 MG1655', 'Evolved', 'Culture ', 'nan', 'nan', 'nan', '10.1093/molbev/msu209'], ['SRR9257370', 'K-12 MG1655', 'Evolved', 'Culture ', 'nan', 'nan', 'nan', '10.1093/molbev/msu209'], ['SRR6941199', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941200', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941201', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941202', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941203', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941204', 'K12 MG1655 (PMF2)', 'WT Ancestor', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941205', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941206', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR6941207', 'K12 MG1655 (PMF2)', 'Clone', 'Culture ', 'nan', 'nan', 'nan', '10.1073/pnas.1708371115'], ['SRR8369110', 'O157:H7', 'nan', 'attached cell culture', 'nan', 'nan', 'nan', '10.1186/s12866-020-02043-8'], ['SRR8369111', 'O157:H7', 'nan', 'attached cell culture', 'nan', 'nan', 'nan', '10.1186/s12866-020-02043-8'], ['SRR8369112', 'O157:H7', 'nan', 'planktonic cell culture', 'nan', 'nan', 'nan', '10.1186/s12866-020-02043-8'], ['SRR8369113', 'O157:H7', 'nan', 'planktonic cell culture', 'nan', 'nan', 'nan', '10.1186/s12866-020-02043-8'], ['ERR2433509', 'K-12 BW25113', 'nan', 'Culture ', '37C', '1min', 'nan', '10.1371/journal.pgen.1007354'], ['SRR6967731', 'K12 MG1655 (PMF2)', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1128/AEM.00404-18'], ['SRR6967732', 'K12 MG1655 (PMF2)', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1128/AEM.00404-18'], ['ERR2955747', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 glycerol', '10.1016/j.dib.2018.11.136'], ['ERR2955748', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 glycerol', '10.1016/j.dib.2018.11.136'], ['ERR2955752', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 + 40 % (v/v) spinach leaf lysate', '10.1016/j.dib.2018.11.136'], ['ERR2720670', 'EAEC 042', 'WT', 'Culture ', 'nan', 'nan', 'nan', '10.1111/mmi.14172'], ['ERR2720672', 'EAEC 042', 'WT', 'Culture ', 'nan', 'nan', 'nan', '10.1111/mmi.14172'], ['ERR2720673', 'EAEC 042', 'delta[aggR]', 'Culture ', 'nan', 'nan', 'nan', '10.1111/mmi.14172'], ['ERR2720674', 'EAEC 042', 'delta[aggR]', 'Culture ', 'nan', 'nan', 'nan', '10.1111/mmi.14172'], ['ERR2720675', 'EAEC 042', 'delta[aggR]', 'Culture ', 'nan', 'nan', 'nan', '10.1111/mmi.14172'], ['ERR2433508', 'K-12 BW25113', 'nan', 'Culture ', '37C', '0min', 'nan', '10.1371/journal.pgen.1007354'], ['ERR2433564', 'K-12 BW25113', 'nan', 'Culture ', '44C', '10min', 'nan', '10.1371/journal.pgen.1007354'], ['SRR6967733', 'K12 MG1655 (PMF2)', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1128/AEM.00404-18'], ['ERR2955749', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 glycerol', '10.1016/j.dib.2018.11.136'], ['ERR2955750', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 + 40 % (v/v) spinach leaf lysate', '10.1016/j.dib.2018.11.136'], ['ERR2955751', 'Sakai', 'O157:H7', 'Culture ', 'nan', 'nan', 'M9 + 40 % (v/v) spinach leaf lysate', '10.1016/j.dib.2018.11.136'], ['SRR6908225', 'FHI4', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908226', 'FHI4', 'HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908227', 'FHI6', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908231', 'FHI8', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908232', 'FHI8', 'HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908233', 'FHI9', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908234', 'FHI9', 'HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908236', 'FHI12', 'HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908237', 'FHI24', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908239', 'FHI25', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908242', 'FHI27', 'HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908243', 'FHI36', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908245', 'FHI43', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908247', 'FHI48', 'HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908251', 'FHI95', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908252', 'FHI95', 'non-HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908253', 'St. Olav40', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908257', 'St. Olav157', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908265', 'St. Olav179', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908266', 'St. Olav179', 'non-HUS', 'Culture ', 'nan', 'nan', 'induced', '10.3389/fmicb.2018.01416'], ['SRR6908269', 'FHI66', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6908276', 'St. Olav63', 'non-HUS', 'Culture ', 'nan', 'nan', 'non-induced', '10.3389/fmicb.2018.01416'], ['SRR6848183', 'DH10B', 'nan', 'Culture ', 'nan', 'nan', 'nan', '10.1093/nar/gky486'], ['SRR6841088', 'K-12 MG1655', 'WT', 'Culture ', 'nan', 'nan', 'MOPS minimal media + glycerol', '10.1128/JB.00411-18'], ['SRR6841095', 'K-12 MG1655', 'WT', 'Culture ', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841106', 'K-12 MG1655', 'Lrp::kanR', 'Culture ', 'nan', 'nan', 'MOPS minimal media + glycerol', '10.1128/JB.00411-18'], ['SRR6841112', 'K-12 MG1655', 'Lrp::kanR', 'Culture ', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841114', 'K-12 MG1655', 'Lrp::kanR', 'Culture ', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['SRR6841115', 'K-12 MG1655', 'Lrp::kanR', 'Culture ', 'nan', 'nan', 'MOPS minimal media + LIV + glycerol', '10.1128/JB.00411-18'], ['ERR2226503', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226504', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226505', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226506', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226507', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226508', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226509', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226510', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226511', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226512', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226513', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226514', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226515', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226516', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226517', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226518', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226519', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226520', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226521', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226522', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226523', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226524', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226525', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226526', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226527', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226528', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226529', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226530', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226531', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226533', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226534', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226535', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226536', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226537', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226538', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226539', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226540', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226541', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226542', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226543', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226544', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226545', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226546', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226547', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226548', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226549', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226550', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226551', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226552', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226553', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226554', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226555', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226556', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226557', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'In-cell replicate, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226558', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226559', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226560', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226561', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226562', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226563', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226564', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226565', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226566', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226567', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226568', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226569', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226570', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226571', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226572', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226573', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226574', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226575', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226576', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226577', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226578', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226579', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226580', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226581', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226582', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Kasugamycin, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226583', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226584', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226585', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226586', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226587', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226588', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226589', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226590', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226591', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226592', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226593', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226594', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226595', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226596', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Modified', '10.1016/j.cell.2018.02.034'], ['ERR2226597', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['ERR2226598', 'K-12 MG1655', 'nan', 'Culture ', 'nan', 'nan', 'Extracted, Untreated', '10.1016/j.cell.2018.02.034'], ['SRR7609035', 'K-12', 'WT', 'Culture ', '37C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR7609038', 'K-12', 'RNase E mutation', 'Culture ', '44C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR7609040', 'K-12', 'RNase E mutation', 'Culture ', '44C', 'nan', 'nan', '10.1038/s41564-019-0500-z'], ['SRR7614719', 'K-12', 'nan', 'nan', 'nan', 'nan', 'DMP', 'nan'], ['SRR7614718', 'K-12', 'nan', 'nan', 'nan', 'nan', 'DMP', 'nan'], ['SRR7614720', 'K-12', 'nan', 'nan', 'nan', 'nan', 'CK', 'nan'], ['SRR9191918', 'K-12 MG1655', 'GLU', 'nan', 'nan', 'nan', 'pH 7', '10.1099/mic.0.000867'], ['SRR9191919', 'K-12 MG1655', 'GLU', 'nan', 'nan', 'nan', 'pH 5.5', '10.1099/mic.0.000867'], ['SRR9191920', 'K-12 MG1655', 'AA2', 'nan', 'nan', 'nan', 'pH 7', '10.1099/mic.0.000867'], ['SRR9191921', 'K-12 MG1655', 'AA2', 'nan', 'nan', 'nan', 'pH 5.5', '10.1099/mic.0.000867'], ['SRR9191922', 'K-12 MG1655', 'AA6', 'nan', 'nan', 'nan', 'pH 7', '10.1099/mic.0.000867'], ['SRR9191923', 'K-12 MG1655', 'AA6', 'nan', 'nan', 'nan', 'pH 5.5', '10.1099/mic.0.000867'], ['SRR9191924', 'K-12 MG1655', 'LM2', 'nan', 'nan', 'nan', 'pH 7', '10.1099/mic.0.000867'], ['SRR9191925', 'K-12 MG1655', 'LM2', 'nan', 'nan', 'nan', 'pH 5.5', '10.1099/mic.0.000867'], ['SRR9191926', 'K-12 MG1655', 'MES1', 'nan', 'nan', 'nan', 'pH 7', '10.1099/mic.0.000867'], ['SRR9191927', 'K-12 MG1655', 'MES1', 'nan', 'nan', 'nan', 'pH 5.5', '10.1099/mic.0.000867']]\n",
            "414\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VLvr1xSWwyc6",
        "outputId": "5fec7d86-7c28-41d7-b228-74dd7964044c"
      },
      "source": [
        "#Compiling GO and gene descriptions from interproscan\n",
        "s = open(\"/content/analysis_results (1).tsv\", \"r\").readlines()\n",
        "geneanno = open(\"c_jejuni_GOterms.txt\", \"w\")\n",
        "\n",
        "all = {}\n",
        "allgenes = []\n",
        "for i in s:\n",
        "  total = i.split(\"\\n\")[0].split(\"\\t\")\n",
        "  gene = total[0]\n",
        "  allgenes.append(gene)\n",
        "  if gene not in all:\n",
        "    all[gene] = total[-1]\n",
        "  else:\n",
        "    if all[gene] != total[-1]:\n",
        "      if total[-1] != \"-\":\n",
        "        all[gene] += str(\",\" + total[-1])\n",
        "\n",
        "print(len(set(allgenes)))\n",
        "print(len(all))\n",
        "\n",
        "for i in all:\n",
        "  geneanno.writelines(i + \"\\t\" + all[i]+ \"\\n\")\n",
        "geneanno.close()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "1447\n",
            "1447\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "y0q6o3Ng8_Bx",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "fe97fd48-cd87-45f8-9b75-4d70576c8446"
      },
      "source": [
        "#GO terms with scores\n",
        "import pandas as pd\n",
        "dfanalysis_results = pd.read_csv(\"/content/analysis_results (1).tsv\", sep = \"\\t\")\n",
        "print(dfanalysis_results)\n",
        "df = dfanalysis_results.iloc[:, [0,8,13]]\n",
        "\n",
        "df.to_csv('C.jejuni_GOanddatabasesMatchscores.csv', index = False, header = [\"GeneID\", \"DatabaseMatchScores\", \"GOterms\"])"
      ],
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "      OIT94561  ...                                 -\n",
            "0     OIT94561  ...                                 -\n",
            "1     OIT95684  ...                        GO:0016783\n",
            "2     OIT94124  ...                                 -\n",
            "3     OIT94124  ...             GO:0004315|GO:0006633\n",
            "4     OIT94111  ...                                 -\n",
            "...        ...  ...                               ...\n",
            "2088  OIT96066  ...                                 -\n",
            "2089  OIT94504  ...                                 -\n",
            "2090  OIT94504  ...                                 -\n",
            "2091  OIT95237  ...  GO:0003735|GO:0005840|GO:0006412\n",
            "2092  OIT95237  ...  GO:0003735|GO:0005840|GO:0006412\n",
            "\n",
            "[2093 rows x 14 columns]\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "blGPIEdsVJvJ"
      },
      "source": [
        "#interproscan\n",
        "!wget ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.51-85.0/interproscan-5.51-85.0-64-bit.tar.gz\n",
        "!tar -pxzf interproscan-5.51-85.0-64-bit.tar.gz\n",
        "!wget ftp://ftp.ensemblgenomes.org/pub/release-51/bacteria//fasta/bacteria_67_collection/campylobacter_jejuni_gca_001877255/pep/Campylobacter_jejuni_gca_001877255.ASM187725v1.pep.all.fa.gz\n",
        "!gunzip Campylobacter_jejuni_gca_001877255.ASM187725v1.pep.all.fa.gz\n",
        "!/content/interproscan-5.51-85.0/interproscan.sh -appl Pfam -goterms -i /content/Campylobacter_jejuni_gca_001877255.ASM187725v1.pep.all.fa -b analysis_results"
      ],
      "execution_count": null,
      "outputs": []
    }
  ]
}